---

title: Cell culture model for acquired chemoresistance of chronic myelogenous leukemia and related methods for identifying agents to overcome resistance
abstract: A method of generating a chronic myelogenic leukemia (CML) acquired chemoresistant culture model is provided. Such a method may comprise providing a na√Øve blast crisis CML cell line; administering/contacting the cell line with a mutation-inducing dose of imatinib; maintaining a culture of the treated cell line for a period of time until the treated cell line relapses and repopulates the culture; and determining the repopulated cell culture is a CML acquired chemoresistant cell line by detecting a BCR-ABL mutation, wherein the acquired chemoresistance is achieved by a BCR-ABL mutation.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09176114&OS=09176114&RS=09176114
owner: CITY OF HOPE
number: 09176114
owner_city: Duarte
owner_country: US
publication_date: 20130619
---
This application is a continuation of U.S. patent application Ser. No. 12 901 139 filed Oct. 8 2010 which is a continuation in part application of U.S. patent application Ser. No. 12 026 554 filed Feb. 5 2008 which claims the benefit of U.S. Provisional Patent Application Ser. No. 60 888 307 filed Feb. 5 2007 all of which are incorporated herein by reference in their entirety.

The present invention was supported by the Department of Defense Grant No. W81XWH 06 1 0268 . The government may have certain rights in the present invention.

Epigenetic disruptions of gene expression such as by DNA methylation and histone modifications are profoundly involved in tumorigenesis. For leukemia the gene hypermethylated in cancer 1 HIC1 is unique because hypermethylation of the gene s promoter region occurs progressively towards the late phases of hematologic malignancies. HIC1 encodes a DNA binding zinc finger transcriptional factor that is essential for mammalian development. The HIC1 gene is inactivated but not mutated in certain human cancers such as chronic myelogeneous leukemia CML and relapsed acute lymphocytic leukemias following chemotherapy. Using mouse genetics the importance of HIC1 in of tumorigenesis has recently been demonstrated. Germline disruption of one copy of HIC1 predisposes mice to a late on set and gender dependent spectrum of malignant tumors wherein promoter hypermethylation of the wild type HIC1 allele is associated with loss of function of this gene. It is also known that HIC1 plays a synergistic role with p53 in suppressing the development of age dependent cancers. Germline disruption of one copy each of HIC1 and p53 on opposite trans chromosomes or same cis chromosomes in mice results in altered tumor spectrum earlier appearance and increased prevalence and aggressiveness of osteosarcomas. Indeed a low frequency of blast crisis megakaryocytic leukemia is found in cis HIC1 and p53 double heterozygous mice. See .

A key mechanism by which HIC1 suppresses tumorigenesis is through its regulation of the stress and DNA damage responsive gene SIRT. SIRT1 is a mammalian orthologue of yeast silent information regulator 2 Sir2 that is required for yeast lifespan extension upon calorie restriction Lin et al. 2000 . An extra copy of Sir2 extends life span in yeast fly and worm. SIRT1 is a class III histone deacetylase whose enzymatic activity is dependent on cofactor NAD. SIRT1 is insensitive to histone deacetylase inhibitor trichostatin A TSA which inhibits class I and II deacetylases. SIRT1 is involved in regulation of a variety of cellular functions including survival glucose homeostasis and fat metabolism through deacetylating histones and non histone proteins Saunders Verdin 2007 . SIRT1 levels increase in response to metabolic stresses such as calorie restriction and nutrient starvation Cohen et al. 2004 Nemoto et al. 2004 . The mammalian SIRT1 promotes cell survival under oxidative stress genotoxic stress and DNA damage through multiple substrates including p53 Luo et al. 2001 Vaziri et al. 2001 Ku70 Cohen et al. 2004 and FOXO proteins. For example SIRT1 promotes cell survival using p53 via a pathway that includes deacetylation of p53 and attenuation of its ability to activate downstream targets to control apoptosis. HIC1 forms a complex with SIRT1 protein. This HIC1 SIRT1 protein complex directly binds to the SIRT1 promoter in vivo to repress SIRT1 gene transcription. Loss of HIC1 expression by promoter hypermethylation upregulates SIRT1 in cancer cells attenuates p53 activity by deacetylation and allows cells to bypass apoptosis and survive stress and DNA damage. Inhibition of SIRT1 function in cells without HIC1 abolishes the resistance to apoptosis.

Chronic myelogenous leukemia CML is a fatal hematopoietic disorder resulting from malignant transformation of bone marrow progenitor cells. The disease progresses from chronic phase to accelerated phase to terminal blast crisis phase. CML is characterized by a reciprocal translocation of chromosome 9 and 22 that creates an oncogenic fusion gene BCR ABL. This gene produces a protein with deregulated BCR ABL tyrosine kinase activity. Imatinib mesylate also known as imitanib Gleevac or STI 571 is a potent ABL tyrosine kinase inhibitor. In most chronic phase patients treatment with imatinib results in complete cytogenic responses CCR or remission and infrequent relapse. However in most blast crisis patients there is a poor response to imatinib treatment and a high frequency of relapse in those patients having an initial response. The molecular mechanisms of the resistance to imatinib may consist of both BCR ABL dependent and independent pathways. BCR ABL dependent pathways are characterized by genetic alterations of the BCR ABL gene.

The clinical resistance to imatinib treatment is mediated primarily by genetic mutations of the BCR ABL kinase domain and to a lesser extent by amplification of the BCR ABL gene. In relapsed CML patients more than 15 BCR ABL mutations have been identified. These mutations confer various degrees of resistance to imatinib. Mechanisms for formation of BCR ABL mutations in CML are not clear. The vast majority of BCR ABL mutations are detected in relapsed patients but pre existing mutations including a T315I mutation are also found in patients before imatinib treatment Gorre et al. 2001 Shah et al. 2002 . The T315I mutation has been identified thus far as being frequent and the most resistant mutation. Located in the center of the imatinib binding site is Thrand the T315I mutation blocks the drug from binding to the ABL kinase. In addition imatinib suppresses proliferation of human CML leukemic progenifor cells but cannot eliminate them in vivo. Thus most subjects in CCR or remission continue to harbor residual leukemia cells Bhatia et al. 2003 . Similarly imatinib treatment results in remission of CML in mouse models of the disease but fails to eliminate leukemic stem cells Hu et al. 2006 Neering et al. 2007 . Because of this in vitro studies of the process by which BCR ABL is mutated in CML cells is difficult because unlike what occurs in vivo nearly all CML cell lines derived from blast crisis CML are sensitive to 1 M STI 571 treatment. Deininger et al. 1997 . Nilotinib AMN107 is a recently developed BCR ABL inhibitor having greater potency. It inhibits most of the known mutants with the exception of the T315I mutation. Similarly the potent dual SRC ABL kinase inhibitor dasatinib BMS 354825 inhibits 14 of 15 BCR ABL mutants but not T315I. However in vivo CML patients with a T315I mutation do not respond to either nilotinib or dasatinib. Without further effective treatment these blast crisis patients are terminal. Accordingly a method of treating these relapsed patients or preventing formation of this resistant mutation is highly desired.

Several resistant CML cell lines have been developed by gradually exposing cells to increasing concentrations of STI 571 Mahon et al. 2000 . However these resistant cell lines all have BCR ABL gene amplification but lack mutations. This is opposite to the results seen in patients. Today most in vitro mutation studies are carried out using murine cell lines such as Ba F3 cells a murine pro B cell line transfected with genetically engineered BCR ABL mutations. La Rosee et al. 2002 Shah et al. 2004 von Bubnoff et al. 2006 von Bubnoff et al. 2005 Weisberg et al. 2005 . Although these cell lines are important for addressing mutant kinase activity they do not reflect in vivo mechanisms of BCR ABL mutagenesis and thereby cannot be used to address mechanisms of BCR ABL mutagenesis in natural cellular and molecular contexts of CML and cannot be applied to development of strategies for preventing such mutations. The use of these cells also excludes the possibility of studying other genetic and epigenetic alterations accompanying the BCR ABL mutagenesis process in mutant CML cells. Thus a CML cell line having one or more BCR ABL mutations is also highly desired because it is useful as a model system for CML disease. Use of such cell lines will facilitate further study of the mechanism of disease development and progression and assist in the further identification of therapeutic treatments for this disease.

A method for making a model of acquired resistance in human cancers which simulates a patient s chemoresistance response is provided. Such a method allows for the study of mechanisms involved in the development of acquired resistance such as through induction of genetic mutations of oncogenic tyrosine kinases as well as through alteration of DNA damage response pathways. Cell line models are also provided. Use of these model systems allow for the screening identification testing and discovery of therapeutic compounds useful for treating cancer and especially cancer relapse from chemoresistant cell growth.

A novel CML resistance model using na ve blast crisis CML cells is also provided. Using this model key features of clinical resistance such as rapid growth of cancer cells after direct exposure of the cancer cells to therapeutically effective concentrations of chemotherapeutic compounds can be replicated. Cancer cells which survive chemotherapeutic treatment also referred to herein as relapse growth exhibit genetic mutations at the BCR ABL locus. This model will serve as a useful system for designing testing screening and identifying new therapeutic strategies for treating chemoresistant cancer. The methods provided herein are also useful for developing cell culture models of acquired resistance in solid tumors such as non small cell lung carcinomas. Over active SIRT1 pathway is a mechanism by which acquired resistance develops in but not limited to hematologic malignancies. The present invention provides treatment and diagnostic modalities based upon this discovery.

In one embodiment a method of generating a chronic myelogenic leukemia CML acquired chemoresistant culture model is provided. Such a method comprises providing a na ve blast crisis CML cell line contacting the cell line with a mutation inducing dose of imatinib maintaining a culture of the treated cell line for a period of time until the treated cell line relapses and repopulates the culture and determining the repopulated cell culture is a CML acquired chemoresistant cell line by detecting a BCR ABL mutation on the native BCR ABL locus wherein the acquired chemoresistance is achieved by the mutation.

In some embodiments the na ve blast crisis CML cell line is a KCL 22 cell line. In another embodiment the mutation inducing dose is a about 2.5 M or higher b between about 2.5 M and about 10 M or c selected from about 2.5 M about 5 M and about 10 M. In another embodiment the BCR ABL mutation is a BCR ABL kinase domain mutation. In one aspect the BCR ABL kinase domain mutation is a T315I BCR ABL mutation.

In another embodiment a SIRT1 inhibitor compound for preventing chemoresistance or treating CML and a method for developing such a SIRT1 inhibitor for treating cancer is provided. Such a method may comprise generating a pharmacophore model of SIRT1 inhibitors for a SIRT1 substrate binding pocket identifying one or more SIRT1 inhibitor test compounds based on the pharmacophore model administering the one or more SIRT1 inhibitor test compounds to a CML acquired resistant culture model determining an effective test dosage for the one or more SIRT1 inhibitor test compounds that blocks CML cell relapse comparing the effective test dosage to an effective dosage of a known inhibitor such as sirtinol and selecting one or more of the SIRT1 inhibitor test compounds as a SIRT1 inhibitor lead compound when the effective test dosage is lower than the effective known inhibitor dosage. In some embodiments the CML acquired resistant culture model is a culture model produced by the presently described methods.

Another embodiment is the generation of a pharmacophore model for discovering any compound or combination of compounds useful for treating or preventing chemoresistance and or cancer. The pharmacophore model may comprise one or more of the steps of initiating mutation identifying the mutation identifying the cells that have undergone cellular repair and testing survival of the mutant cells. The mutation or mutations may be from a variety of cancer related genes. There may be other components that also have effect on blocking cancer relapse from drug treatment that can be added to the model. Candidate therapeutically effective drugs may then be tested against the pharmacophore model for efficacy alone or in comparison to a known cancer treatment standard. Preferably the cancer is CML and the treatment standard is sirtinol.

SIRT1 inhibitor lead compounds may be optimized. In some aspects optimizing the one or more lead compounds is accomplished by using the one or more lead compound to improve the pharmacophore model. In other aspects optimizing the one or more lead compounds is accomplished by isothermal titration calorimetry ITC . In some embodiments the method further comprises determining the efficacy of the one or more lead compounds and or determining the cytotoxicity of the one or more lead compounds.

In another embodiment a method for inhibiting or delaying tumor growth in a cancer patient is provided. The cancer may be any cancer including but not limited to leukemia. In some embodiments the cancer is chronic myelogenous leukemia CML . Such a method may comprise administering an effective amount of a SIRT1 inhibitor to the cancer patient. In some embodiments the SIRT1 inhibitor is one or a combination of sirtinol a sirtinol analogue splitomicin a splitomicin analogue napthol a napthol derivative an indole an indole derivative siRNA shRNA and antisense RNA. In other embodiments the SIRT1 inhibitor is a lead compound produced by the methods described herein.

In other embodiments the method may further comprise administering an effective dose of a tyrosine kinase inhibitor in combination with the SIRT1 inhibitor. In some aspects the tyrosine kinase inhibitor is one or a combination of imatinib gefitinib nilotinib dasatinib and VX 680. In other aspects the tyrosine kinase inhibitor may be any other suitable substance that inhibits tyrosine kinase activity.

In yet other embodiments methods of treating or preventing BCR ABL drug resistance in chronic myelogenous leukemia comprise administering a therapeutically effective amount of one or more SIRT1 modulators to the cells or subject in need thereof. Thus in one aspect the invention provides a method of inhibiting growth of a tumor cell. If the tumor cell is a leukemia cell it may be chronic myelogenous leukemia acute myelogenous leukemia chronic lymphocytic leukemia acute lymphocytic leukemia or hairy cell leukemia. In another aspect the invention provides a method of inhibiting relapse growth of cancer cells or preventing chemoresistance. This method comprises use of a modulator of SIRT1 such as a SIRT1 inhibitor. A modulator includes inhibitors of SIRT1 protein inhibitors of the SIRT1 gene transcription or translation inhibition or both . A SIRT1 inhibitor may be a napthol compound for example sirtinol or splitomicin an indole siRNA a derivative of a SIRT1 inhibitor an analogue of a SIRT1 inhibitor or any combination thereof. Chemoresistance such as BCR ABL drug resistance may be caused by administration of a cancer treating drug such as STI 571 imatinib nilotinib dasatinib or another cancer treating drug. Alternatively drug resistance may be caused by another drug or compound administered to the patient environmental factors or may be a naturally occurring resistance. The drug resistance may be the result of a genetic mutation such as the T315I mutation of the BCR ABL gene. The SIRT1 inhibitor may also treat or prevent insulin and transferrin induced resistance.

A cell culture model for chronic myelogenous leukemia CML acquired chemoresistance and related methods are provided herein. Compounds for treating CML and methods for the discovery of those compounds are also provided. The following definitions are to assist with review of the present disclosure and are meant to encompass any equivalents.

 Agonist refers to a ligand that interacts with or binds to its receptor to up regulate accelerate or activate the activity of a compound receptor gene or protein.

 Antagonist refers to a ligand that interacts with or binds to its receptor to downregulate suppress or inhibit the activity of a compound receptor gene or protein.

 Antisense molecule refers to a nucleic acid molecule that hybridizes to all or a portion of a target gene or all or a portion of an mRNA encoded by a target gene. Hybridization of an antisense molecule to a target gene or a portion thereof may inhibit expression of the target gene by inhibiting transcription of the gene sequence while hybridization of an antisense molecule to a transcript encoded by a target gene may inhibit expression of the target gene by inhibiting translation of the transcript into a polypeptide sequence.

 Chemotherapeutic agent refers to any chemical compound or treatment method that induces cell damage results in cell death or both. Specific chemotherapeutic agents include but are not limited to kinase inhibitors such as tyrosine kinase inhibitors TKI which include imatinib gefitinib erlotinib lapatinib CI 1033 PKI 166 GW 2016 as well as others that will be known to one of ordinary skill in the art. Other chemotherapeutic agents include imatiactinomycin D adriamycin androgens asparagine azathioprine BCG bleomycin camptothecin cisplatin epirubicin etopside gemcitabine hydroxyurea interferon alpha interferon beta interferon gamma mitomycin C paclitaxel thioguanine 5 fluorouracil 6 mercaptopurine or other drugs. In addition chemotherapeutic agent may refer to radiation and waves such as electroemissions gamma radiation microwaves UV irradiation or X rays. Other chemotherapeutic agents may include natural or synthetic antibodies tyrosine kinase inhibitors enzymatic inhibitors growth factor inhibitors metastases inhibiting compounds or oncogenic protein inhibitors such as compounds that inhibit RAS protein kinase or DNA topoisomerase.

 SIRT1 inhibitor refers to one or more compound that inhibits SIRT1 activity. Such inhibition includes direct as well as indirect inhibition of SIRT1 activity. Exemplary SIRT1 inhibitors include but are not limited to one or more agent or compound which results in inhibition of SIRT1 function inhibition of expression of SIRT1 protein inhibition of transcription or translation of the SIRT1 gene or both. For example a SIRT1 inhibitor includes sirtinol as well as its derivatives and other small molecule compounds able to reduce or inhibit SIRT1 activity. SIrt1 inhibitors include sirtinol a sirtinol analogue or derivative splitomicin a splitomicin analogue napthol a napthol derivative an indole an indole derivative siRNA shRNA antisense RNA or any combination thereof.

 Duration refers to the amount of time a desired gene is expressed and may be measured for example in months weeks days hours minutes and or seconds.

 Inhibit with regards to an activity means to suppress the activity either by decreasing the level or rate of the activity blocking or preventing the activity entirely or preventing an increase in the activity under conditions in which the activity would normally be increased.

 In combination with refers to two or more substances being administered simultaneously or in series close enough in time to bring about a therapeutically effective result.

 Leukemic disorder refers to a cancerous disorder of blood forming tissues e.g. spleen bone marrow lymphatics liver characterized by excessive leukocyte production. The term encompasses myeloid leukemias such as for example acute myeloid leukemia AML chronic myeloid leukemia CML and various subtypes thereof and lymphocytic leukemias such as for example acute lymphocytic leukemia ALL chronic lymphocytic leukemia CLL hairy cell leukemia HCL and various subtypes thereof.

 Pharmaceutically acceptable carrier refers to a pharmaceutically acceptable material composition or vehicle that is involved in carrying or transporting a compound of interest from one tissue organ or portion of the body to another tissue organ or portion of the body. For example the carrier may be a liquid or solid filler diluent excipient solvent or encapsulating material or some combination thereof. Each component of the carrier must be pharmaceutically acceptable in that it must be compatible with the other ingredients of the formulation. It also must be suitable for contact with any tissue organ or portion of the body that it may encounter meaning that it must not carry a risk of toxicity irritation allergic response immunogenicity or any other complication that excessively outweighs its therapeutic benefits.

 Polynucleotide refers to any polyribonucleotide polydeoxyribonucleotide or hybrid polyribo polydeoxyribonucleotide including naturally occurring polynucleotides synthetic polynucleotides or any chemically enzymatically or metabolically modified forms of naturally occurring polynucleotides. The term encompasses both single and double stranded molecules including DNA DNA DNA RNA or RNA RNA duplexes as well as molecules that are a mixture of single and double stranded regions. Polynucleotide also refers to triple stranded molecules comprising DNA RNA or both DNA and RNA. Polynucleotides may contain any of the standard pyrimidine or purine bases i.e. adenine guanine cytosine thymine uracil as well as any modified or uncommon bases such as tritylated bases or inosine. In addition the backbone of a polynucleotide may be modified for stability or for other reasons. Polynucleotides also refers to relatively short polynucleotides often referred to as oligonucleotides and to peptide nucleic acids PNAs formed by conjugating bases to a peptide backbone.

 Prodrug as used herein refers to a derivative of a pharmaceutically or therapeutically active drug that is transformed into the active drug by an enzymatic or chemical process. Prodrugs may be developed to alter the metabolic stability or transport characteristics of a drug to mask side effects or toxicity of a drug or to improve or alter other characteristics of the drug. See for example Notari R. E. 1985. Theory and practice of prodrug kinetics. Methods Enzymol 112 309 323 Bodor N. 1981. Novel approaches in prodrug design. Drugs of the Future 6 165 182 Bundgaard H. 1985 Design of prodrugs bioreversible derivatives for various functional groups and chemical entities Chap. 1 in Design of Prodrugs H. Bundgaard Ed. Elsevier New York 1985.

 RNA interference RNAi refers to a post transcriptional gene silencing PGSR process whereby one or more exogenous small interfering RNA siRNA molecules are used to silence expression of a target gene.

 siRNAs short interfering RNAs are double stranded RNA molecules generally around 15 30 nucleotides in length that are complementary to the sequence of the mRNA molecule transcribed from a target gene. Following introduction into a cell the siRNA molecule associates with one or more cellular proteins to form a siRNA protein complex RISC which then binds to the mRNA transcript of the target gene. RISC binding results in degradation of the mRNA molecule thereby preventing translation.

 shRNAs small hairpin RNAs are short hairpin turned RNA sequences that may be used to inhibit or suppress gene expression.

 Subject refers to any animal including a human having a cell that may be treated by the methods or products discovered or tested by the methods of this disclosure.

 Route of administration may refer to any administration pathway known in the art including but not limited to aerosol enteral nasal ophthalmic oral parenteral rectal transdermal e.g. topical cream or ointment patch or vaginal. Transdermal administration may be accomplished using a topical cream or ointment or by means of a transdermal patch. Parenteral refers to a route of administration that is generally associated with injection including infraorbital infusion intraarterial intracapsular intracardiac intradermal intramuscular intraperitoneal intrapulmonary intraspinal intrasternal intrathecal intrauterine intravenous subarachnoid subcapsular subcutaneous transmucosal or transtracheal.

 Therapeutically effective amount therapeutically effective concentration or therapeutically effective dose is an amount of a compound that produces a desired therapeutic effect in a subject such as preventing or treating a target condition alleviating symptoms associated with the condition or producing a desired physiological effect. A population of cells may be contacted with a therapeutically effective amount of a compound to study its effect in vitro or to produce a desired therapeutic effect ex vivo or in vitro. The precise therapeutically effective amount is an amount of the composition that will yield the most effective results in terms of efficacy of treatment in a given subject. This amount will vary depending upon a variety of factors including but not limited to the characteristics of the therapeutic compound including activity pharmacokinetics pharmacodynamics and bioavailability the physiological condition of the subject including age sex disease type and stage general physical condition responsiveness to a given dosage and type of medication the nature of the pharmaceutically acceptable carrier or carriers in the formulation and the route of administration. One skilled in the clinical and pharmacological arts will be able to determine a therapeutically effective amount through routine experimentation namely by monitoring a subject s response to administration of a compound and adjusting the dosage accordingly. For additional guidance see Remington The Science and Practice of Pharmacy 21Edition Univ. of Sciences in Philadelphia USIP Lippincott Williams Wilkins Philadelphia Pa. 2005.

 Treating or treatment of a condition may refer to preventing the condition slowing the onset or rate of development of the condition reducing the risk of developing the condition preventing or delaying the development of symptoms associated with the condition reducing or ending symptoms associated with the condition generating a complete or partial regression of the condition or some combination thereof.

According to embodiments of the disclosure and as discussed in detail in Example 1 below a cell culture model for chronic myelogenous leukemia CML acquired chemoresistance and related methods for generating such a model is provided. In some embodiments the cell culture model is generated by utilizing a na ve blast crisis CML cell line treated with a cytotoxic agent and allowed to relapse and acquire BCR ABL mutations. In some embodiments the na ve blast crisis CML cell line is a KCL 22 cell line.

Blast crisis CML cell line KCL 22 is insensitive to treatment with 1 M imatinib Deininger et al. 1997 . Treating KCL 22 cells with mutation inducing doses of imatinib induced apoptosis of the KCL 22 cells however relapse occurred within two weeks of the development of T315I BCR ABL mutation.

A mutation inducing dose mutation inducing concentration or mutation inducing amount is an amount of a compound that when exposed to a cell stimulates a nucleic acid mutation in the cell. In some embodiments a mutation inducing dose is about 2.5 M about 5 M about 10 M about 2.5 M or higher or between about 2.5 M and about 10 M. In some embodiments a mutation inducing dose is a concentration that may have a therapeutic effect when administered to a subject.

Following the initial apoptosis upon STI 571 treatment with dosages that are equivalent or nearly equivalent to therapeutically effective doses found in human plasma the KCL 22 cells developed the T315I mutation of the BCR ABL kinase domain at high frequency in two weeks. However when small molecule inhibitors of SIRT1 were used in combination with STI 571 the BCR ABL mutation was prevented and CML cells were eliminated without relapse. These results demonstrate T315I mutation can be rapidly induced by Imatinib treatment but is preventable.

Therefore the culture system developed provides among other things an in vitro system to study mechanisms of BCR ABL mutagenesis in the natural molecular and cellular contexts of CML. The methods and system provided herein can be used to develop strategies to treat and prevent formation of such a mutation. The discovery that SIRT1 inhibitors can block T315I mutation and completely eliminate CML cells when administered with imatinib also provides a novel treatment modality for CML patients. In addition the T315I mutant cells generated from this system also provide invaluable tools for studying mechanisms of resistance after mutation and strategies to eradicate these highly resistant cells.

In contrast to previous cell line models that are involved with gene amplification altered BCR ABL expression and or additional chromosomal rearrangements Mahon et al. 2000 Rosenhahn et al. 2007 Tipping et al. 2003 genetic mutations of BCR ABL primarily account for resistance in the model. Development of BCR ABL mutations in this model is highly reproducible and occurs in 2 weeks with a single dose of imatinib treatment. T315I mutation has been shown in CML cell line KBM5 after several months of treatment with gradually increasing concentrations of imatinib Ricci et al. 2002 but the mutation induction time in KCL 22 cells is much shorter. Although a relatively short period of time 3 6 weeks is also reported for developing BCR ABL mutations in primary CML cells cultured in vitro Koptyra et al. 2006 Jiang et al. 2007 the model described herein is simpler and more advantageous because for example it utilizes a commercially available cell line and routine culture conditions. Therefore this model is valuable for studying mechanisms of acquired resistance and developing strategies to prevent relapse.

Finding that BCR ABL mutations are dependent on BCR ABL gene expression is in line with the previous finding that BCR ABL can promote self mutagenesis Koptyra et al. 2006 which indicates that genome instability caused by BCR ABL transformation is perhaps the driving force for its mutations. Consistent with this notion we observe that clonal cells can derive mutations without pre existing mutations from the original patient and that these clonal cells can develop distinct BCR ABL mutations as well as resistance without mutations. Development of resistant BCR ABL mutations may resemble adaptive mutations promoting survival and growth in bacteria under stressful conditions which involves multiple DNA repair pathways Rosenberg 2001 Karpinets et al. 2006 . BCR ABL transformation alters regulation of multiple DNA repair pathways causing genome instability Melo Barnes 2007 . It would be interesting to determine how altered DNA repair may influence mutations with this model in the future.

Although BCR ABL cDNA has been widely used to study its mutagenesis comparison between the mutagenesis potential of randomly integrated BCR ABL cDNA versus the native BCR ABL locus has not been shown before. Based on the studies discussed above the BCR ABL translocation locus itself may play a role in promoting mutations as mutagenesis on the locus is far more efficient than on randomly integrated BCR ABL cDNA in the same cells. This locus dependent influence is validated by showing that HPRT mutation rate is relatively constant although the BCR ABL mutation rate is more dynamic. Further distinct clonal mutations resulted from the cell cloning process which affects the mutagenesis process. This indicates that there is a dynamic process for BCR ABL mutagenesis on its native locus. Such BCR ABL mutations may be influenced by the epigenome structure of the translocation locus and the environmental change or cloning process can result in subtle alteration of the local epigenome producing altered mutation hot spots.

The specific inhibition of BCR ABL kinase activity in KCL 22 cells is a complex process. Cellular phosphorylation in KCL 22 cells is not affected by overexpression of K1176R kinase inactive BCR ABL or by efficient delivery of BCR ABL antibody. Mutation phenotype in mutant BCR ABL overexpressing cells does not change with 2.5 M imatinib treatment but mutant BCR ABL blocks mutations at 5 M imatinib. The precise mechanism for this difference is not yet clear. Because cells harboring mature tetramer BCR ABL protein Smith et al. 2003 McWhirter et al. 1993 wholly from mutant monomers may not be able to grow in culture most cells for analysis should carry hybrid BCR ABL protein with wild type and mutant monomers. The e13a2 BCR ABL expressed in KCL 22 cells is known to have high tyrosine kinase activity Lucas et al. 2009 . The hybrid BCR ABL may retain normal phosphorylation ability on most substrates but may affect certain substrates regulating BCR ABL mutations and its effect would not be shown unless strong selection is applied. Alternatively the mutant BCR ABL may affect mutations independent of its kinase activity. K1176R BCR ABL retains certain functions such as the intact cellular localization and adhesion regulation Wertheim et al. 2002 and may continue recruiting proteins necessary for mutagenesis but in a less efficient way.

Although ROS production was previously found to promote BCR ABL mutations Koptyra et al. 2006 inhibition of ROS by high concentrations of anti oxidants was unable to prevent BCR ABL mutations and relapse as discussed above. However these two studies use quite different cell culture systems. KCL 22 cells have a basal level of ROS compared with high levels of ROS in blast crisis CML cells previously studied although low levels of ROS are also noticed in a subpopulation of CML cells Koptyra et al. 2006 . In addition primary CML cell culture uses growth factor supplemented medium Koptyra et al. 2006 Jiang et al. 2007 whereas KCL 22 cells are growth factor independent. Given that ABL kinase domain mutations can be detected even in normal human progenitor cells cultured in the growth factor medium Jiang et al. 2007 growth factor culture may affect ROS production and mutations.

According to some embodiments the present disclosure also includes novel mechanisms for acquired resistance through active induction of genetic mutations of oncogenic tyrosine kinases in human cancer. As discussed above HIC1 suppresses tumorigenesis is through regulation of stress and DNA damage responsive gene SIRT. SIRT1 is a key gene for controlling active induction of genetic mutations on tyrosine kinases through altering DNA damage pathways. HIC1 directly regulates SIRT1 transcription as shown in . Inhibitors of SIRT1 will inhibit acquired resistance through prevention of genetic mutations and thus are especially useful in the clinical setting where a combination of cancer therapeutics is applicable or utilized. The present disclosure however is not limited to CML. Rather as is understood by a person of ordinary skill in the art the mechanisms of acquired resistance and functions of SIRT1 and its inhibitors described herein can also be applied to treatment of acquired resistance in other cancers. In addition the mechanisms of acquired resistance and functions of other genes and their inhibitors can be applied to the treatment of acquired resistance in cancer.

Clonal cells derived from parental KCL 22 cells also predominantly develop resistance to imatinib by BCR ABL mutations including the T315I E255K and Y253H mutations. The mutations exhibited by the clonal cells encompass the most frequently observed mutations in CML patients Shah et al. 2002 Soverini et al. 2006 . Furthermore the rapid relapse is similar to that seen in blast crisis patients. The ability to form BCR ABL mutations in clonal cells indicates that pre existing rare mutant cells from the original patient are not required for development of resistance. The BCR ABL mutation frequency varies among clones while HPRT hypoxanthine phosphoribosyl transferase mutation frequency remains relatively constant. Unlike the broad mutation spectrum of HPRT observed in parental or clonal cells BCR ABL mutations may be limited to one mutation in a parental cell or a clone. Additionally while the spontaneous mutation frequency of HPRT remains relatively constant throughout cell passages frequency of BCR ABL mutation declines in later passages. See for example . The data provided herein indicate that BCR ABL mutations upon imatinib treatment are not always derived from random DNA replication error during clonal expansion or cell propagation but are actively induced by imatinib treatment. The disclosure includes a novel model system for testing and identifying therapeutic modalities and compounds for treating CML. The disclosure includes systems methods and cell lines based on the discovery that the targeting of a tyrosine kinase as an anti cancer treatment may in itself be mutagenic and thereby induce DNA mutations thereby directly or indirectly contributing to clinical acquired chemoresistance.

SIRT1 activation is essential for efficient BCR ABL oncogenic transformation and development of a CML like myeloproliferative disorder. Further SIRT1 activation by BCR ABL is more selective to hematopoietic stem progenitor cells which produce massive expansion of granulocyte macrophage lineage and SIRT1 knockout inhibits such biased differentiation and myeloproliferative disorder. This is consistent with the role of BCR ABL in promoting proliferation of myeloid progenitor cells Jamieson et al. 2004 . Although activated by BCR ABL inhibition of SIRT1 overcomes CML cell resistance to imatinib and has a synergistic effect with BCR ABL inhibition to induce apoptosis indicating that SIRT1 regulated pathways including Ku70 may confer a survival advantage for CML progenitor cells in addition to those classic survival pathways . SIRT1 is also an important intermediate factor for BCR ABL functions in CML to deacetylate and activate Ku70 thereby promoting CML genome instability through increasing infidelity DNA repair . It was also found that high levels SIRT1 promote de novo genetic mutations in prostate cancer cells upon DNA damage. Because SIRT1 is overexpressed in a variety of cancers SIRT1 likely plays an important role in tumorigenesis to promote cancer genome instability. This in turn facilitates cancer cells to rapidly evolve and develop mutations for acquired resistance.

The studies described herein improve the understanding of SIRT1 and its roles in tumorigenesis. Further the studies provide a strong rationale for developing inhibitors of SIRT1 activity or expression to overcome CML resistance to BCR ABL inhibitors. Although the current application is directed to developing cancer treatments directed to SIRT1 inhibitors other cancer treatments may be developed as well. SIRT1 activation is a novel mechanism for CML stem progenitor cell proliferation and survival and to promote cancer genome instability resulting mutations for acquired resistance. Accordingly SIRT1 targeting is a valuable cancer treatment pathway.

The novel tissue culture method provided herein closely simulates in vivo CML relapse on imatinib treatment using KCL 22 cells. By direct exposure of cells to Imatinib with concentrations pertinent to those in patient plasma the T315I mutation of BCR ABL can be induced rapidly with high frequency. The resultant T315I mutant cells differ from parental KCL 22 cells in size morphology and cell cycle and they are highly resistant to various treatments. SF medium or supplying insulin in serum medium will also provide CML cells resistance to Imatinib treatment. Finally the combination of Imatinib with SIRT1 inhibitors will prevent CML relapse and abolish growth factor induced resistance. Thus the methods outlined herein provide a platform for studying strategies for preventing formation of drug resistance and screening for small molecule inhibitors for blocking CML relapse.

The present discoveries may be used for methods for treating or preventing cancer cell growth treating or preventing resistance of a cancer cell to chemotherapy or treating or preventing the relapse growth of one or more cancer cell. Using the culture model described above key features of clinical resistance such as rapid relapse through BCR ABL mutations after imatinib treatment can be replicated. This model may serve as a system for designing testing screening and identifying new therapeutic strategies for treating CML. For example the resistance model can be used to screen modulators of enzymatic activity or small molecule inhibitors such as those which inhibit SIRT1 chemotherapeutics compounds or anti cancer modalities having therapeutic efficacy or those able to inhibit BCR ABL mutation based resistance. The resistance model may also be used for further studies of resistance mechanisms which will allow for the design of new therapeutic strategies such as use of small molecule inhibitors to prevent the induction of those mutations that accompany a cancer drug treatment.

According to one embodiment a method for screening for a candidate SIRT1 inhibitor for reducing chemoresistance or relapse in a cancer cell culture can be broadly applied and has the following steps. First a base level of chemoresistance or relapse is established in a cancer cell culture such as a cancer cell line after administration of one or more chemotherapeutic agents by treating the cancer cell culture with the chemotherapeutic agent under conditions which induce chemoresistance or relapse in the cell culture. Then the candidate SIRT1 inhibitor is administered to previously untreated cells of the cell culture before during and or after administering the one or more chemotherapeutic agents. Finally the level of chemoresistant or relapsed cells after treatment with the candidate SIRT1 inhibitor and the chemotherapeutic drug is measured. A reduction or absence of chemoresistant or relapsed cells as compared to base level of chemoresistance or relapse established at the outset of the model indicates that the compound is a SIRT1 inhibitor.

According to another embodiment a SIRT1 inhibitor compound for preventing chemoresistance or treating CML and a method for developing such a SIRT1 inhibitor for treating cancer is provided. In some embodiments such a method may include the following steps. First a pharmacophore model of SIRT1 inhibitors of a SIRT1 substrate binding pocket is generated. Then one or more SIRT1 inhibitor test compounds are identified. Next the one or more SIRT1 inhibitor test compounds are administered to a CML acquired resistant culture model such as the model described herein. Next an effective test dosage for the one or more SIRT1 inhibitor test compounds that block CML cell relapse is determined and compared to an effective sirtinol dosage. An effective sirtinol dosage is an amount of sirtinol that is able to prevent in whole or in part chemoresistance related to BCR ABL mutation relapse after treatment with a tyrosine kinase inhibitor or elicits another desirable effect in a CML cell including but not limited to inhibition of cell growth or induce cell death. Finally one or more of the SIRT1 inhibitor test compounds is selected as a SIRT1 inhibitor lead compound when the effective test dosage is lower than the effective sirtinol dosage.

As described below a computational pharmacophore model of SIRT1 inhibitors for blocking substrate binding is provided. In one embodiment 3D pharmacophore search from NCI chemical library of 250 253 compounds was performed followed by two rounds of the cell based screening. As described below a group of one or more lead compounds that inhibited CML acquired resistance at 1 5 M and blocked SIRT1 deacetylase activity was identified. According to one embodiment the one or more lead compounds are water soluble and have better inhibitory effect than other known SIRT1 inhibitors. In some embodiments the lead compounds have two distinct SIRT1 binding moieties one to block substrate binding and the other to block NAD binding. A computational docking study confirmed that these compounds can be docked into both substrate and NAD binding pockets. Blocking both substrate binding and NAD binding pocket individually has synergistic effect on SIRT1 inhibition. In one embodiment a nitrogen bond linker structure was identified that enhances specific hydrophilic interaction between the lead compounds and SIRT1 and also increases the water solubility of the lead compounds.

The cancer or cancer cell is usually a hematologic disorder or a solid tumor. Commonly the hematologic disorder is chronic myelogenous leukemia CML . An exemplary cancer cell associated with a solid tumor is non small cell lung carcinoma NSCLC . A therapeutically effective amount of a SIRT1 modulator or a combination of multiple SIRT1 modulators is administered to a subject in need thereof. Frequently a therapeutically effective amount of a modulator such as a SIRT1 inhibitor is administered. A SIRT1 inhibitor includes one or more inhibitors of sirtuins class III histone protein deacetylases such as sirtinol or its analogue which is administered in combination with the chemotherapeutic agent. Generally the SIRT1 modulator is administered at or about the same time as the chemotherapeutic agent but can also if desired be administered prior to or subsequent to the administration of the chemotherapeutic agent. The SIRT1 modulator may either be an agonist or antagonist of SIRT1 and may be any molecule compound or agent that acts to modulate SIRT1. Such modulators include but are not limited to derivatives analogues small molecules decoy molecules drugs or prodrugs polynucleotides particularly antisense molecules and RNA interference using siRNA shRNA polypeptides antibodies including chimeric antibodies or any other substance that acts on SIRT1 in an intended manner. If the SIRT1 modulator is a SIRT1 antagonist acting to inhibit SIRT1 the inhibitor is preferably a napthol compound for example sirtinol or splitomicin EX 527 an indole or its derivative siRNA shRNA or any combination thereof.

Drug resistance or cancer cell relapse may be caused by administration of a chemotherapeutic agent such as imatinib gefitinib nilotinib dasatinib or another cancer treating drug. Imatinib is a tyrosine kinase inhibitor. It interferes with BCR ABL protein function. Gefinitib inhibits EGFR kinase. BCR ABL drug resistance may be caused by administration of a chemotherapeutic agent such as imatinib nilotinib dasatinib VX 680 a dual Aurora kinase and BCR ABL inhibitor or another cancer treating drug. Alternatively resistance may be caused by another drug or compound administered to the patient environmental factors or may be a naturally occurring resistance. Relapse in chronic myeloid leukemia patients treated with imatinib such as STI 571 is an example of one form of clinical chemoresistance associated with a point mutation or amplification of the BCR ABL gene. Thus the SIRT1 modulator may also be administered in conjunction with a tyrosine kinase inhibitor.

Both the SIRT1 modulator and the chemotherapeutic agent are to be applied in therapeutically effective amounts and for any duration necessary to treat the cancer or prevent relapse of the cancer. A clinician can gauge the dosages for each required by the subject using known methods of optimizing drug performance and delivery by taking into account clinical data regarding tolerances age gender severity of the disease health of the subject and the like. Preferably the SIRT1 modulator is an inhibitor that is administered to the subject in a pharmaceutically acceptable carrier. The route of administration may be any route effective to carry the therapeutic SIRT1 modulator to the cancer cells in the subject including any of the routes discussed above. In addition the SIRT1 modulator may also be used treat or prevent insulin and transferrin induced resistance as well as EGFR epidermal growth factor receptor associated resistance.

A method of preventing chemoresistance in CML cells comprising administering a therapeutic drug or combination of therapeutic drugs that prevent formation of a T315I mutation and other BCR ABL mutations is also provided. These BCR ABL mutations in cancer cells particularly CML cells are responsible for anti apoptotic activity and as such prevent the cancerous cells from responding to traditional chemotherapy radiotherapy or other cytotoxic agents. Preferably the therapeutic drug is comprises a SIRT1 antagonist such as sirtinol a sirtinol analogue splitomicin an indole siRNA shRNA Aurora kinase inhibitor or any combination thereof.

Another embodiment improves the efficacy of a cytotoxic agent directed to CML cells in a subject by administering to the subject at least one cytotoxic agent directed to CML cells and administering to the subject a SIRT1 inhibitor in a therapeutically effective amount so that the SIRT1 inhibitor enhances the efficacy of the cytotoxic agent relative to the effect of the cytotoxic agent in the absence of the SIRT1 inhibitor. The cytotoxic agent may be a BCR ABL inhibitor such as STI 571 imatinib nilotinib or dasatinib or any combination thereof and the SIRT1 inhibitor is siRNA a napthol compound sirtinol splitomicin an indole or any combination thereof. This method also treats or prevents insulin and transferrin induced resistance. The methods of the present disclosure may be performed on any animal with cancer or CML but are preferably for humans.

Having described the invention with reference to the embodiments and illustrative examples those in the art may appreciate modifications to the invention as described and illustrated that do not depart from the spirit and scope of the invention as disclosed in the specification. The Examples are set forth to aid in understanding the invention but are not intended to and should not be construed to limit its scope in any way. The examples do not include detailed descriptions of conventional methods. Such methods are well known to those of ordinary skill in the art and are described in numerous publications. Further all references cited above and in the examples below are hereby incorporated by reference in their entirety as if fully set forth herein.

CML cell lines KCL 22 and K562 were purchased from German Collection of Cell Cultures Braunschweig Germany and grown in RPMI 1640 medium with 10 fetal bovine serum Hyclone . STI 571 was provided by Novartis Basel Switzerland. 6 thioguanine 6 TG was purchased from Sigma and 2.5 g ml final concentration was used for selection. Sirtinol splitomicin nicotinamide trichostatin A 5 aza 2 deoxycytidine and HAT were also purchased from Sigma. For serum free culture basic supplements ITS 1 1884 EGF and HDL were purchased from Sigma and insulin from Roche.

5 10KCL 22 or K562 cells in 1 ml medium per well in 24 well plates and treated with various combinations of drugs. Cells were maintained in these cultures without changing medium. Aliquots of cells at specified time points were removed and cell numbers were counted on a hematocytometer. Cell viability was assessed by trypan blue exclusion whenever necessary. Typically after three to five weeks in culture when their medium volume significantly decreased fresh drug free medium was supplied to the cells.

Total RNA and DNA are isolated using standard protocols. For sequencing the EGFR kinase domain the EGFR kinase domain is amplified by RT PCR of total RNA or by PCR of genomic DNA with a high fidelity DNA polymerase Strategene using primers previously described Pao et al. 2004 . PCR products are cloned into the pCR2.1 vector using TOPO TA Cloning kit Invitrogen . At least ten clones for each treatment are sequenced. For mutant enriched PCR sequencing of T790M mutation the EGFR exon 20 is amplified using primers described Inukai et al. 2006 . After digestion of the first round PCR product with BstUI NEB the second round nested PCR is performed and the PCR products are sequenced directly.

A standard two layer soft agar culture was performed with a bottom layer of 0.7 agarose and top layer of 0.35 agarose. Five hundred cells per well in 6 well plates were seeded with warm top agar and were incubated for three weeks. Plates were then stained with 0.005 Crystal Violet for 1 hour and colonies were scored with aid of microscope. For resistance assay in soft agar one million cells were added per well and imatinib STI 571 or 6 TG added to both top and bottom agar to their final concentrations. To clone or recover soft agar colonies for further analysis individual colonies were plucked and expanded in liquid culture.

Cell cycle was analyzed by fixing cells with 70 ethanol at 20 C. overnight. After washing cells were resuspended in phosphate buffered saline containing 1 mg ml RNase A incubated for 30 min at 37 C. and then stained with propidine iodine 50 g ml for 30 min at room temperature before flow cytometry analysis. Cell proliferation was analyzed using a XTT Cell Proliferation kit Roche Applied Science and apoptosis was analyzed with annexin V kit BD Pharmingen as per the manufacturer s instruction.

For spectral karyotyping analysis 10 mitotic cells of each sample were analyzed 5 by GTG band analysis and 5 by 24 color karyotyping. Triple color FISH was performed with LSI ABL1 AASS 9q34.1 BCR 22q11.2 probes Vysis Inc. Huntly et al. 2003 and 200 cells of each sample were evaluated. These assays were done by City of Hope Cytogenetics Core Laboratory.

The ABL kinase domain ABL k was amplified by RT PCR of total RNA or by PCR of genomic DNA with a high fidelity DNA polymerase Strategene using the primers for the 579 bp kinase domain using cDNA templates shown below as previously described Gorre et al. 2001 .

To confirm mutations the ABL kinase domain was also amplified by PCR using genomic DNA as templates with the intron primers below which span ABL exon 6 for T315I mutation T315I and intron primers below which span ABL exon 5 for E255K and Y253H mutations E255K Y253H .

PCR products were cloned into the pCR2.1 vector using TOPO TA Cloning kit Invitrogen . At least ten clones for each treatment were sequenced by Sequencing Facility of Beckman Research Institute. For analysis of genomic DNA mutations PCR products were directly sequenced without subcloning. For HPRT sequencing the codon sequence was amplified by RT PCR using primers previously described Osterholm et al. 1995 . PCR products were purified with a PCR product clean up kit Qiagen and sequenced directly. Reverse transcription PCR for sequencing BCR ABL oligomerization and Src homology 3 2 SH3 2 domains was also performed. For sequencing the BCR ABL SH3 2 domain primers described previously Ray et al. 2007 were used. For the oligomerization domain the following primers BCR ABL oligo were used.

For sequencing HPRT the codon sequence was amplified by reverse transcription PCR using the primers below 

Cells were counted and diluted to 5 cells per ml and seeded onto 96 well plate with 100 l or 0.5 cell per well. Individual cell seeding was then confirmed by microscopy and single cell clones were grown and expanded for further analysis.

BCR ABL expression and phosphorylation were directly analyzed by Western blot using anti c ABL monoclonal antibody BD Pharmingen 554148 and anti phospho tyrosine antibody Upstate Biotechnology Inc. 05 321 . To validate BCR ABL phosphorylation BCR ABL from 500 g of total cell lysate of KCL 22 cells was isolated with 2 g of anti c ABL and 100 l of 50 slurry of protein A agarose beads Upstate Biotechnology Inc. . Alternatively protein G agarose beads or a mixture or protein A and G agarose beads may be used for isolating BCR ABL.

For protein analysis by Western blot the following antibodies are used rabbit monoclonal anti SIRT1 1 5000 Epitomics and anti GAPDH 1 5000 Trevigen . Apoptosis analysis was performed using a TMR red In situ Cell Death Detection Kit Roche as per manufacturer s protocol. Normal human lung lysate are purchased from ProSci.

ROS was analyzed using redox sensitive fluorochrome 2 .7 dichlorofluorescein diacetate Sigma as described previously Koptyra et al. 2006 . When coupled with the apoptosis analysis cells were labeled with annexin V first and then 2 .7 dichlorofluorescein diacetate. An H2AX phosphorylation assay kit Millipore 17 344 was used to analyze DNA damage by flow cytometry as per the manufacturer s suggestions.

Total cellular RNA with Trizol Invitrogen using a standard protocol. First strand DNA was synthesized and HIC1 expression by quantitative real time RT PCR using a kit with SYBR Green label Invitrogen as per the manufacture s instruction on a BioRad machine OPTICON. The following are the HIC1 primers spanning introns 5 GGACGGACCAGCAGGACA 3 exon 1a and 5 GCGCTGGTTGTTGAGCTG 3 Exon 2 . ref NT010718.15 Hs1710875 1561637 1565694 chromosome 17 genomic contig reference assembly 

SIRT1 expression was analyzed by Western blot using 1 5000 diluted rabbit monoclonal SIRT1 antibody Epitomics . Controls used were GAPDH or actin as loading controls with rabbit anti GAPDH Trevigen or anti actin Sigma at 1 5000 dilution. GAPDH ref NT009759.15 Hs129916 6497433 6502281 chromosome 12 genomic contig reference assembly primers used are as follows 

For production of lentiviral vectors four million 293T cells are co transfected with 15 g of the vector 15 g of gag pol 5 g of VSV G and 5 g of Rev plasmids by the method of calcium phosphate co precipitation Kowolik C. M. P. Yam Y. Yu and J. K. Yee. 2003. HIV vector production mediated by Rev protein transduction. 8 324 331 . The supernatant is collected at 24 hours and 36 hours after transfection. The supernatants are pooled and passed through a 0.45 um filter concentrated by ultracentrifugation. To determine vector titer 1 10293T cells are seeded in a six well plate in the presence of 4 mg ml polybrene and cells are transduced for 5 hours and analyzed by FACS for GFP expression within 24 hours.

Oligonucleotides for ABL shRNA GTTGGTTCATCATCATTCA SEQ ID NO 29 were synthesized and cloned into the pSicoR vector Ventura et al. 2004 that contains a selection cassette for puromysin by a standard protocol. A scrambled shRNA was subcloned into the vector as a mock control The VSV G G protein of vesicular stomatitis virus pseudotyped lentiviral vectors were produced using a four plasmid transfection system as described previously Kowolik et al. 2003 . High titer lentiviral stocks typically 1 to 3 10infectious units ml were used for the studies with multiplicity of infection MOI around 5. For SIRT1 knockdown in transformed CD34 cells SIRT1 shRNA sequences were cloned into the HIV7 SF RFP lentiviral vector. MIG210 or MIGR1 GFP and shRNA RFP double transduced cells were sorted by flow cytometry for CD34GFPRFP for in vitro apoptosis and proliferation study described above. For BCR ABL knockdown one millions cells were infected overnight with recombinant lentivirus by multiplicity of infection of 3 each in the presence of 8 g ml Polybrene. Under this condition the transduction rate in these cells was typically about 99 .

The assay was performed as described Xiao et al. 2003 . KCL 22 cells were pre selected for four days in HAT medium to remove pre existing HPRT mutations. The efficiency of HAT selection was confirmed by plating these cells on soft agar with 2.5 g ml 6 thioguanine which produced zero colony. HAT selected cells were then treated with 0.5 M CPT for 1 hour and used for soft agar clonogenic assay with 6 thioguanine. The rest of HAT selected cells were cultured in medium without selection. Soft agar colonies were scored after three weeks as described Yuan et al. 2008 .

Amphotropic retroviral vectors for wild type p210 BCR ABL K1176R p210 mutant and the empty vector MIG R1 were produced using Phoenix Ampho packaging cells ATCC and the transduction was carried out as previously described Ramaraj et al. 2004 Pear et al. 1998 . Briefly KCL 22 cells were transduced with the aforementioned vectors at a multiplicity of infection of around 6. Cells were spin infected by centrifugation at 1000 g for 90 min and then returned to the incubator and cultured overnight. After removing viruses cells were expanded in culture for 5 days and green fluorescent protein GFP expressing cells were isolated by fluorescent activated cell sorting FACS . For second round transduction FACS enriched cells were re infected with the above vectors using the same conditions. Genomic DNA sequencing analysis of imatinib resistant soft agar colonies was performed using the intron primers for the T315I mutation on the endogenous BCR ABL locus and exon primers for 579 bp transduced BCR ABL cDNA as described above. To increase PCR efficiency on the transduced BCR ABL cDNA another reverse primer 5 TAGTCCAGGAGGTTCCCGTAG SEQ ID NO 21 was used to pair with the same exon forward primer used for 579 bp cDNA which produced a 321 bp PCR product.

Quantitative real time PCR was performed with SuperScript III platinum two step qRT PCR kit with SYBR Green Invitrogen as per the manufacturer s instruction on the BioRad machine OPTICON. For BCR ABL RNA analysis we extracted total cellular RNA with Trizol Invitrogen using a standard protocol. The primer pairs for ABL genomic DNA analysis were 5 GCCTGTCTCTGTGGGCTGAAG 3 SEQ ID NO 6 and 5 CAAGGCGTCTGCTGGCATTA 3 SEQ ID NO 13 primers for BCR ABL RNA analysis were 5 CGTGCAGAGTGGAGGGAGAAC 3 SEQ ID NO 30 and 5 GCATCTGACTTTGAGCCTCAGG 3 SEQ ID NO 31 . PCR cycling conditions were 94 C. 5 min followed by 94 C. 30 s 60 C. 30 s 55 C. 30 s for 40 cycles.

A fluorescent ASO PCR was carried out as previously describedwith modifications. Purified T315I positive RNA from T315I BCR ABL cells was diluted with T315I negative RNA to 1 10 1 100 and 1 1000 and all RNAs were reverse transcribed using random hexamers and SuperScript III kit Invitrogen . PCR was carried out with 1 l of RT reaction each with optimal conditions of 45 cycles and 60 C. annealing temperature. The primers for ASO PCR are as follows forward outer primer 5 CGTGAAGACCTTGAAGGAGGACACCATG 3 SEQ ID NO 32 reverse outer primer 5 FAM TTCTCCAGGTACTCCATGGCTGACGAGA 3 SEQ ID NO 33 reverse T315I mutation primer 5 FAM TCCAGGAGGTTCCCGTAGGTCATGAACTAAA 3 SEQ ID NO 34 forward primer 5 CCCGGGAGCCCCCGTTCTATATCATAAC 3 SEQ ID NO 35 . PCR products were diluted and run on ABI 3100 Genetic Analyzer Applied BioSystems . Care was taken to avoid contamination by performing PCR reactions in a separate room from that for nucleic acid extraction and PCR product electrophoresis and personal return to PCR reaction room in the same day is prohibited after he handles nuclei acid extraction and electrophoresis.

Bi PAP assay was performed as previously describedwith the primer pairs T315I F 5 GGAGCCCCCGTTCTATATCATCAddT 3 SEQ ID NO 36 and T315I R 5 AGGTTCCCGTAGGT CATGAACTCAddA 3 SEQ ID NO 37 and PAPase all purchased from BioVision USA . The PCR cycling conditions were 1 cycle of 95 C. 60 s 60 C. 30 s 62 C. 30 s 68 C. 45 s and 72 C. 45 s followed by 40 or 50 cycles of 95 C. 30 s 60 C. 30 s 62 C. 30 s 68 C. 40 s and 72 C. 5 min. Similar to ASO PCR Bi PAP reactions were performed in a separate room from that for nucleic acid extraction and PCR product analysis.

Treatment of KCL 22 Cells with 2.5 M STI 571 or Higher Leads to Relapse and Formation of KCL 22M Cells Having a T315I Mutation.

In contrast to the in vivo resistance observed in blast crisis CML patients treated with STI 571 CML cell lines derived from blast crisis patients are sensitive to STI 571 treatment. STI 571 at 1 M selectively kills CML cells in BCR ABL dependent manner with the exception of KCL 22 cells whereas 10 M STI 571 can result in non specific cell killing. In chronic phase CML patients STI 571 is given at 400 mg day that produces the average peak plasma concentration at 4.4 M and trough concentration at 2.0 M. For blast crisis patients STI 571 dosage is increased to 600 mg day. Therefore effects of imatinib concentrations at 1 2.5 5 and 10 M were examined on the survival of KCL 22 cells during prolonged culture.

5 10KCL 22 or K562 cells were seeded in 1 ml medium per well in 24 well plates and treated with various combinations of drugs. Cells were maintained in these cultures without changing medium. Aliquots of cells were taken out at specified time points and cell numbers counted using a hematocytometer. Cell viability was accessed by trypan blue exclusion where necessary. Typically after three to five weeks in culture when medium volume significantly decreased fresh drug free medium was supplied to the cells to restore the wells to the original volume for prolonged culture.

It was found that KCL 22 cells were insensitive to 1 M STI 571 treatment as they continued to grow but at a lower rate than in the absence of the drug. STI 571 at 2.5 M and above effectively suppressed cell growth and induced partial cell death over time . Small clusters of cells formed after about 10 days in treatment groups with 2.5 M and above of STI 571 and these cells appeared visibly larger with frequent bizarre shapes . After two weeks they repopulated the culture indicating the relapse on the drug treatment. These emerging cells named KCL 22M grew equally well as KCL 22 cells and no longer responded to presence of imatinib in the medium . They formed fewer and smaller soft agar colonies . The abnormal size and shape of KCL 22M cells were confirmed by flow cytometric analysis showing increase on both forward scatter for size and side scatter for complexity . KCL 22M cells also exhibited different cell cycle status from KCL 22 cells by increasing S G2 population .

Molecular characterization of KCL 22M cells was also performed. The KCL 22 cells were resistant to imatinib treatment. By spectral karyotyping and FISH analyses it was found that KCL 22M cells maintained the same cytogenetic profile as KCL 22 cells i.e. 51 X del X p11.2p22.3 der 1 10 q10 p10 6 8 8 t 9 22 q34.1 q11.2 der 17 19 q10 q10 19 i 21 q10 der 22 t 9 22 and carried two Philadelphia chromosomes in all cells examined . This cytogenetic data was in line with previously reported karyotype for KCL 22 cells Kubonishi et al. 1983 Rosenhahn et al. 2007 . In contrast to another KCL 22 cell derived imatinib resistant cell line KCL 22R that had additional translocations Rosenhahn et al. 2007 no novel chromosomal rearrangements were shown in KCL 22M cells. Using real time PCR the ABL DNA content and BCR ABL RNA level in KCL 22M cells remained the same as in KCL 22 cells .

Imatinib effectively inhibited tyrosine phosphorylation by direct Western blot analysis of total cell lysate or immunoprecipitation of BCR ABL protein . Tyrosine phosphorylation was not altered by imatinib treatment in KCL 22M cells . Since genetic mutations are predominant mechanisms for in vivo resistance of STI 571 it was examined whether mutations occurred in the KCL 22M cells. Both cDNA and genomic DNA were sequenced for BCR ABL kinase domain using the strategies described by Gorre et al 2001 . See for example . In the 579 bp cDNA region of BCR ABL kinase domain covering the ATP binding pocket and activation loop a single mutation C to T nucleotide change that resulted in amino acid T315I mutation at ABL was found in KCL 22M but not parental KCL 22 cells . This is the same mutation as identified in patients. The mutant clones represented 40 for 2.5 M STI 571 or 30 for 5 M STI 571 treatment in all clones sequenced respectively. Given that KCL 22 cells have a normal copy of ABL gene which is also be amplified during RT PCR half of clones are expected to be wild type. Therefore the results show that at least 60 to 80 of KCL 22M clones may carry T315I mutation. Sequencing of genomic DNA further confirmed such a mutation. To search for mutations on other BCR ABL domains important for its kinase activity oligomerization and SH3 2 domains were sequenced Zhao et al. 2002 Smith et al. 2003 but no additional mutations were identified. Even when the SH3 2 sequencing fragment was extended through most of the 579 bp kinase domain only the T315I mutation was found. Because the T315I mutation is one of the most powerful mutations generated upon relapse in response to STI 571 treatment this model encompasses the BCR ABL mutagenesis observed in clinical relapse.

To determine whether the KCL 22M cells were a mixture of T315I mutants and non mutants a limiting dilution was performed to obtain individual mutant cells from KCL 22M relapsed on 2.5 and 5 M imatinib respectively. Ten clones for each BCR ABL mutation were sequenced using genomic templates. It was found that 20 clones carried the T315I mutation . Clonal resistant cells were separately derived by plating KCL 22 cells on soft agar with 2.5 or 5 M imatinib and cell colonies were grown. After three weeks resistance cell colonies were randomly picked and expanded for DNA sequence analysis. Twenty clones 10 from each concentration of imatinib carried the T315I mutation . KCL 22 cells at various passages relapsed on imatinib and all recurrent cells had the T315I mutation. KCL 22 cells were also found to relapse in similar time courses with repeated doses as with a single dose . These results indicate that KCL 22 cells develop acquired resistance predominantly through the T315I BCR ABL mutation.

The T315I mutation rate was measured by plating KCL 22 cells on soft agar with imatinib. A standard two layer soft agar culture was performed with bottom layer of 0.7 agarose and top layer of 0.35 agarose. One million cells per well in 6 well plates on warm top agar with imatinib in both top and bottom agar layers to their final concentrations were incubated for three weeks. Plates were then stained with 0.005 Crystal Violet for 1 hour and colonies were scored with aid of microscope. The mutation rate with 5 and 10 M imatinib was around 3 10 and 5.7 10with 2.5 M imatinib . The mutation rate in liquid culture was also measured by serial two fold dilutions of KCL 22 cells followed by treatment with 5 M imatinib. The lowest number of cells that consistently relapsed on imatinib was determined. The medium T315I mutation rate was about 1 12 500 or 8 10. This is in general agreement with the rate observed in soft agar analysis given that the plating efficiency for KCL 22 cells in soft agar was about 25 . These results indicate that imatinib treatment of KCL 22 cells results in acquired resistance through rare T315I mutation. This model encompasses a key BCR ABL mutagenesis process that occurs in clinical relapse. This is the first CML resistance model through BCR ABL mutations that can be rapidly reproduced within two weeks after exposing cells to in vivo effective concentrations of imatinib.

To determine whether there is a pre existing T315I mutation in KCL 22 two sensitive methods were used to detect T315I mutations in cDNA by allele specific oligonucleotide PCR or genomic DNA by bidirectional pyrophosphorolysis activated polymerization allele specific amplification from untreated KCL 22 cells 1 a modified fluorescent allele specific oligonucleotide PCR assay Willis et al. 2005 and 2 a bidirectional pyrophosphorolysis activated polymerization allele specific amplification assay Shi et al. 2007 . Both methods reliably detected at least 1 mutant allele but no T315I mutation was found in KCL 22 cells . The sensitivity of these methods could not be further increased without sacrificing specificity and therefore such methods should not be used to completely rule out the existence of rare mutant cells in KCL 22 cells. However after continuous culture of KCL 22 cells in a refractory dose of imatinib 1 M for 2 weeks T315I mutation became readily detectable by conventional DNA sequencing . Because T315I is the most resistant mutation for BCR ABL inhibitors Weisberg et al. 2005 von Bubnoff et al. 2006 Shah et al. 2004 this model encompasses one key BCR ABL mutagenesis process in clinical relapse. However directly exposing cells to mutation inducing concentrations of imatinib did not work for CML cell lines K562 and KU812 that underwent rapid apoptosis as previously reported Deininger et al. 1997 . This is likely because the cells underwent apoptosis before BCR ABL mutations could be fully developed. This method may be applied to other CML cell lines. Compared with the standard approach that requires the cells to be exposed to multiple rounds of gradually increasing concentrations of the drug for a period of several months the culture model described herein offers a unique advantage in that BCR ABL mutations can be rapidly produced within 2 weeks after a single exposure of cells to concentrations of imatinib that are effective in vivo.

In vivo most BCR ABL mutations are found in relapsed CML patients although pre existing T315I mutation is detected in some CML patients before imatinib treatment Shah et al. 2002 Soverini et al. 2006 Roche Lestienne et al. 2002 . Whether the pre existing BCR ABL mutant cells originating in the patient is a requirement for development of resistance was investigated. Individual KCL 22 cell clones were isolated by limiting dilution or soft agar plating without drug treatment. Eleven liquid culture clones L1 L11 and thirteen soft agar clones Ag1 Ag13 were expanded for analysis Table 1 below . Most of the clones failed to relapse but four relapsed in two weeks with high frequency at different concentrations of imatinib. These were clones L1 L7 Ag3 and Ag11 Table 1 and . After sequence analysis of relapsed cells it was determined that clone L1 relapsed on 2.5 and 5 M with E255K BCR ABL mutation clone L7 relapsed on all doses of the drug with Y253H BCR ABL mutation clone Ag3 relapsed without BCR ABL kinase domain mutations on 2.5 and 5 M imatinib but with T315I mutation on 10 M imatinib clone Ag 11 relapsed with T315I mutation on 2.5 M imatinib which is similar to parental cells Table 1 . In Table 1 no mutations have been detected in clones L1 L7 Ag3 and Ag11 before STI treatment. ND was not done. The asterisk indicates that there was relapse after 50 days and remaining sensitivity to 2.5 M STI 571.

These mutations conferred resistance of clonal cells to imatinib with T315I having the greatest protective effect against proliferation inhibition by high concentrations of the drug . Compared to parental cells more BCR ABL mutations emerged from these clones. The ability of these clones to develop resistance through BCR ABL kinase domain mutations indicates that a pre existing mutant cell originating from a patient is not required for resistance.

The effects of imatinib treatment on ROS and DNA damage was examined in the culture method system described above were examined. Following imatinib treatment the bulk ROS level was reduced in the nonapoptotic fraction of KCL 22 cells while ROS increased in the apoptotic fractions . Accordingly the bulk DNA damage analyzed by H2AX staining decreased in the nonapoptotic fraction of cells . In previous experiments 200 M of the anti oxidant vitamin E or N acetylcysteine is able to reduce ROS and BCR ABL mutations. However in the studies described herein even a 100 higher concentration of NAC 25 mM or a 40 higher vitamin E 8 mM failed to prevent imatinib induced BCR ABL mutations and relapse . These data reveal that BCR ABL mutations in KCL 22 cells cannot be explained simply due to a change in ROS.

It was then determined whether BCR ABL gene expression is required for its own mutations. KCL 22 cells express a e13a2 BCR ABL fusion transcript and the junction region sequence does not meet optimal shRNA design criteria Reynolds et al. 2004 . As previously reported Li et al. 2003 shRNA that targets this region produced poor gene knockdown. Therefore a shRNA that targeted the ABL sequence was used Zhelev et al. 2004 to knock down BCR ABL. It was found that BCR ABL knockdown rapidly inhibited proliferation and induced apoptosis of both KCL 22 and KCL 22M cells showing continued dependence of BCR ABL for proliferation and survival of BCR ABL mutant CML cells. A lower apoptosis rate was observed in KCL 22M cells indicating that cells acquire additional survival advantage during development of resistance. In contrast a BCR ABL negative leukemia cell line HL 60 proliferation was not inhibited nor was apoptosis induced when ABL was knocked down showing that effects in CML cells are BCR ABL specific.

Both KCL 22 and KCL 22M cells with BCR ABL knockdown re grew after 2 weeks. BCR ABL expression was partially restored in re grown knockdown KCL 22 cells but remained at a lower level than in mock knockdown cells . When treated with imatinib re grown BCR ABL knockdown KCL 22 cells failed to relapse in liquid culture or form imatinib resistant colonies on soft agar . Similarly BCR ABL knockdown immediately followed by imatinib treatment also blocked relapse in KCL 22 cells but failed to prevent KCL 22M cells from regrowing . Although the shRNA knocks down both ABL and BCR ABL loss of the DNA repair protein ABL would have been expected to further increase genetic mutation which would contradict the above findings. Therefore these unexpected results show that formation of BCR ABL mutations is dependent on BCR ABL gene expression and that combination of BCR ABL knockdown and imatinib can block acquired resistance of KCL 22 cells on the drug.

To determine roles of BCR ABL kinase activity for its mutation wild type p210 BCR ABL cDNA and K1176R p210 BCR ABL mutant cDNA were overexpressed in KCL 22 cells. The mutant BCR ABL has an abolished kinase activity Ramaraj et al. 2004 Wertheim et al. 2002 or controls empty MIG retroviral vectors Pear et al. 1998 . The transduced cells were isolated by FACS for GFP expression. Overexpression of wild type BCR ABL increased cellular phosphorylation but overexpression of the mutant BCR ABL did not reduce cellular phosphorylation indicating that K1176R BCR ABL is not a dominant negative mutant in these cells. Neither wild type nor mutant BCR ABL expression changed cell growth or soft agar cloning efficiency .

When treated with 2.5 M imatinib wild type BCR ABL overexpressing cells developed significantly higher numbers of imatinib resistant colonies than empty vector or mutant BCR ABL transduced cells and the majority of resistant colonies 8 out of 10 from wild type BCR ABL overexpressing cells did not harbor the T315I mutation of the endogenous BCR ABL . This indicated that the transduced BCR ABL cDNA may function as BCR ABL gene amplification for resistance. In contrast all resistant colonies in the empty vector or mutant BCR ABL transduced cells harbored T315I mutation of the endogenous BCR ABL .

When treated with 5 M imatinib resistance provided by wild type BCR ABL cDNA was largely diminished and the majority of resistant colonies 7 out of 10 from wild type BCR ABL overexpressing cells harbored T315I mutation of the endogenous BCR ABL . Unexpectedly under these conditions mutant BCR ABL completely blocked imatinib resistant colony formation . Similar results were obtained when a second round of transduction was carried out using these vectors on the first round of GFP enriched cells aiming to further increase cDNA expression. These results indicate a role of BCR ABL kinase activity for its mutation when cells are under treatment with a high concentration of imatinib. However overexpression of mutant BCR ABL did not enhance reduction of total cellular or a BCR ABL substrate CRKL protein phosphorylation upon imatinib treatment . Therefore functionally intact BCR ABL may be important for generating mutations but kinase activity may not be absolutely required.

BCR ABL cDNA has been previously expressed in non CML cells to generate random BCR ABL mutations 5 7 10 15 and is presumed to have similar mutagenesis capability. Mutations on the transduced wild type BCR ABL cDNA were examined in KCL 22 cells. The transduced gene was distinguished from the endogenous BCR ABL by using exon primers that span multiple introns for genomic DNA PCR and sequencing. All 20 imatinib resistant clones from 2.5 and 5 M drug treatment had visible GFP expression indicating that they all carried functional transduced BCR ABL cDNA. Clones that did not harbor T315I mutation of the endogenous BCR ABL tended to have a higher yield of PCR products revealing a higher copy number of the transduced BCR ABL in these clones for resistance . Only one clone developed a BCR ABL kinase domain mutation T315I on the transduced BCR ABL among all 20 clones sequenced . Therefore although the transduced BCR ABL cDNA was capable of mutagenesis in KCL 22 cells the mutation efficiency was significantly lower than that of the endogenous BCR ABL 9 of 20 clones p 0.008 . This difference is not a result of experimental artifacts for the following reasons.

First wild type BCR ABL cDNA was transduced into KCL 22 cells using retroviral virions and would readily integrate into the genome of replicating KCL 22 cells. Nonintegrated viral DNA would have been lost during the lengthy procedure for expanding transduced cells before and after FACS sorting colony formation of imatinib resistant cells on soft agar followed by expanding clonal cells in liquid culture for assay. The stable integration of the cDNA was also evidenced by nearly homogeneous GFP expression of the expanded clonal cells in liquid culture.

Second mutations were analyzed within the clonal population of wild type BCR ABL transduced cells and both endogenous and transduced alleles for each clone were analyzed and compared. This is an internally controlled comparison and therefore mutation efficiency is not affected by external factors such as retroviral transduction efficiency.

Third although KCL 22 cells have two translocation chromosomes the lower mutation rate of transduced BCR ABL cDNA is unlikely a result of lower copy number of the transduced BCR ABL cDNA. The cDNA copy number may vary from clone to clone but at least one copy of the cDNA is expected for each of 10 clones clones 1 2 4 5 7 8 and 10 under 5 M and clones 1 5 and 7 under 2.5 M that did not efficiently amplify the longer cDNA template 579 bp whereas the other 10 clones would have higher copy numbers. However the single clone that developed BCR ABL cDNA mutation clone 5 under 2.5 M carried a low copy number of cDNA. Accordingly increasing cDNA copy number did not increase its mutation frequency. Together these results show that the native BCR ABL translocation locus in KCL 22 cells has inherently high mutagenesis potential and expression of functional BCR ABL from the locus promotes acquisition of BCR ABL mutations.

Because blast crisis CML cells are genetically unstable overexpression of BCR ABL alters the fidelity of DNA double strand break repair Slupianek et al. 2006 and increases expression and activity of error prone DNA polymerase which increases DNA replication error Canitrot et al. 1999 . Whether DNA replication error is a cause of rare mutations in clonal or parental cells during clonal expansion and cell propagation was assessed. BCR ABL mutation rates versus spontaneous mutation rates introduced by DNA replication error were determined. The latter was measured by spontaneous mutations on the HPRT hypoxanthine phosphoribosyl transferase gene that resulted in cells resistant to 6 thioguanine 6 TG . As shown in BCR ABL mutation rate was compared side by side with HPRT mutation rate in four relapse prone clones L1 L7 Ag3 and Ag11 and two never relapse clones L3 and Ag6 . HPRT showed relatively constant mutation rate 0.8 to 2.5 10 among all clones regardless of their ability to relapse whereas BCR ABL mutation rate was highly clone dependent from none clone L3 and Ag6 to 7 10. The stable HPRT mutation rate among clones reflects the nature of random mutations introduced by DNA replication error in these clones during clonal expansion which appears different from BCR ABL mutations that are highly clone dependent.

Treatment of CML cell lines K562 and BV173 with low doses of imatinib for a short time will specifically induce DNA damage in these cells but has no effect on non BCR ABL expressing leukemic or normal cells Czechowska et al. 2005 . Whether imatinib treatment increases HPRT mutation rate through elevating overall DNA damage level was determined. Since na ve CML cells cannot survive the prolonged culture required for a clonogenic assay recurrent cells derived from parental and clonal cells were used for treatment with 6 TG and 2.5 M. As shown in imatinib treatment did not alter overall DNA damage levels among these cells.

HPRT mutation types in 6 TG resistant cells were also examined. Loss of function for human HPRT has been well characterized with a broad mutation spectrum including large truncation deletion insertion and point mutations on codons Burkhart Schultz et al. 1996 Podlutsky et al. 1998 . From DNA sequencing analysis of 6 TG resistant soft agar colonies derived from clones Ag11 L1 and parental KCL 22 cells numerous HPRT truncation deletion insertion and point mutations were identified in all these colonies . This is consistent with previous observations Burkhart Schultz et al. 1996 Podlutsky et al. 1998 . The broad spectrum of HPRT mutations differs from the single mutations seen for the above described parental cells and clones indicating the possibility of a different mutagenesis pathway for BCR ABL on imatinib.

The T315I mutation frequency in parental KCL 22 cells was stable in early passages p8 and p16 but declined significantly in the late passage p36 . Thus the relapse of late passage KCL 22 cells was delayed although the recurrent cells still harbored T315I mutation. In contrast HPRT mutation rates remained relatively constant even in the late passages of KCL 22 cells. This indicates that BCR ABL mutations are actively induced by imatinib treatment.

The development of different mutations from clonal cells was examined using the model system of the present disclosure. It was verified that the clonal cells did develop different mutations. L1 and Ag11 cells were plated on soft agar with 2.5 M imatinib and ten imatinib resistant colonies were isolated after three weeks. All resistant L1 colonies carried the E255K mutation only and all resistant Ag11 colonies carried the T315I mutation only indicating clone specific mutation patterns exist for resistance.

Whether the parental mutation type T315I could be restored in clonal cells when re supplied with a culture environment with the mixture of clonal cells was determined. Clones L1 L7 or Ag3 respectively were mixed with an equal number of cells from a pool consisting of eight never relapse clones N pool . Identical mutation phenotypes were maintained for relapsed L1 L7 and Ag3 even after treatment with 2.5 or 5 M imatinib . Whether T315I mutation dominates over other mutations for resistance development was also determined. Equal numbers of cells from clones L1 L7 Ag3 and Ag11 were mixed together and treated with imatinib. In recurrent cells from liquid culture both E255K and T315I mutations were readily detected which was consistent with the fact that L1 and Ag 11 were fast relapse clones whereas L7 and Ag3 had slower relapse time courses indicating that T315I is not superior to E255K for imatinib resistance. Together these data indicate a certain plasticity of mutagenesis process in CML cells that may allow induction of different mutations in clonal cells. This plasticity is particularly evident in clone Ag3 which exhibits concentration dependent induction of T315I mutation at 10 M imatinib. Mutation phenotypes of some clonal cells are distinguished from that of parental cells and become clone dependent. Such results evidence that each clone adapts to a stable resistance mechanism.

The limitation of BCR ABL mutations to certain clonal cells prompted further examination into whether the ability to form BCR ABL mutations is restricted to a rare subpopulation of cells such as stably formed mutant cells or premutant cells that are not yet mutated but destined to form mutations. Cell pool analysis was performed by evenly dividing by volume 1 10KCL 22 cells calculated 25 mutants under 5 M imatinib into 24 wells with an average of calculated 1 mutant cell per well and the cells grew for 10 days in the absence of imatinib. If mutant cells were fully derived from a fixed mutant subpopulation they would have been randomly seeded into wells according to Poisson distribution. Therefore at least one mutant or pre mutant cell would have been expect to be in each of the 15 wells. After 10 days cells in all wells exhibited similar growth and reached an average of 1.8 10cells per well. All cells in each individual well were then analyzed for imatinib resistant clones by clonogenic assay with 5 M imatinib. Only 3 of the 15 wells produced resistant colonies p 0.0008 Table 2 . Even if it was assumed that all mutants or pre mutants happened to be seeded into these three wells an unlikely event enrichment of mutant or pre mutant cells should have been seen in these three wells. However an increased mutation frequency was not observed Table 2 .

To validate this finding another cell pool assay was performed by evenly dividing 1 10KCL 22 cells calculated 48 mutants under 2.5 M imatinib into 40 wells in the presence of 2.5 M imatinib an average of 1.2 mutant cell per well. Similarly 20 KCL 22M cells were evenly divided into 20 wells in the presence of 2.5 M imatinib. By Poisson distribution 28 wells should have received mutant or pre mutant cells for KCL 22 plating and 12 wells should have received cells for KCL 22M plating. However only one well of KCL 22 cells relapsed and re grew. In contrast five wells of KCL 22M cells re grew p 0.006 Table 3 . These data indicate that most BCR ABL mutant cells found in KCL 22 cells upon imatinib treatment are unlikely to have been derived from a fixed mutant or pre mutant subpopulation although the presence of a small portion of pre existing mutant cells can not be completely ruled out. Together with the cell cloning analysis these results indicate that the emergence of BCR ABL mutations may be a dynamic process influenced by culture and environmental conditions.

In combination with the materials and methods described in Example 1 above the following additional materials and methods were used.

Study of human samples was approved by the Institutional Review Board. CD34 cells were isolated from frozen bone marrow samples and transduced by retroviral vector MIG210 BCR ABL or empty vector M1GR1 as previously described Ramaraj et al. 2004 . Cells were harvested 48 hours later and labeled with anti CD34 APC. CD34 GFP cells were collected by flow cytometry. After cells were expanded in culture for seven days total RNA and protein lysate were prepared using standard protocols.

CML cells were seeded at 4 10 well in 96 well plates and co transfected with SIRT1 promoter luciferase reporters 2.8 kb and 90 bp promoter fragments Nemoto et al. 2004 and a control plasmid pmaxGFP Lonza . FeOfection reagent GENOVIS was used to transfect 0.18 g reporter and 0.02 g pmaxGFP each well per the manufacturer s instruction. Six hours after transfection cells were treated with imatinib and harvested after another 30 hs. Luciferase activity was measured using Dual Luciferase Assay System Promega and normalized to GFP reading in each lysate.

RNA was extracted with Trizol Invitrogen synthesized the first strand DNA with Superscript III kit Invitrogen and analyzed gene expression using SYBR GreenER qPCR SuperMix kit Invitrogen . For Western blots rabbit monoclonal anti human SIRT1 Epitomics rabbit anti mouse SIRT1 Upstate Biotech mouse monoclonal anti c ABL BD Pharmingen mouse monoclonal anti Ku70 Neomarker rabbit polyclonal anti acetylated p53 Cell Signaling and rabbit polyclonal anti acetylated FOXO1 Santa Cruz Biotech were used. To analyze Ku70 acetylation Ku70 was pulled down from total cell lysate with anti Ku70 and protein A agarose beads Upstate Biotech followed by acetylation detection with rabbit anti acetyl lysine antibody Cell Signaling .

For animal studies Kaplan Meier survival analysis was performed and statistical significance was calculated using log rank test. For other data analysis t test was performed. Two tailed analysis was used in all cases and P

The effects of SIRT1 stable knockdown on blockage of CML relapse by SIRT1 inhibitors were determined. SIRT1 shRNAs were designed as described previously Reynolds et al. 2004 . The first SIRT1 shRNA Sh1 was subcloned into a lentiviral vector pSicoR Ventura et al. 2004 which contains an expression cassette for green fluorescent protein GFP . A scrambled shRNA was subcloned into the vector as a mock control. The VSV G G protein of vesicular stomatitis virus pseudotyped lentiviral vectors were produced using a four plasmid transfection system as described Kowolik et al. 2003 . These vectors transduced KCL 22 and K562 cells with high efficiency and significant SIRT1 knockdown was observed in both cell lines . Mock or SIRT1 knockdown cells were enriched by fluorescent activated cell sorting FACS for GFP expression. No significant growth inhibition by SIRT1 knockdown on CML cells was observed. Treatment with imatinib resulted in relapse of the mock knockdown KCL 22 cells after two weeks. The SIRT1 knockdown showed significantly delayed or abolished relapse with relapse ranging from complete blockage during the two month culture to delay by 27 days . SIRT1 expression in the relapsed SIRT1 knockdown cells was restored to the same level as that in the mock knockdown cells indicating that relapse in the SIRT1 Sh1 knockdown cells is mediated by those cells which contain little SIRT1 knockdown in the original knockdown pool .

SIRT1 shRNAs were based on sirtuin type 1 SIRT1 mRNA complete cds Accession No. AF083106 and are as follows 

The heterogeneity of SIRT1 knockdown in the pooled population was confirmed using limiting dilution to clone individual SIRT1 knockdown and mock knockdown cells. About one third of SIRT1 knockdown clones did not have significant SIRT1 knockdown while SIRT1 level in mock knockdown clones remained unchanged.

Additional SIRT1 knockdown was also generated using different shRNA targets. Sh2 was cloned into a vector similar to pSicoR CMV GFP with a PKG puro expression cassette instead of CMV GFP cassette . This allows for enrichment of transduced cells using puromycin selection. SIRT1 Sh2 exhibited a higher knockdown efficiency than Sh1 . The mock knockdown KCL 22 cells relapsed after two weeks but SIRT1 Sh2 knockdown completely blocked the relapse for two months . Whether the vector sequence affects shRNA function was also assessed. SIRT1 Sh1 was subcloned into pSicoR PGK puro vector and found to function similarly to Sh1 in pSicoR CMV GFP vector only able to delay the relapse. This indicates that it is the level of knockdown but not the type of vector used that affects resistance. SIRT1 knockdown using both Sh1 and Sh2 resulted in a further decreased SIRT1 level completely blocking relapse of KCL 22 cells for two months . This indicates that SIRT1 is a gene that regulates CML acquired resistance by promoting BCR ABL mutagenesis during imatinib treatment.

To determine the consequence of SIRT1 activation KCL 22 and K562 cells were treated with a small molecule inhibitor of SIRT1 sirtinol with or without imatinib. Sirtinol alone inhibited the growth and induced apoptosis of both cell lines and the combination of sirtinol with imatinib further enhanced their inhibitory effects and apoptosis induction and . Sirtinol and imatinib both affected cell cycle of CML cells reducing S G2 M and increasing sub G1 population in KCL 22 cells while rapidly increased sub G1 and apoptotic fraction in K562 cells .

Since the level of SIRT1 knockdown affected the CML resistance to further determine specific roles of SIRT1 on CML cells SIRT1 was knocked down using a third and most potent set of lentiviral shRNA shSIRT1 3 . It was found that the knockdown suppressed proliferation of KCL 22 and K562 cells and had synergistic effects with imatinib on induction of apoptosis in these cells . Further it was found that overexpression of SIRT1 failed to protect CML cells from apoptosis induced by imatinib or BCR ABL knockdown although SIRT1 overexpression protected CML cells from oxidative stress which is consistent with the fact that BCR ABL expression activates multiple survival pathways Melo et al. 2007 .

Next SIRT1 knockdown was found to significantly inhibit soft agar colony formation of CML cells . To examine tumor growth in vivo CML cells were transduced overnight with SIRT1 shRNA and transplanted the cells into non obese diabetic severe combined immunodeficiency NOD SCID mice. SIRT1 knockdown significantly delayed the tumor growth of xenografted CML cells and prolonged the survival of mice P

Further to determine the effect of SIRT1 inhibition for BCR ABL transformation in primary CD34 cells we co infected the cells with MIG210 and SIRT1 shRNA. SIRT1 knockdown increased apoptosis of BCR ABL transduced CD34 cells but had minimal effect on normal CD34 cells . SIRT1 knockdown suppressed proliferation and colony forming cell CFC formation of the transduced CD34 cells . SIRT1 knockdown induced FOXO1 and p53 acetylation in BCR ABL transduced CD34 cells . Together these results indicate that SIRT1 knockdown suppresses growth and transformation of human CML cells.

Using lentiviral shRNA to knock down SIRT1 it was found that the ability for KCL 22 cells to form BCR ABL mutations was dependent on the residual amount of SIRT1 left and the shSIRT1 3 vector that resulted in the most robust knockdown completely inhibited BCR ABL mutations . Accordingly the relapse of KCL 22 cells on imatinib was delayed more with better SIRT1 knockdown with complete block of relapse by shSIRT1 3 . To determine if SIRT1 deacetylase activity is required for BCR ABL mutagenesis wild type or H363Y dominant negative deacetylase deficient SIRT1 Vaziri et al. 2001 was overexpressed in KCL 22 cells. Expression of either wild type or H363Y SIRT1 did not affect cell growth or induce apoptosis however H363Y SIRT1 expression significantly reduced BCR ABL mutations . Together these results reveal that high levels of SIRT1 in CML cells promote BCR ABL mutations for acquired resistance a process that is dependent on SIRT1 deacetylase function.

The potential downstream targets of SIRT1 that are involved in controlling the CML cell resistance are determined. p53 is a key downstream target of SIRT1 for apoptotic control in solid tumors. However p53 is mutated in both KCL 22 and K562 cells indicating that other SIRT1 targets might be involved. Involvement of another SIRT1 target peroxisome proliferator activated receptor gamma coactivator 1 alpha PGC 1 for induction of BCR ABL mutations through regulating reactive oxygen species ROS is examined. The ROS production is dramatically increased in blast crisis CML which results in accumulation of DNA damage product 7 8 dihydro 8 oxo 2 deoxyguanosine 8 oxoG . Overexpression of BCR ABL in murine cells induces ROS production DNA damage and BCR ABL mutations. However cellular ABL plays an important role in cellular DNA damage repair and treatment with STI 571 will also inhibit ABL and inactivate cellular DNA repair system. Most of the cellular ROS is produced by electron transport chain of active mitochondrial metabolism. PGC 1 is a master activator for mitochondrial biogenesis and respiration that promotes ROS production. SIRT1 plays a critical role in this regulation by deacetylating PGC 1 and thus activating its function. STI 571 treatment inhibits BCR ABL function and cell growth but simultaneously reduces ability of cellular DNA damage repair and results in T315I mutation unless SIRT1 is inactivated by sirtinol or PGC 1 is inactivated.

A search for essential SIRT1 pathways involved in regulating both CML cell survival and DNA damage responses was performed. Among known SIRT1 substrates p53 is mutated in both KCL 22 and K562 cells. Another target Ku70 is an important downstream effector for SIRT1 mediated cell survival Cohen et al. 2004 and is also an important component of error prone non homologous end joining NHEJ DNA repair machinery Khanna et al. 2001 . As shown in the acetylation levels of endogenous Ku70 in CML cells are increased in proportion to the levels of SIRT1 knockdown. When Ku70 is knocked down the combination of Ku70 shRNA with imatinib resulted in more apoptosis of KCL 22 cells than each individual agent . However the combination of Ku70 knockdown with imatinib did not result in a synergistic induction of apoptosis that was seen with the combination of SIRT1 knockdown and imatinib treatment indicating that Ku70 contributes in part to SIRT1 mediated CML cell survival. Ku70 knockdown completely blocked BCR ABL mutations and KCL 22 cell relapse on imatinib indicating that Ku70 is an important downstream target of SIRT1 for promoting BCR ABL mutations and acquired resistance. Therefore high levels of SIRT1 in cancer cells may activate Ku70 by deacetylation to promote error prone DNA damage repair which results in mutations for acquired resistance.

To further determine the role of SIRT1 in DNA damage repair stable KCL 22 cell clones bearing a NHEJ reporter construct EJ5 GFP were generated. For comparison with homologous recombination HR repair KCL 22 cell clones carrying a HR reporter DR GFP were also generated. After introducing DNA damage on these reporters using endonuclease I SceI it was found that NHEJ was much more efficient than HR for DNA damage repair in KCL 22 cells and that SIRT1 knockdown suppressed both HR and NHEJ activity with more prominent effect on NHEJ consistent with SIRT1 regulation of Ku70 functions as described above.

In addition to modulating Ku70 SIRT1 deacetylates HR repair factor Nijmegen Breakage Syndrome 1 NBS1 a component of MRN MRE11 RAD50 NBS1 complex and regulates recruitment of NBS1 and RAD51 to DNA damage foci for repair. It was found that both NBS1 and RAD51 knockdown also suppressed BCR ABL mutations and CML cell relapse on imatinib . This result was surprising because HR repair was not efficient in KCL 22 cells. In addition BCR ABL expression also compromises the fidelity of HR repair Nowicki et al. 2004 . Thus high levels of SIRT1 in CML cells may alter functions of cellular repair machineries in particular error prone DNA damage repair which promotes mutations for acquired resistance.

Aurora kinases are evolutionarily conserved family of serine threonine kinases with three homologous genes Aurora A B and C in mammals. Aurora A is essential for bipolar spindle assembly during mitosis and Aurora B ensures proper chromosome attachment to the mitotic spindle while Aurora C is involved in regulation of cilia and flagella. Aurora A is overexpressed in various types of human cancer and its gene amplification overrides the mitotic spindle assembly checkpoint results in defective spindle formation and multinucleation and increases cellular resistance to chemotherapeutic agent paclitaxel. Selective inhibitors of Aurora kinases have been developed for treatment of various cancers.

The increase of G2 M cell number enlarged cell size and bizarre morphology of KCL 22M cells show potential defects in cell mitosis which will deregulate Aurora kinases in these cells. Using Western blot it was found that the overall level of Aurora B in KCL 22 and KCL 22M cells are similar but Aurora A was abnormally stabilized in KCL 22M cells that might account for morphological and cell cycle changes in these cells . It has been found that the Aurora kinase inhibitor VX 680 potently inhibits T315I BCR ABL through a different structural mechanism from STI 571. Three CML patients with T315I mutation responded to VX 680 treatment indicating that clinical responses of CML patients with T315I can be enhanced with inhibition of Aurora kinases mutant BCR ABL or both. Aurora kinase plays a role in the resistance of T315I mutation in CML cells. VX 680 is a useful approach for eradicating mutant cells. Aurora A is abnormally stabilized in KCL 22M cells that have been treated with STI 571 but is degraded by treatment with SIRT1 inhibitor sirtinol . Abnormal stabilization of Aurora A can be caused by mutations of its DNA sequences for destruction boxes A and D due to the pressure imposed by STI 571 treatment. Where inhibition of BCR ABL alone is insufficient to eradicate T315I BCR ABL mutant cells combination treatment with SIRT1 inhibitors along with a dual BCR ABL and Aurora inhibitor such as VX 680 is useful for eradicating the resistant cells.

Whether stabilization of Aurora A in KCL 22M cells might be due to alterations of its destruction signals is determined. Aurora A is destructed after mitotic exit through ubiquibin ligase which is regulated by two conserved short amino acid sequences an N terminal A box and a C terminal D box and mutations of these sequences stabilize Aurora A. Besides phosphorylation of serine S51 in the A box of human Aurora A or S53 in also inhibits its destruction. There is structural similarity of kinase domains of BCR ABL and Aurora and cross reactivity of their inhibitors. These results show that Aurora A is involved in resistance of KCL 22M cells.

The effects of Aurora A knockdown for KCL 22M cells was also determined. Aurora A shRNAs was designed and tested. The knockdown of Aurora A was confirmed by Western blot. Whether this knockdown reduces G2 M cell population and restores normal cell morphology in KCL 22M cells by flow cytometry and whether it promotes apoptosis or inhibits growth of KCL 22M cells is also assessed.

To determine the role of Aurora A overexpression in CML chemoresistance Aurora A is expressed with an exogenous promoter in KCL 22 cells to examine whether it accelerated resistance and in STI 571 sensitive K562 cells to examine whether it helped these cells to develop resistance. Wild type and or mutant Aurora A is used depending on the sequencing results from the first experiment. Full length cDNA of Aurora A was PCR amplified and subcloned into an expression cassette with Simian virus 40 SV40 promoter in a lentiviral vector carrying CMV GFP which is similar to the vector used for shRNA packaging. When a mutant is needed site directed mutagenesis is performed in the subcloned Aurora A vector. Recombinant viruses are then produced and used to infect KCL 22 or K 562 cells and overexpression of Aurora A is verified by Western blot. Infected cells are isolated by FACS sorting for GFP expression and if necessary cloning of individual cells. Resistance in these cells was measured by rate of mutation time for relapse and concentrations of STI571 needed to repress cell growth and induce apoptosis.

Knockdown of BCR ABL has a greater impact on KCL 22 cells than KCL 22M cells and other gene changes such as Aurora A also have important roles in resistance of KCL 22M cells. Overexpression of Aurora A can render KCL 22 cells more resistant to STI 571 treatment and may develop resistant cells even without genetic mutations and persistent Aurora A may also render K 562 cells resistant to 1 M STI 571 treatment. Simultaneous knockdown of BCR ABL and Aurora A can have a more significant impact than individual knockdown on cell growth and apoptosis of KCL 22M cells. Treatment with VX 680 kills KCL 22M and KCL 22 cells efficiently as it inhibits both Aurora kinases and wild type or T315I BCR ABL.

It is examined whether the mechanisms that SIRT1 inhibition enhances CML apoptosis and prevents relapse on STI 571 in vitro applies to in vivo treatment. First a xenograft model of human CML cells was used in non obese diabetic severe combined immunodeficient NOD SCID mice and examined whether the combined treatment with sirtinol and STI 571 eliminates KCL 22 cells in recipient mice and prolongs their survival without relapse. Also examined was another CML cell line KU 812 for xenograft study. KU 812 is very sensitive to STI 571 in vitro but in vivo one third of mice with KU 812 xenograft relapse on STI 571 treatment after 48 to 60 days. It is determined whether the combination of STI 571 and sirtinol blocks this relapse. Second murine bone marrow retroviral transduction and transplantation model was used to define specific roles of SIRT1 in vivo for BCR ABL transformation and STI 571 resistance with SIRT1 knockout mice. Many hallmarks of human CML are faithfully reproduced in BALB c mouse models employing retroviral transduction of bone marrow cells with P210 BCR ABL followed by transplantation to syngeneic recipients. Mice develop CML like myeloproliferative disease characterized by massive extramedullary hematopoiesis in spleen liver and bone marrow with striking peripheral blood granulocytosis and die within 3 4 weeks after transplantation. Treatment with STI 571 prolongs survival in 80 of mice while the rest exhibit primary resistance and no mice can be cured by the treatment. It is examined whether homozygous SIRT1 knockout inhibits development of myeloproliferative disease in this model or significantly enhances the effects of STI 571 treatment to inhibit resistance and even cure the disease and whether combination of sirtinol and STI 571 will significantly improve treatment of the disease generated with wild type mouse bone marrow. In vitro colony forming unit CFU assay is used to determine whether combination of sirtinol with STI 571 will suppress blast forming unit erythroid BFU E and CFU granulocyte monocyte CFU GM colonies from late phase CML patients more efficiently than STI 571 itself.

Since no animal studies have been conducted with sirtinol before the pharmacokinetics of the compound was first determined. Female NOD SCID mice 6 to 8 weeks of age were given a single dose of sirtinol intraperitoneally i.p. and at seven time points namely 0.5 1 2 4 6 8 and 24 hours mice are sacrificed and blood bone marrow and liver are collected for HPLC MS analysis.

Second the effects of the combination of sirtinol and STI 571 for treatment of xenografted CML cells were determined. Induction of tumors by KCL 22 cells in NOD SCID mice was tested. Mice were irradiated with 270 Rads and 4 to 6 hours later transplanted with 3 million cells in phosphate buffered saline each through tail vein. Of 12 recipients one died within one week likely due to infection and the other 11 recipients developed tumors between 3 to 5 weeks. Most of mice exhibited visible tumors on neck eye and abdomen. Necropsy and histopathological examination revealed that these solid tumors were derived from lymph nodes throughout the body with the highest frequency in axillary pancreatic and renal nodes and tumors frequently infiltrated kidney eye and muscle . A significant number of human cells were not detected in peripheral blood by flow cytometric analysis of CD45 cells in all 11 mice before they were sacrificed for tumor burden. Microscopically visible tumors were not found in the spleen liver and lung.

To facilitate non invasive assessment of in vivo anti tumor activity of drugs KCL 22 cells stably expressing firefly luciferase were generated. Cells were examined for their responses for STI 571 and sirtinol treatment to ensure that luciferase expression does not alter effects of drugs. Luciferase expressing KCL 22 cells are transplanted into NOD SCID mice. Mice with established tumors by luciferase imaging are divided into 4 groups with 5 mice each group 1 vehicle control group 2 oral administration of STI 571 twice daily with a morning dose of 50 mg kg and an evening dose of 100 mg kg group 3 single i.p. injection of sirtinol with a dose producing stable plasma concentration of at least 50 M group 4 combination of STI 571 and sirtinol. Mice were imaged 5 to 7 days after treatment and total body bioluminescence was collected to quantify the changes of tumor progression and regression.

The ability of sirtinol and imatinib to block resistance related to BCR ABL mutations and non mutants was determined in acute lymphocytic leukemia ALL cells. In humans adult BCR ABL bearing ALL is highly chemoresistant and treatments are rarely successful resulting in a poor survival rate ranging from 

In combination with the materials and methods described in Examples 1 and 2 above the following additional materials and methods were used.

The use of animals was approved by the Institutional Animal Care and Use Committee. For CML tumor xenograft assay three million virally transduced cells were inoculated subcutaneously into the right flank of NOD SCID mice conditioned by 270 Rad irradiation. Tumor length L height H and width W were measured weekly with a caliper. Tumor volume is estimated from the formula V L H W 6 . Mice were euthanized when the tumor volume reached 1000 mm. For bone marrow transduction transplantation studies SIRT1 mice were backcrossed to BALB c background for eight generations. Retroviral transduction of bone marrow cells and transplantation were performed as described previously Pear et al. 1998 . PE labeled lineage Gr 1 Mac 1 B220 Ter119 and CD3e antibodies BD Pharmingen and APC labeled CD150 antibody Biolegend were used for cell analysis or sorting.

For the in vivo drug treatment study imatinib was prepared freshly in pure HO and tenovin 6 was prepared in 20 cyclodextrin w v Sigma C0926 and 10 DMSO v v as described Lain et al. 2008 and both were filtered sterile. At day 10 post transplantation drugs were administrated continuously for 10 days. Imatinib was administrated by oral gavage with 75 mg kg in the morning and 125 mg kg in the afternoon. Tenovin 6 was given by intraperitoneal injection at 50 mg kg in the morning. Vehicle control animals were treated with solution containing 20 cyclodextrin and 10 DMSO.

To further understand roles of SIRT1 in CML the impact of SIRT1 knockout on CML disease development in a mouse bone marrow transduction transplantation model Pear et al. 1998 was examined. Many hallmarks of human CML are reproduced in this model and recipient mice develop a CML like myeloproliferative disease and die within 2 4 weeks after transplantation. Since the efficiency for generating the CML like disease in this model is about 100 in BABL c strain SIRT1 knockout mice Cheng et al. 2003 were backcrossed to the BALB c strain for at least eight generations. A portion of SIRT1 mice survived through adulthood with relatively normal development although smaller body stature and closed eye lids were noticed in some mice as previously described Cheng et al. 2003 . Consistent with a previous report Narala et al. 2008 no significant difference in blood lineage differentiation and hematopoietic stem cell frequency were found in SIRT1 mice compared to wild type littermates .

Bone marrow cells from 3 month old SIRT1 and SIRT1 mice were transduced with BCR ABL MIG210 or control vector MIG R1 both bearing green fluorescent protein as a marker and transplanted 2.5 10or 1 10transduced cells into lethally irradiated BALB c recipients . It was found that the transduction and engraftment efficiency for both wild type and knockout cells were similar and mice receiving 2.5 10BCR ABL transformed SIRT1 bone marrow cells developed the CML like disease within three to four weeks with signature characteristics of the disease markedly elevated white blood cell WBC counts predominated by granulocytes splenomegaly and multiple organ involvement Pear et al. 1998 . Mice receiving BCR ABL transformed SIRT1 cells showed significantly delayed disease development with markedly lower peripheral blood WBC counts P

It was found that SIRT1 expression was dramatically upregulated in bone marrow cells from mice receiving BCR ABL transformed SIRT1 cells . To determine cell populations with SIRT1 expression change bone marrow cells were sorted for expression of hematopoietic stem progenitor cell marker CD150 Kiel et al. 2005 and green fluorescent protein GFP marker from the retroviral vector for transformation. It was found that SIRT1 activation occurred specifically in GFPCD150 transformed progenitor cells but not in GFPCD150 cells or non transformed cells similar to that seen for human CD34 progenitor cells . To further validate this finding CML progenitor cells were sorted using classic LinSca1cKit LSK markers and found that SIRT1 RNA was significantly elevated in CML progenitor cells .

Next it was determined whether treatment of CML mice with a SIRT1 inhibitor would deter the disease. Due to the solubility an effective concentration of sirtinol was not obtainable in vivo. Therefore a more potent and water soluble inhibitor tenovin 6 that inhibits SIRT1 and SIRT2 was tested. Treatment with tenovin 6 at 5 M and higher concentrations sensitized CML cells to imatinib induced apoptosis . It was found that tenovin 6 administrated at 50 mg kg day significantly extended CML mouse survival . No noticeable toxicity was detected with tenovin 6 alone or in combination with imatinib . Some mice with the combination treatment were the longest survivors . These results show that SIRT1 activation is required for efficient BCR ABL transformation of hematopoietic stem progenitor cells and CML disease development.

In combination with the materials and methods described in Examples 1 2 and 3 above the following additional materials and methods were used.

H3255 cell line is obtained from National Cancer Institute and H1650 cell line from American Type Culture Collection ATCC . Gefitinib is obtained from AstraZeneca or purchased from Euroasia Pharmaceuticals and sirtinol is purchased from Sigma. For resistance assay 2.5 10H3255 or H1650 cells are seeded per well in 6 well plates overnight and treat with drugs in 3 ml culture medium each. Cells are maintained in culture without medium changes. Relative cell numbers are analyzed over time in triplicate wells. Cells are fixed with 4 formaldehyde stained with 0.1 crystal violet and quantified by using Odyssey Infrared Imaging System LI COR Biosciences . When relapse occurs emerging cells reach confluence rapidly and are expanded into larger culture dishes. If the H3255 cells are unable to survive the applied dosage of gefitinib then a lower therapeutic dose is used such as 0.4 M or the lowest therapeutically effective dose.

Using the above procedures resistance models may be reproduced for H1650 cells. In addition a faster relapse for H3255 cells with T790M mutation may be obtained e.g. in two weeks . Inhibition of SIRT1 by sirtinol or by shRNA may have a synergistic or additive effect with gefitinib for inducing apoptosis and suppressing growth. The relapse normally obtained through mutation or non mutation mechanisms will be blocked or delayed.

The resistance models may be also reproduced for other cancer cell lines such as for example another NSCLC cell line PC 9 that also relapses on gefitinib treatment.

Cells were pre selected for four days in HAT medium to remove pre existing HPRT mutations. The efficiency of HAT selection was confirmed by plating these cells on soft agar with 2.5 g ml 6 thioguanine which produced zero colony. HAT selected cells were then treated with 0.5 M CPT for 1 hour and then cultured for 10 days before used for soft agar clonogenic assay with 6 thioguanine selection. The rest of HAT selected cells were cultured in medium without selection. The change of H2AX was analyzed with H2AX Assay Kit Upstate Biotech .

Five million KCL 22 cells were transfected with 15 g linearized repair reporter construct DR GFP or EJ5 GFP by electroporation and cells were selected for puromycin resistance. Individual clones were plucked from soft agar and expanded to screen for clones carrying an intact copy of the reporter constructs by Southern blotting as described previously Bennardo et al. 2008 Weinstock et al. 2006 . Clonal KCL 22 cells with an intact copy of reporters were transduced by shSIRT1 shRAD51 or shNBS1 for 24 h followed by electroporation with 50 g I SceI encoding plasmid plus 10 g Ds Red. After another 48 h culture the cells were analyzed by flow cytometry for GFP and Ds Red expression to determine the repair efficiency. GFP cells were the successfully repaired cells and repair rate was normalized to Ds Red transfection efficiency.

Bisulfite genomic DNA sequencing was performed as previously described Frommer et al. 1992 Herman et al. 1996 . Briefly 1 g of genomic DNA is treated with sodium bisulfite for 16 hours and the DNA is then recovered using Wizard DNA clean up system Promega . PCR products are cloned into pCR2.1 vector with TOPO TA cloning kit InVitrogen . At least ten clones each were sequenced for DNA methylation analysis. For quantitative HIC1 promoter methylation analysis the MassArray DNA methylation analysis system Sequenom was used as per the manufacturer s protocol at City of Hope Functional Genomic Core. The methylation for individual or aggregate CpG sites was analyzed with EpiTyper software v1.0 Sequenom .

This assay was performed as described previously. We used STAT5A or 5B antibody Cell Signaling for immunoprecipitation and used anti Flag antibody Santa Cruz as a control. The following primers were used for ChIP PCR analysis for STAT5A binding site 5 gcatctctgacctctcagca 3 sense SEQ ID NO 38 and 5 cagaaacaaaattcccagcttt 3 antisense SEQ ID NO 39 for STAT5B binding site 5 gggattggtatgaaggaacg 3 SEQ ID NO 40 and 5 agcgaaactccgtctcaaaa 3 antisense SEQ ID NO 41 for GAPDH 5 tctggggactaggggaagga 3 sense SEQ ID NO 42 and 5 ccgcaaggagagctcaaggt 3 antisense SEQ ID NO 43 .

KCL 22 cells were seeded at 5 10 ml in 12 well plate with 2.5 M imatinib or DMSO as mock control. Twelve hours later actinomycin D 2 g ml was added and at 0 1 2 4 and 8 hours after actinomycin treatment total RNA was extracted. First strand cDNA was reverse transcribed from 1.0 g of total RNA using Superscript III kit Invitrogen . One microliter of the first strand cDNA synthesis reaction mixture was used for PCR amplification in a total volume of 50 l. GAPDH was amplified with 25 cycles and SIRT1 was amplified with 30 cycles. The primer sequences used were 

The role of SIRT1 in regulating de novo DNA damage in CML cells was investigated. SIRT1 and mock knockdown KCL 22 cells were treated with hypoxanthine aminopterin thymidine HAT to remove preexisting HPRT mutation followed by treatment with camptothecin CPT to induce DNA damage. Cells were then analyzed for survival proliferation and HPRT mutations. SIRT1 Sh1 knockdown cells exhibited two fold less survival over time than mock cells . HPRT mutation rate in SIRT1 knockdown cells was decreased by 64 fold . Similar results were obtained by both SIRT1 Sh1 and SIRT1 Sh2 knockdown .

Additionally SIRT1 knockdown reduced spontaneous HPRT mutations when these cells were maintained for one month in culture . Furthermore SIRT1 knockdown robustly suppressed CPT induced de novo HPRT mutations in prostate cancer cells indicating that SIRT1 mediated mutagenesis is independent of cancer cell types. These results indicate that SIRT1 is a key protein promoting genetic instability and the evolution of oncogenes to resist and survive drug therapy and overcome DNA damage . These findings are surprising given that SIRT1 is shown to promote DNA damage repair Oberdoerffer et al. 2008 Wang et al. 2008 . It was also found that SIRT1 knockdown inhibited DNA damage responses by reduction of H2AX staining . SIRT1 inhibitors did not increase H2AX expression in CML cells in the presence or absence of imatinib in contrast to HDAC inhibitor trichostatin A TSA that elicited robust H2AX induction . These results indicate that although a high level of SIRT1 may enhance DNA damage responses in cancer cells it can also increase de novo genetic mutations. Such paradoxical functions of SIRT1 may be a result of altered DNA damage repair mechanisms in cancer as further described below.

Promoter hypermethylation of HIC1 increases progressively towards late phases of CML Issa et al. 1997 . We initially hypothesized that upregulated SIRT1 upon HIC1 gene silencing may be important for CML chemoresistance. Indeed we found that SIRT1 expression levels increased in blast crisis CML cell lines KCL 22 and K562 correlated with HIC1 promoter hypermethylation and silencing in these cells and . To examine SIRT1 expression in primary human CML we isolated CD34 CML progenitor cells. It was found that SIRT1 protein levels were tightly correlated with BCR ABL expression levels from chronic to blast crisis phase patients . Noticeably more advanced phase samples showed high BCR ABL levels along with high SIRT1 expression. This finding prompted the determination of the source of the change in SIRT1 expression during BCR ABL transformation. Normal CD34 cells were isolated from healthy donors and transduced with BCR ABL retroviral vector MIG210 Pear et al. 1998 . SIRT1 protein was hardly detectable in normal CD34 cells by Western blot but was dramatically upregulated by BCR ABL transduction . SIRT1 RNA level was also significantly upregulated by BCR ABL . BCR ABL transduction did not increase SIRT1 expression in normal human fibroblasts WI38 cells mouse fibroblast NIH3T3 cells human renal cancer cell line 293 and myeloid leukemia cell TF1 nor did it further increase SIRT1 levels in blast crisis CML cells . Accordingly BCR ABL transduction selectively activates SIRT1 expression in normal human hematopoietic progenitor cells.

To further examine regulation of SIRT1 expression in CML KCL 22 and K562 cells were treated with the BCR ABL kinase inhibitor imatinib. SIRT1 protein levels decreased in a drug concentration dependent manner. In contrast the treatment did not change SIRT1 expression in KCL 22M cells that express the imatinib resistant T315I kinase domain mutant BCR ABL or in a prostate cancer cell line . To determine specificity of imatinib effect on SIRT1 BCR ABL was knocked down using lentiviral shRNA. As a result it was found that SIRT1 protein levels decreased upon BCR ABL knockdown in KCL 22 KCL 22M and K562 cells . These data further support that BCR ABL regulates SIRT1 expression in CML cells.

Since SIRT1 expression is subjected to multiple levels of regulation including protein phosphorylation Ford et al. 2008 mRNA stability Abdelmohsen et al. 2007 and transcription Nemoto et al. 2004 Chen et al. 2005 it was determined how BCR ABL activates SIRT1 expression. It was found that BCR ABL knockdown only affected endogenous SIRT1 protein expression but not the exogenously introduced SIRT1 indicating that BCR ABL does not affect SIRT1 protein stability per se. After blocking transcription by actinomycin D imatinib treatment did not alter the half life of SIRT1 mRNA and thus its stability . To examine if BCR ABL expression activates SIRT1 promoter reporter constructs of the SIRT1 promoter were transfected into KCL 22 and K562 cells followed by imatinib treatment. Imatinib treatment significantly inhibited SIRT1 promoter activity . Together this data indicates that BCR ABL transformation of human hematopoietic progenitors selectively activates SIRT1 promoter and transcription.

As shown in SIRT1 expression was found to be tightly correlated with BCR ABL levels in CD34 hematopoietic progenitors from human CML patients . SIRT1 expression was hard to detect in normal CD34 progenitors but expression of BCR ABL dramatically increased SIRT1 protein and RNA levels . BCR ABL expression in normal fibroblast WI38 cells and several solid tumor cell lines did not increase SIRT1 expression indicating a relatively specific activation mechanism in hematopoietic progenitor cells. Conversely inhibition of BCR ABL by shRNA knockdown or imatinib in blast crisis CML cells reduced SIRT1 expression . As shown in BCR ABL expression activated SIRT1 promoter with a major responsive element located between 1.48 kb and 2.8 kb . The binding of STAT5 signal transducer and activator of transcription 5 in this region was identified as a novel mechanism for SIRT1 activation in hematopoietic cells . The rVista program Loots Ocharenko 2004 was used to search for potential transcriptional factors binding at this region that are related to BCR ABL functions. Two putative binding sites for signal transducer and activator of transcription 5 STAT5A and 5B located at 1838 and 2235 were identified. No other STAT members were found in this region.

STAT5 is a key signal transducer of BCR ABL in CML Ilaria et al. 1996 and is also activated in AML and acute lymphoid leukemia ALL Van Etten 2007 . STAT5 knockout suppresses BCR ABL transformation and development of CML like disease in mice Ye et al. 2006 Hoelbl et al. 2006 . Therefore it may serve as a common activator of SIRT1 in leukemia. To validate the role of STAT5 a STAT5 knockdown was performed using shRNA in CML cells. It was found that SIRT1 expression was decreased accordingly . Using chromatin immunoprecipitation ChIP assay it was found that both STAT5A and 5B were associated with the SIRT1 promoter . Together these results indicate that BCR ABL activates SIRT1 transcription in human CD34 progenitor cells for which STAT5 is a key mediator.

As shown in top SIRT1 inhibition by small molecule inhibitors or shRNA knockdown sensitized CML cells to imatinib for growth suppression and induction of apoptosis. SIRT1 knockdown inhibited soft agar colony formation and growth of xenografted CML cells in immunodeficient mice . To further examine SIRT1 functions in CML a mouse model employing retroviral BCR ABL transduction of mouse bone marrow cells was used followed by transplantation of these cells to syngeneic recipients Pear et al. 1998 . Many hallmarks of human CML are reproduced in this model and recipient mice develop a CML myeloproliferative disease and die in 3 4 weeks. SIRT1 knockout mice were backcrossed to BALB c background. The SIRT1 knockout did not affect normal hematopoiesis stem cell frequency or engraftment but the knockout inhibited BCR ABL transformation and development of the myeloproliferative disease . As in human progenitor cells SIRT1 expression was activated by BCR ABL in mouse HSC. SIRT1 knockout significantly reduced the number of mouse CML stem cells identified as cKitCD34 side population in BABL c strain Hu et al. 2006 . It has shown that SIRT1 knockdown or inhibition resulted in increased p53 acetylation and enhanced imatinib mediated apoptosis in human CML CD34 cells but not in normal cord blood CD34 cells . Similarly AML CD34 cells showed increased SIRT1 expression and increased apoptosis on SIRT1 inhibitor treatment ad G . The results of the studies described herein indicate that SIRT1 activation is required for efficient BCR ABL transformation. SIRT1 inhibition suppresses disease development and induces apoptosis in leukemic but not normal stem cells.

Imatinib is the frontline treatment for CML however acquired resistance develops frequently due to secondary BCR ABL mutations. A novel cell culture model in which a blast crisis human CML cell line KCL 22 following initial response to imatinib or nilotinib and dasatinib relapses in two weeks with acquisition of T315I mutation Yuan et al. 2010 has been developed as described above. Acquisition of BCR ABL mutations is dependent on expression of functionally intact BCR ABL Yuan et al. 2010 . Inhibition of SIRT1 by small molecules nicotinamide tenovin 6 sirtinol or splitomicin blocks relapse of CML cells upon imatinib or nilotinib and dasatinib treatment regardless if they can enhance apoptosis and . Tenovin 6 and nicotinamide blocked the relapse with a dose as low as 1 M below the concentrations to increase imatinib mediated cell killing C and A . In contrast HDAC inhibitor trichostatin A TSA fails to block relapse and .

It was found that treatment with the potent second generation BCR ABL inhibitors nilotinib and dasatinib also resulted in relapse and acquisition of T315I mutation in KCL 22 cells but the combination with tenovin 6 or sirtinol effectively blocked the recurrence Four KCL 22 cell clones were generated three of which can acquire different BCR ABL mutations upon imatinib treatment i.e. E255K clone L1 Y253H clone L7 and T315I clone Ag11 whereas clone Ag3 develops resistance without BCR ABL mutations. The combination of sirtinol with imatinib blocked relapse of all clonal KCL 22 cells on imatinib. In contrast TSA failed to block relapse of these clonal cells . These results indicate that combination of SIRT1 inhibition with BCR ABL inhibition is a powerful approach to overcome acquired resistance through BCR ABL mutations.

By gene knockdown it was found that the ability for KCL 22 cells to form BCR ABL mutations or to relapse was proportional to the residual level of SIRT1 protein left after SIRT1 knockdown . Further the shSIRT1 2 vector moderately reduced SIRT1 level and was insufficient to increase significant apoptosis also reduced BCR ABL mutations . This is in line with the effect of tenovin 6 described herein and indicates that BCR ABL mutation is more sensitive to SIRT1 function than apoptosis induction. To further validate that SIRT1 deacetylase activity is required for BCR ABL mutagenesis wild type or H363Y deacytylase deficient SIRT1 was overexpressed in KCL 22 cells. The expression of either wild type or H363Y SIRT did not affect cell growth but H363Y SIRT1 expression significantly reduced BCR ABL mutations . Together these results show that high levels of SIRT1 in CML cells promote BCR ABL mutations for acquired resistance a process that is dependent on SIRT1 deacetylase function.

The ability of SIRT1 to promote mutations is not restricted to BCR ABL in CML cells as SIRT1 knockdown in prostate cancer cells also dramatically suppressed mutations on the HPRT induced by DNA damaging agent camptothecin . It was further shown that an important mechanism for SIRT1 in BCR ABL mutagenesis and CML cell survival was through deacetylation and activation of Ku70 an essential factor for error prone non homologous end joining DNA repair Khanna et al. 2001 and a regulator of BAX mediated apoptosis Cohen et al. 2004 . Knockdown of Ku70 completely blocked BCR ABL mutations and CML cell relapse on imatinib treatment right . Together these studies indicate that SIRT1 plays an important role in molecular pathogenesis of CML and development of acquired resistance by promoting genetic mutations and cancer genome instability. SIRT1 may play a similar role in other leukemia and solid tumors where it is aberrantly activated.

Expression of HIC1 in CML cell lines KCL 22 and K562 was examined by quantitative real time RT PCR for the major HIC1 transcript from its promoter 1a. The gene was silenced or down regulated in both cell lines compared to full HIC1 expression in WI 38 cells and silencing in MCF 7 cells . This is a consequence of promoter hypermethylation Chen et al. 2003 Issa et al. 1997 . Western blot analysis showed that SIRT1 was overexpressed in both cell lines after HIC1 gene silencing .

Total cellular RNA was extracted with Trizol Invitrogen using a standard protocol. The first strand DNA was synthesized and HIC1 expression was analyzed by quantitative real time RT PCR using a kit with SYBR Green label Invitrogen as per the manufacture s instruction on BioRad machine OPTICON. HIC1 primers were used spanning introns 5 GGACGGACCAGCAGGACA 3 exon 1a SEQ ID NO 14 and 5 GCGCTGGTTGTTGAGCTG 3 exon 2 SEQ ID NO 15 . SIRT1 expression was analyzed by Western blot using 1 5000 diluted rabbit monoclonal SIRT1 antibody Epitomics . GAPDH was analyzed as a loading control with a rabbit antibody Trevigen at 1 5000 dilution.

Sirtinol inhibits SIRT1 and other sirtuin deacetylases with an ICabout 50 to 130 M Grozinger et al. 2001 Mai et al. 2005 Ota et al. 2005 . STI 571 treatment alone induced rapid apoptosis in K562 cells with ICof about 0.5 M while it had a mild effect on KCL 22 cells Mahon et al. 2000 . Treatment with 50 M sirtinol alone inhibited the growth of both cell lines and the combination of two drugs synergized their inhibitory effects . Sirtinol alone significantly induced annexin V positive apoptotic cells in both lines and when combined with STI 571 it induced more dramatic apoptosis than each individual drug . Sirtinol and STI 571 both affected cell cycle of CML cells by reducing S G2 and increasing sub G1 population in KCL 22 cells while they rapidly increased sub G1 and apoptotic fraction in K562 cells . Cell cycle was analyzed by fixing cells with 70 ethanol and then stained with propidine iodine 50 g ml for 30 min at room temperature. Cell apoptosis was analyzed with annexin V kit BD Pharminggen as per the manufacturer s instruction.

Using the CML acquired resistance model as described herein prevention of CML relapse on STI 571 was examined. Inhibition of SIRT1 with small molecule inhibitors prevented CML relapse on STI 571. KCL 22 cells were treated with STI 571 at 1 2.5 and 5 M in combination with various concentrations of sirtinol. At 50 M and above sirtinol effectively eliminated KCL 22 cells in two to three weeks and blocked relapse of KCL 22 cells on all three concentration of STI 571 in culture for two to three months and no viable cells were visible under microscope . As with STI 571 treatment with sirtinol alone resulted in relapse after two weeks . The combination of sirtinol and STI 571 is a powerful therapeutic approach for inhibiting acquired resistance of CML.

Another sirtuin specific inhibitor splitomicin Bedalov et al. 2001 Hirao et al. 2003 was also tested for its ability to block relapse. Splitomicin is also a naphthol compound and structurally similar to sirtinol. Nicotinamide is a natural inhibitor of SIRT1 Avalos et al. 2005 Bitterman et al. 2002 . At 300 M splitomicin and 15 mM nicotinamide relapse of KCL 22 cells on 5 M STI 571 was effectively blocked during prolonged culture as shown in . In contrast to sirtinol splitomicin and nicotinamide did not by themselves induce significant cell death and did not dramatically enhance cell killing by STI 571 during the first two weeks . These results indicate that prevention of relapse does not require enhanced cell killing.

The T315I mutation is resistant to treatment with nilotinib Weisberg et al. 2005 as well as dasatinib Shah et al. 2004 . In murine cells transduced with wild type BCR ABL the T315I mutation is also commonly induced by these drugs von Bubnoff et al. 2006 von Bubnoff et al. 2005 . Using the KCL 22M cell line combined treatment with sirtinol and STI 571 was able to inhibit cell growth .

Effects of inhibitors of deacetylases and DNA methyltransferases DNMT on chemoresistance of BCR ABL positive leukemia. In vitro enhanced killing of CML cells occurs when imatinib is combined with DNMT inhibitor 5 aza 2 deoxycytidine AZA La Rosee et al. 2004 or with HDAC inhibitors Kawano et al. 2004 Yu et al. 2003 . Whether these inhibitors prevent CML relapse on STI 571 was examined. TSA an HADC inhibitor at 1 M alone had little toxicity on KCL 22 cells but when combined with STI 571 massive apoptosis and dramatic cell killing was observed on all dosages of STI 571 . After ten days cells on all dosages of STI 571 relapsed . These results indicate that an initial rapid and massive cell killing does not necessarily prevent relapse of CML on imatinib treatment. However the combination of AZA with STI 571 while killing cells more slowly than TSA did successfully block relapse on 2.5 and 5 M . When AZA TSA and STI 571 were all combined initial rapid cell death occurred due to the presence of TSA and relapse was blocked on 2.5 and 5 M of imatinib due to the presence of AZA . TSA and AZA alone or in combination could not reactivate HIC1 expression or reduce SIRT1 level despite their effects on KCL 22 cells. This is in contrast to HIC1 reactivation by these drugs in carcinoma cells Narayan et al. 2003 Rathi et al. 2003 indicating that hypermethylated HIC1 promoter is more difficult to reactivate in CML cells. For KCL 22M cells TSA had no effect on cell growth whereas 1 M AZA inhibited its growth but did not promote significant cell death. The combination of 1 M AZA with 50 M sirtinol induced partial cell death .

Using the acquired resistance model described herein the effects of sirtinol and TSA was further investigated using clonal cells. These clones of KCL 22 cells develop acquired resistance to imatinib through different mutations in three clones E255K for clone L1 Y253H for clone L7 and T315I for clone Ag11 and non mutation mechanism in clone Ag3 Yuan et al. 2008 . As shown in the combination of sirtinol with 2.5 M imatinib blocked relapse in all four clones in contrast all clones relapsed on the combination of TSA with imatinib. Clone Ag3 developed resistance through non BCR ABL mutation mechanism by 2.5 M imatinib alone but developed resistance through T315I mutation upon the combined imatinib and TSA treatment. These data indicate that HDAC inhibitors transiently provide enhancement of cell killing but promote genetic mutations of BCR ABL and acquired resistance.

With the exception of KCL 22 cells no other CML cell lines are known to survive 1 M STI 571 treatment in serum containing culture. Whether serum free medium supplied with select growth factors provides a better culture environment for developing CML chemoresistance was determined. Serum free SF medium with insulin transferring epidermal growth factor EGF and high density lipoprotein HDL was used. SF medium with the basic supplements 5 g ml insulin and 5 g ml transferring was sufficient to support growth of both K562 cells and KCL 22 cells. Pre incubation of K562 cells in SF medium with basic supplements for two days made K562 cells as refractory as KCL 22 cells for treatment with 1 M STI 571 C . Sirtinol alone or in combination with STI 571 resulted in rapid killing of both K562 and KCL 22 cells and no relapse over prolonged culture. Sirtinol is able to overcome CML resistance in SF culture. Addition of basic supplements to routine serum media also provided K562 cells transient resistance to treatment with 1 M STI 571 which lasted up to two days. Again treatment with 50 M sirtinol abolished this transient resistance .

Transient resistance conferred by insulin and transferrin supplement is of interest as insulin receptor is overexpressed in about 90 primary CML cells. The key insulin downstream target phosphatidylinositol 3 kinase PI3K is required for BCR ABL mediated transformation and inhibition of PI3K Akt mTor pathway has been explored to overcome CML chemoresistance. SIRT1 is a key gene for regulating insulin secretion and directly regulates Akt downstream target FOXO proteins through deacetylation. Therefore the effect of insulin itself for transient resistance to imatinib and the response of sirtinol treatment were examined. The addition of a broad range of concentrations of insulin from 1 ng ml to 5 g ml provided about equal transient resistance of K562 to the treatment with 1 M STI 571 and again sirtinol alone or in combination with STI 571 abolished this transient resistance . Together these data show that activation of insulin pathway is sufficient to render CML cells transient resistance in vitro and it can be inhibited by SIRT1 inhibition.

Sirtinol inhibits NAD dependent class III histone deacetylases sirtuin family while TSA inhibits class I and II histone deacetylases HDACs . The sirtuin family deacetylases are structurally unrelated to HDACs. Both SIRT1 and HDACs are involved in regulating gene transcription and post translational deacetylation of numerous proteins. However two inhibitors exhibit sharply different outcomes on blocking KCL 22 relapse on STI 571. Gene expression microarray approach was used to search for differential transcriptional regulation and to use a proteomic approach to search for differential protein acetylation profiles.

The effects of these inhibitors on global gene transcription and global protein acetylation are analyzed. Total RNA is extracted from four groups of KCL 22 cells untreated treated with STI 571 alone treated with STI 571 and sirtinol and treated with STI 571 and TSA at three time points 2 4 and 8 days before relapse occurs. Each sample is collected in triplicate. Samples are then analyzed using Affymetrix expression arrays followed by statistical analyses of gene expression changes among different groups. Relevant genes are identified and RT PCR or Western blot is used to confirm their expression. The functional significance of new genes targets in CML chemoresistance is further studied by knockdown in KCL 22 cells similar to that described for SIRT1.

Total cell lysate is extracted from the four groups of KCL 22 cells at three time points as described for the expression array. Lysates are immunoprecipitated with anti acetylated lysine antibody and bound proteins are eluted for gel electrophoresis. The gel is stained with Coomassie and differential bands are sliced for mass spectrum analysis. Once key acetylation targets are identified they are verified individually by immunoprecipitation and Western blot. The following antibodies for protein analysis by Western blot are used rabbit monoclonal anti SIRT1 1 5000 Epitomics rabbit polyclonal anti PGC 1 1 1000 Chemicon and anti GAPDH 1 5000 Trevigen .

Acquired resistance through secondary mutations on the targeted oncogenes are also predominant in other cancers such as c Kit receptor and the platelet derived growth factor receptor PDGFR in gastrointestinal stromal tumors and epidermal growth factor receptor EGFR in non small cell lung cancer NSCLC . SIRT1 in NSCLC resistance with and without secondary mutations would be assessed in cell lines H3255 and H1650. The H3255 line carries activating mutation L858R and the H1650 cells carry activating deletion E746 A750 on the EGFR kinase domain which render these cells sensitive to treatment with EGFR inhibitors gefitinib and erlotinib. Both cell lines can relapse upon gefitinib treatment and H3255 cells develop resistance with the secondary T790M mutation Engelman et al. 2006. Allelic dilution obscures detection of a biologically significant resistance mutation in EGFR amplified lung cancer. 116 2695 2706 whereas H1650 cells develop resistance through mechanisms other than secondary mutations Kwak et al. 2005. Irreversible inhibitors of the EGF receptor may circumvent acquired resistance to gefitinib. 102 7665 7670. A previous H1650 resistance model was derived by exposing cells directly to 20 M gefitinib and a H3255 model was derived by exposing cells to gradually increasing concentrations of gefitinib starting with 40 nM for a few months. For patients receiving gefitinib 250 mg d the mean steady state plasma concentration of the drug ranges from 0.4 to 1.4 M and higher concentrations of gefitinib result in off target effects and toxicity Cohen et al. 2004. United States Food and Drug Administration Drug Approval summary Gefitinib ZD1839 Iressa tablets. 10 1212 1218 Baselga et al. 2002. Phase I safety pharmacokinetic and pharmacodynamic trial of ZD1839 a selective oral epidermal growth factor receptor tyrosine kinase inhibitor in patients with five selected solid tumor types. 20 4292 4302. . These previous methods did not use therapeutically effective concentrations of gefitinib. The NSCLC model provided herein employs approaches as those used in the described CML model studies.

2.5 10cells of H3255 or H1650 per well are seeded in 6 well plates. Multiple wells of each cell line are plated to allow sampling of cells at different time points. After overnight seeding cells are treated with 1 M gefitinib or a therapeutically effective dose. The drug is maintained in the culture until cells relapse and re grow as described for the CML studies. Fresh medium is supplied to restore the original volume during the prolonged culture as needed. Relative cell numbers are measured at various time points by staining cells with crystal violet and quantifying cell mass with Infrared Imaging System.

The relapsed cells are expanded and tested for their resistance to gefitinib as compared to parental cells. Total RNA is extracted for synthesis of cDNA and sequencing of EGFR kinase domain exons 18 24 and genomic DNA is extracted for direct sequencing to verify mutations detected by cDNA sequencing using primers described previously Pao et al. PNAS 101 13306 13311 2004 . Because H3255 harbors more than 40 copies of EGFR that results in allelic dilution of the T790M mutation a mutant enriched PCR sequencing assay is used to detect the mutation. Inukai et al. Cancer Res 66 7854 7858 2006 . Briefly after round one of amplification of both wild type and mutant alleles the wild type PCR products are selectively cleaved by restriction enzyme BstUI which allows second round preferential amplification of the mutant allele for sequencing. The mutant enriched PCR sequencing assay can detect one mutant allele among 1000 copies of wild type alleles and thus is sensitive enough for analysis of T790M mutation in H3255 cells. Using the above described method genetic mutants in the NSCLC acquired resistant cells are identified.

SIRT1 expression levels in lung cancer cells are assessed. SIRT1 protein level was previously found increased in lung cancer cell lines H460 and H209 that bear wild type EGFR. SIRT1 expression in H3255 and H1650 cells before and after relapse as compared to normal human lung lysates is analyzed by Western blot. Combination treatment of sirtinol at various concentrations with 1 M gefitinib may enhance apoptosis and inhibit growth of H3255 and H1650 cells. The apoptosis is analyzed by TUNEL staining and growth inhibition analyzed by crystal violet staining described above. The combination of the two drugs to block relapse of these two cell lines are examined for the prolonged culture up to two months. Third mock and SIRT1 knockdown are generated in H3255 and H1650 NSCLC cells using lentiviral shRNA vectors developed for CML resistance studies as described above. The knockdown of SIRT1 is confirmed by Western blotting. Whether SIRT1 knockdown delays or blocks relapse of NSCLC cells upon gefitinib treatment is examined.

To develop a potent SIRT1 inhibitor a pharmacophore model of SIRT1 inhibitors for the substrate binding pocket of SIRT1 was developed based on a sirtuin ligand database having 233 compounds specific to all sirtuin pockets. These highly conserved sirtuin pockets are based on small sirtuin protein crystal structures that show three pockets in each sirtuin protin for Zinc Zn Nicotinamide adenine dinucleotide NAD and a substrate. SIRT1 is about twice larger than other sirtuins and there is no crystal structure currently available.

The pharmacophore model was validated by selective identification of effective inhibitors for the SIRT1 substrate binding pockets after query of the ligand database. Using this pharmacophore model a search of a NCI library that included 250 253 compounds via Unity SYBYL 3D Flexible Database was performed. This search coupled with active compound references leadlikeness and diverse structure representation led to the identification of a top hit list of 23 inhibitor compounds. Using the culture model for CML acquired resistance described above these inhibitor compounds were tested to determine their ability to block CML cell relapse. Four of the identified compounds tested NCI compounds 163580 261408 628445 and 687305 were more effective in preventing CML cell relapse upon imatinib treatment than an effective dose of sirtinol. The effective dose of sirtinol that was used in this study was 50 M.

Compounds 628445 and 163580 were chosen as models to search for similar compounds against the NCI database because the structure of compound 687305 could not be confirmed by mass spectrum analysis and the compound 261408 was found too toxic to cells. Compounds derived from the 628445 search did not exhibit improvement of inhibitory activity. A similarity search for compound 163580 resulted in 17 potential NCI compounds 12 of which were available for testing . Nine of these 12 compounds derived from the similarity search of compound 163580 showed a significantly improved ability to block CML relapse with working concentrations as low as 1 M . Most of them were water soluble and had a structural feature distinct from other known SIRT1 inhibitors as described further below. As shown in eight of these nine improved compounds exhibited two structural moieties in common naphthol 22 at one end and isonicotinamide 24 or a similar structure at the other end. These two moieties were connected through a hydrophilic linker containing two nitrogen bond. Consistently the other four compounds lacking the isonicotinamide like moiety did not exhibit an improved inhibitory activity 27 39 40 or exhibited only moderate improvement 28 .

The compounds also showed SIRT1 inhibitory effects. For example compounds 22 N 2 hydroxyl 1 naphthyl methylene isonicotinohydrazide and 24 N 1 1 hydroxy 2 naphthyl ethylidene isonicotinohydrazide outperformed sirtinol and tenovin 6 as analyzed by a FRET fluorescent energy transfer based biochemical assay and cell based analysis of acetylation of FOXO1 a cellular substrate of SIRT1 . In sum the addition of isonicotinamide or a similar structure to the naphthol structural base would likely increase the inhibitory activity. Further isonicotinamide was shown to antagonize the effect of nicotinamide by blocking base exchange step of sirtuin deacetylation reaction in mM concentration Sauve et al. 2005 . However it is noted that isonicotinamide is not a per se requirement as it only exists in compounds 22 and 24. Therefore it is likely the isonicotinamide like structure feature that is important for the compounds.

In addition a SIRT1 catalytic core structure was constructed by homology modeling based on human SIRT2 structure PDB ID 1J8F and docking the compounds to the SIRT1 homology model indicated that these inhibitors bound at the substrate binding pocket . Surprisingly these compounds could also be docked into the NAD binding pocket . However docking into the NAD binding is consistent with the presence of the moiety of isonicotinamide like structure in these compounds showing that this moiety may block NAD binding NAD usage or interfere with base exchange and deacetylation steps through the NAD binding pocket. Given that inhibitors from individual binding pockets are generally weak simultaneous blocking substrate and NAD binding may have a synergistic effect on SIRT1 inhibition.

Optimization of lead compounds is carried out by two complimentary approaches Improvement of the pharmacophore and isothermal titration calorimetry.

Structure activity relationship SAR analysis is carried out on the lead NCI compounds identified through preliminary screens as well as on compounds collected from literature. The pharmacophore model is optimized. A focused compound library consisting of 30 to 40 compounds is designed based on the SAR study and the improved pharmacophore model. All compounds are synthesized by the City of Hope Synthetic Chemistry Core.

Synthesized compounds are first screened for their effective concentrations to block development of the BCR ABL T315I mutation using the CML KCL 22 cell culture model described above Yuan et al. 2010 . KCL 22 cells in liquid culture are treated with 2.5 M imatinib and various concentrations of SIRT1 inhibitors alone and in combination. Cells are monitored for cell death and relapse and the lowest concentrations of compounds that block relapse on imatinib in two to three weeks are identified and confirmed with prolonged culture. The ability of the compounds to block other BCR ABL mutations is similarly performed using three KCL 22 clonal cell lines previously developed Yuan et al. 2010 .

Inhibition of recombinant SIRT1 deacetylase activity is performed using a FRET based SIRT1 activity assay Cayman Chemicals and Cisbio that does not produce altered substrate SIRT1 interaction and ICof each compound is determined. To further validate non isotopic biochemical assay H labeled acetylated histone peptide Millipore is used as a substrate for SIRT1 deacetylation assay. The released H labeled acetate after the reaction is exacted and quantified by scintillation.

SIRT1 inhibitor induced acetylation of cellular SIRT1 substrates Ku70 and FOXO proteins are determined by immunoprecipitation of these proteins followed by Western blot analysis for acetyl lysine. Acetylation of FOXO1 is also directly examined by flow cytometry based analysis as done in .

Next cross reactivity of an inhibitor to HDACs and other sirtuins is examined using commercially available deacetylase assay kits Biomol and Millipore . A second round of optimization is optionally performed. A further refined pharmacophore model may be deduced on the basis of the above studies of the designed compounds. The 3D pharmacophore search may be used to query an 80 000 diverse compound library purchased by City of Hope and the UCSF ZINC 13 million purchasable compound library to identify similar compounds for lead optimization. About 30 50 compounds with predicted favorable ADMET properties are synthesized and biological tests are carried out as described above. Three dimensional Quantitative Structure Activity Relationship 3D QSAR models are built for compounds whose ICfor SIRT1 is determined. The predictive power of the QSAR correlation is used to estimate the bioactivity for new compounds and to guide the modification and refinement of existing structures to obtain optimal activity.

ITC is an emerging technique for drug discovery. ITC determines the enthalpy change associated with binding and provides a complete thermodynamic profile of the interaction including binding affinity number of binding sites entropy change and Gibbs energy of binding Ladbury et al. . This approach offers an advantage for lead compound optimization in the absence of SIRT1 crystal structure. ITC study of the compounds is carried out at the City of Hope X ray Crystallography Core using TA Instruments Nano ITC Low Volume. For these measurements the SIRT1 200 L is titrated 25 injections 2.5 L each with each inhibitor. The concentrations of the ligand and receptor are optimized to ensure accurate fitting e.g. a clear sigmoidal curve . In addition the ligand is titrated into the buffer to account for heat of dilution and subtracted from the ligand receptor titration. The data is processed using the TA software package. The thermodynamic profiles obtained from these compounds guide modification to obtain a high affinity and specific SIRT1 inhibitor i.e. compounds optimized for a more favorable negative enthalpy change or a more favorable positive entropy change to improve affinity and selectivity for SIRT1. New compounds are then synthesized for further ITC measurements followed by cell based analyses as described in the above section.

An all around docking algorithm has been designed to perform docking of the designed compound library cross validated by QSAR and ITC analysis results into the predicted binding sites on the SIRT1 structure model. The protein ligand binding free energies may be generated from CHARMM molecular dynamic MD simulation by implementing enhanced sampling algorithms Li et al. 2007 Li et al. 2006 . The best predicted ligand binding mode then leads the optimization of improving binding affinity and the specificity.

Ligand binding mode shows how the inhibitor interacts with the protein. A homology model of SIRT1 has been built based on the structure of SIRT2 that shares more than 70 sequence homology for the substrate binding pocket. The lead compounds are examined by docking analysis with the SIRT1 homology model using Sybyl Tripos and Glide Schrodinger software followed by molecular dynamic simulation. The docking and free energy component analysis will reveal amino acids of SIRT1 that interact with the compounds. As shown above in docking of compound 22 results in hydrogen bonding with residues Ser370 Lys408 Glu416 and Asn417 of SIRT1 were identified and energetically favorable hydrophilic interactions were predicted between the compound and the substrate binding pocket. The conserved amino acids of SIRT1 that interact with most compounds docking into the substrate binding pocket are identified. The predicted ligand binding pocket and binding mode are validated by mutant SIRT1 proteins generated by site directed mutagenesis the ITC studies described above and Biacore analyses. Biacore is another quantitative assay for specific interaction between a ligand and a protein immobilized on a sensor surface. Biacore analysis is performed at the City of Hope High Throughput Screen Core. The ligand binding pocket and the binding mode are further validated once the structure for SIRT1 catalytic core domain or a larger fragment is obtained. The ligand binding mode also guides the lead optimization to improve binding affinity and selectivity as described above.

The structures of several SIRT1 homologous proteins are available however the crystal structure of SIRT1 either catalytic core or full length has not been solved. SIRT1 is the largest sirtuin protein with very long N and C termini that are flexible and difficult to be crystallized. Accordingly additional structural information for SIRT1 enhances understanding of precise drug inhibition mechanism and efforts to determine specificity of SIRT1 inhibitors.

First different constructs are expressed by creating a his tagged SMT3 SIRT1 fusion. The N terminal SMT3 typically improves expression and permits highly specific cleavage of the tag by the SMT3 specific protease Ulp1 as previously described Mossessova et al. 2000 Sun et al. 2009 Ahmed et al. 2010 . Initially a fragment that spans the catalytic core residues 220 494 based on BLAST searches of SIRT1 and the PDB and full length protein are generated. Recombinant SIRT1 is expressed in bacteria and purified using standard chromatographic methods. Briefly the clarified lysate is passed over a Ni NTA column washed extensively and eluted with imidazole and the SMT3 tag is cleaved with histag Ulp1 and dialyzed. A second Ni NTA column removes the SMT3 and Ulp1. The cleaved SIRT1 protein is concentrated and passed over a size exclusion column Superdex G75 . Progress is monitored by SDS PAGE. An ion exchange column is used if additional steps are required. Approximately 5 to 100 mgs of protein are typically generated using this system.

In addition limited proteolysis techniques are used on the full length protein to experimentally define the molecular boundaries of a stable construct. Briefly the purified full length SIRT1 is exposed to papain and subtilisin at different concentrations for a defined period. SDS PAGE is used identify stable fragments. These fragments are excised from the gel and subjected to N terminal protein sequencing and mass spectrometry. New constructs are generated expressed and purified based on this information. Deacetylase activity of truncated SIRT1 is validated by biochemical assays as described above.

Next a generate crystal structure of the stable SIRT1 fragments are generated in the presence and absence of an inhibitor or an acetylated p53 peptide. Crystallization and structure determination are performed at the City of Hope X ray Crystallography Core. Specifically a Mosquito Crystallization robot is used to generate 200 nL drops for each condition. Each screen 4 are initially tested consists of 96 different crystallization conditions. Crystallization trials are monitored robotically using the Formulatrix Rock Imager at 4 C. and 22 C. in total 768 conditions are initially tested per sample . Promising leads are optimized and the diffraction properties of each lead are tested. The data is collected and analyzed. The apo structure acetylated peptide bound and inhibitor bound structures are analyzed and compared to known SIRT2 SIRT3 and SIRT5 structures. Common and unique features of SIRT1 crystal structures for substrate and inhibitor binding are identified.

Successful determination of the structure will point to residues that are important for ligand and inhibitor binding. This structural information is used to improve the SIRT1 homology. Residues that are important for inhibitor binding are confirmed by site directed mutagenesis. Deacetylase activity of mutant proteins is measured as described above and interaction of mutant proteins with compounds is characterized by ITC as described above.

Next the effects of in vitro inhibitor treatment on acetylation of SIRT1 targets in human CD34 CML cells are examined. Human CD34 CML cells are isolated and cultured in vitro using a protocol previously described Bhatia et al. 2003 Holtz et al. 2002 . Acetylation of p53 and FOXO proteins upon inhibitor treatment is analyzed by Western blotting or by flow cytometry based assay as described above. The ability of inhibitors to selectively alter gene expression associated with SIRT1 signaling is evaluated by microarray analysis. Affymetrix GeneChip Human Genome U133 Plus 2.0 Arrays were used to analyze gene expression changes in LSC upon imatinib and HDAC inhibitor treatment Zhang et al. 2010 . SIRT1 related gene expression patterns are determined by analysis of expression changes in CD34 CML cells following SIRT1 shRNA knockdown. Gene Set Enrichment Analysis GSEA is used to determine pathways significantly altered by SIRT1 expression. Connectivity Map CMAP Lamb et al. 2006 is used for the gene signatures derived above to explore the functional connection of genetic perturbation and drug activities. Signaling pathways affected by specific SIRT1 inhibitors should significantly overlap with those by SIRT1 shRNA knockdown but should be distinct from those affected by HDAC inhibitors Zhang et al. 2010 . In vivo target inhibition specificity is examined as described below.

Another approach that may be used is to mutate surface residues with high entropy e.g. predicts residues 235 238 within the catalytic core should be mutated to alanines http nihserver.mbi.ucla.edu SER . In addition the SIRT2 and other sirtuin homologues may be used to identify productive crystal contacts n.b. the human SIRT2 homologue diffracted to 1.7 . In both cases site directed mutagenesis is used to modify these residues and the assays described above are used to ensure that these mutations do not affect the activity. Functional constructs are subjected to similar crystallization trials.

The protocol for bone marrow transduction and transplantation is performed as previously described Pear et al. 1998 . Briefly donor mice were primed by 5 fluorouracil 3 days before harvest. Bone marrow cells were transduced with retroviral vectors by co sedimentation. Typically 2.5 10transduced mononuclear cells are transplanted to each recipient. A typical study consists of four groups of mice vehicle imatinib SIRT1 inhibitor and imatinib plus SIRT1 inhibitor with 12 mice per group. Drugs are given for 10 consecutive days beginning on day 10 after transplantation as described Wolff et al. 2001 . Imatinib is administered orally 50 mg kg in the morning and 100 mg kg in the afternoon with 8 hours apart. Two of the lead compounds discussed above are used to set up assays initially. Pharmacokinetics is performed as described below to determine the optimal drug doses and delivery routes before they are used for treatment. Two doses or routes each inhibitor are examined. After 10 day treatment peripheral blood is collected and analyzed by differential blood cell counts using HemaVet Drew Scientific to determine the effects of the drug treatment. Mice are monitored for up to six months and Kaplan Meier survival analysis is performed with log rank test used for calculating statistical significance.

CML stem cells are identified as a fraction of side population GFPcKitCD34Hoe in BABL c background Hu et al. 2006 . To determine if SIRT1 inhibitor treatment eradicates LSC bone marrow of the above described CML mice in different treatment groups is harvested. Lineage positive cells are depleted using immunomagnetic beads EasySep Stem cell Technology . The enriched cells will then be labeled with Hoechst 33342 CD34 and c Kit for LSC analysis as shown in the Preliminary Studies. To further examine the eradication of CML stem cells serial bone marrow transplantation is performed by transplanting 5 10bone marrow cells from mock and drug treated groups into lethally irradiated secondary recipients. The disease development and LSC is examined similarly as in primary recipients. The same regimen of drug treatment is tested in normal mice to determine impact of drugs on normal HSC.

The effect of drug treatment on cell cycle and apoptosis of LSC is examined. LSC is labeled with Pyronin Y and Hoechst 33342 to identify G0 and G1 population. Apoptosis of LSC is monitored by annexin V staining. To determine the in vivo target specificity of a SIRT1 inhibitor in mouse LSC microarray based gene expression analysis is carried out by the City of Hope Functional Genomics Core and Division of Biostatistics. Mock and BCR ABL transformed HSC is isolated from SIRT1 and SIRT1 mice to identify the altered signaling pathways by SIRT1 knockout using Affymetrix mouse 430 2.0 arrays coupled with a molecular amplification technique developed in the Core for analyzing small number of cells. Since LSC in SIRT1 mice are few LSC may be harvested from bone marrow and spleen and pool LSC from 5 to 10 mice for the assay if necessary. Gene expression profiling will then be performed on LSC derived from wild type CML mice immediately after 10 day treatment with SIRT1 inhibitors or vehicle 8 mice each. SIRT1 inhibitor induced signaling pathway alteration is compared with SIRT1 knockout to determine if the inhibitor selectively inhibits SIRT1 signaling in vivo.

CD34 cells are isolated from samples from CML patients who have not received Imatinib and normal healthy controls using immunomagnetic columns Miltenyi . CD34CD38 and CD34CD38 cells are selected using flow cytometry sorting. Cells are cultured in medium supplemented with low concentrations of growth factors with the following treatments i No treatment ii SIRT1 inhibitor alone iii Imatinib alone and iv SIRT1 inhibitor combined with Imatinib. Cell proliferation is examined by 5 and 6 CFSE labeling and apoptosis by annexin V labeling. A proliferation index PI is calculated based on the CFSE fluorescence profile as described previously Bhatia et al. 2003 Holtz et al. 2002 . The percentage of annexin V cells for total as well as dividing and non dividing cells CFSE low and high respectively is calculated. Colony forming cells CFC assays are performed by plating cells in methylcellulose progenitor culture for 14 18 days. A similar approach is used to study the efficacy of the drug on AML stem cells using samples from newly diagnosed untreated AML patients.

To analyze the effect of drug treatment on CML and normal human HSC capable of initiating multilineage engraftment in NOD SCID mice SCID repopulating cell SRC the NOD SCID IL2R chain KO NSG model that supports improved engraftment of human hematopoietic cells is used. Although larger numbers of CML cells are required to establish engraftment in NSG mice compared to normal cells it has been found that engraftment with BCR ABL cells is consistently observed and can be reproducibly measured. This assay has been used to demonstrate targeting of CML SRC by HDAC inhibitor and Imatinib combination Zhang et al. 2010 . CML and normal human CD34 cells CML 1 10per mouse normal 2 10cells mouse are exposed in vitro to a SIRT1 inhibitor Imatinib or the combination of the two agents are followed by injection into sublethally irradiated 300 cGy 8 week NSG mice via tail vein injection. Blood samples are obtained to monitor human cell engraftment 4 weeks after injection by flow cytometry for human CD45 cells . At 8 12 weeks animals are euthanized and marrow content of femurs and tibiae spleen cells and blood obtained. Human cell engraftment is analyzed by flow cytometry after labeling with anti human CD45 antibody. Specific human cell subsets are detected with antibodies to human CD34 CD38 CD14 CD11b CD33 Glycophorin A CD19 and CD3. The proportion of BCR ABL human cells is evaluated by Q PCR analysis of cells engrafted in mouse tissue for BCR ABL and BCR human levels. A similar approach is used to analyze the effects of drug treatment on AML stem cells.

The initial dose will start with the equivalent to ICdose of SIRT1 inhibitors in vitro via intravenous IV injection. All animals are monitored for 4 hr immediately following administration of the inhibitor and will continue to be monitored for 1 week to observe for signs of delayed toxicity which includes loss of appetite changes in waste elimination hunched posture coat ruffling eye crustiness and changes in activity level. The mice will also be weighed 24 hr post treatment and animals that lose 20 or more of its body weight are euthanized. Dose escalation by 1.5 fold increase or reduction by is carried out until the MTD is reached or until solubility limits the procedure. Once single dose MTD is identified multi dose studies are carried out accordingly.

A three phase single dose PK study is performed in mice at the MTD following intravenous IV or oral administration to determine the half life T of the SIRT1 inhibitors. Blood samples are collected at different time points 0.25 48 hours for LC MS analysis of drug concentrations. Once the Tis determined phase 2 PK studies are conducted to characterize the dose dependent PKs volume of distribution Vd clearance CL maximal drug concentration C and the systemic bioavailability following both IV and oral single dose administrations. Phase 3 PK studies will collect normal tissues for an assessment of drug distribution.

A SIRT1 inhibitor for in vivo use must also consider factors such as stability clearance toxicity and solubility so metrics such as formulation PK toxicity and efficacy data are used to guide further chemical modifications to optimize inhibitors and to choose additional backup inhibitors.

The references listed below and all references cited in the specification above are hereby incorporated by reference in their entirety as if fully set forth herein.

