---

title: Methods and systems for processing polynucleotides
abstract: The present disclosure provides compositions, methods, systems, and devices for polynucleotide processing. Such polynucleotide processing may be useful for a variety of applications, including polynucleotide sequencing.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09567631&OS=09567631&RS=09567631
owner: 10X GENOMICS, INC.
number: 09567631
owner_city: Pleasanton
owner_country: US
publication_date: 20131212
---
This application claims the benefit of U.S. Provisional Patent Application No. 61 737 374 filed Dec. 14 2012 which is incorporated herein by reference in its entirety for all purposes.

The instant application contains a Sequence Listing which has been submitted electronically in ASCII format and is hereby incorporated by reference in its entirety. Said ASCII copy created on Jan. 14 2014 is named 43487 703.201 SL.txt and is 4 650 bytes in size.

The processing of polynucleotides and polynucleotide fragments is a critical aspect of a wide variety of technologies including polynucleotide sequencing. Polynucleotide sequencing continues to find more widespread use in medical applications such as genetic screening and genotyping of tumors. Many polynucleotide sequencing methods rely on sample processing techniques solely utilizing random fragmentation of polynucleotides. Such random uncontrolled fragmentation can introduce several problems in downstream processing. For example these methods may produce fragments with large variation in length including a large number or fraction of sequences that are too long to be sequenced accurately. This results in a loss of sequence information. Current methods of processing may also damage polynucleotides resulting in incorrect sequence information and or the loss of sequence information. These and other problems may be significantly amplified by relatively minor operator variability. Thus there is a significant need for improved methods that provide better control over all aspects of polynucleotide fragmentation and processing. In particular there is need for polynucleotide processing methods that consistently provide fragments of appropriate size and composition for any downstream application including sequencing.

This disclosure provides methods compositions systems and devices for processing polynucleotides. In one example a method provided herein comprises a providing a target polynucleotide b fragmenting said target polynucleotide to generate a plurality of non overlapping first polynucleotide fragments c partitioning said first polynucleotide fragments to generate partitioned first polynucleotide fragments wherein at least one partition of said partitioned first polynucleotide fragments comprises a first polynucleotide fragment with a unique sequence within said at least one partition and d fragmenting said partitioned first polynucleotide fragments to generate a plurality of non overlapping second polynucleotide fragments.

In some of the methods provided in this disclosure a third and fourth set of polynucleotide fragments are generated by performing the method described above and additionally performing a method comprising a fragmenting said target polynucleotide to generate a plurality of non overlapping third polynucleotide fragments b partitioning said third polynucleotide fragments to generate partitioned third polynucleotide fragments wherein at least one partition of said partitioned third polynucleotide fragments comprises a third polynucleotide fragment with a unique sequence within said at least one partition and c fragmenting said partitioned third polynucleotide fragments to generate a plurality of non overlapping fourth polynucleotide fragments.

The third polynucleotide fragments may overlap with the first polynucleotide fragments. The fourth polynucleotide fragments may overlap with the second polynucleotide fragments.

In some cases at least one of the first second third and fourth polynucleotide fragments are generated by an enzyme. The enzyme may be a restriction enzyme. The restriction enzyme used to generate the first polynucleotide fragments may be different from the restriction enzyme used to generate the third polynucleotide fragments. The restriction enzyme used to generate the second polynucleotide fragments may be different from the restriction enzyme used to generate the fourth polynucleotide fragments. The restriction enzymes may have a recognition site of at least about six nucleotides in length.

The fragments can be of a variety of lengths. For example the first and or third polynucleotide fragments may have a median length of least about 10 000 nucleotides. The second or fourth polynucleotide fragments may have a median length of less than about 200 nucleotides.

The fragments can be attached to barcodes. For example the second polynucleotide fragments and or the fourth polynucleotide fragments may be attached to barcodes to generate barcoded second and or fourth polynucleotide fragments. The barcodes may be polynucleotide barcodes. The attachment of the barcodes to the polynucleotide fragments may be performed using an enzyme. The enzyme may be a ligase. The barcoded fragments may be pooled. Unpooled or pooled barcoded fragments may be sequenced.

In some cases one or more steps of the methods described in this disclosure may be performed within a device. The device may comprise at least one well. The well may be a microwell. Any of the partitioning steps described in this disclosure may be performed by dispensing into a microwell.

The microwell or well may comprise reagents. These reagents may be any reagent including for example barcodes enzymes adapters and combinations thereof. The reagents may be physically separated from a polynucleotide sample placed in the microwell. This physical separation may be accomplished by containing the reagents within a microcapsule that is placed within a microwell. The physical separation may also be accomplished by dispensing the reagents in the microwell and overlaying the reagents with a layer that is for example dissolvable meltable or permeable prior to introducing the polynucleotide sample into the microwell. This layer may be for example an oil wax membrane or the like. The microwell may be sealed at any point for example after addition of the microcapsule after addition of the reagents or after addition of either of these components plus a polynucleotide sample.

Partitioning may also be performed by a variety of other means including through the use of fluid flow in microfluidic channels by emulsification using spotted arrays by surface acoustic waves and by piezoelectric droplet generation.

Additional methods of fragmenting nucleic acids that are compatible with the methods provided herein include mechanical disruption sonication chemical fragmentation treatment with UV light and heating and combinations thereof. These methods may be used to fragment for example the partitioned first or third polynucleotide fragments described above.

Partitioning may be done at any time. For example the first polynucleotide fragments and or the third polynucleotide fragments may each be further partitioned into two or more partitions before further processing.

This disclosure provides methods for pseudo random fragmentation of polynucleotides. In some cases such methods comprise a providing a target polynucleotide b fragmenting said target polynucleotide to generate a plurality of first polynucleotide fragments c partitioning said first polynucleotide fragments to generate partitioned first polynucleotide fragments such that at least one partition comprises a first polynucleotide fragment with a unique sequence within said at least one partition and d fragmenting said partitioned first polynucleotide fragments with at least one restriction enzyme in at least one partition to generate a plurality of second polynucleotide fragments wherein said partitioned first polynucleotide fragment is fragmented with at least two restriction enzymes across all partitions.

In some cases at least two restriction enzymes are disposed within the same partition. In some cases at least two restriction enzymes are disposed across a plurality of different partitions.

The pseudo random fragmentation methods can be performed in order to yield fragments of a certain size. In some cases at least about 50 of the nucleotides within a target polynucleotide are within about 100 nucleotides of a restriction site of a restriction enzyme used to perform pseudo random fragmentation. In some cases at most about 25 of the nucleotides within a target polynucleotide are within about 50 nucleotides of a restriction site of a restriction enzyme used to perform pseudo random fragmentation. In some cases at most about 10 of the nucleotides within a target polynucleotide are more than about 200 nucleotides from a restriction site a restriction enzyme used to perform pseudo random fragmentation.

The pseudo randomly fragmented polynucleotides may be attached to barcodes to generate barcoded polynucleotide fragments. The barcoded polynucleotides may be pooled and sequenced.

The number of partitions holding the partitioned first polynucleotide fragments may be at least about 1 000 partitions. The volume of these partitions may be less than about 500 nanoliters.

Each enzyme may occupy an equivalent number of partitions or each enzyme may occupy a different number of partitions.

This disclosure provides methods for recycling certain unwanted reaction side products back into starting materials that can be used to generate a desired product. In some cases these methods comprise a providing a first polynucleotide a second polynucleotide a first restriction enzyme and a second restriction enzyme wherein said first polynucleotide comprises a target polynucleotide or a fragment thereof and b attaching said first polynucleotide to said second polynucleotide to generate a polynucleotide product wherein said first restriction enzyme cuts a polynucleotide generated by attachment of said first polynucleotide to itself said second restriction enzyme cuts a polynucleotide generated by attachment of said second polynucleotide to itself and neither said first restriction enzyme nor said second restriction enzyme cuts said polynucleotide product.

The first polynucleotide may be generated in the same reaction volume as the polynucleotide product or in a different reaction volume. The target polynucleotide may be for example a fragment of genomic DNA.

The second polynucleotide may be generated in the same reaction volume as the polynucleotide product or in a different reaction volume. The second polynucleotide may be for example a barcode or an adapter.

The first restriction enzyme may have a recognition site of at most about four nucleotides in length. The second restriction enzyme may have a recognition site of at least about six nucleotides in length. The first restriction enzyme may have a recognition site of about four nucleotides in length. The second restriction enzyme may have a recognition site of at least about five nucleotides in length.

The first and second restriction enzymes may generate ligation compatible ends. These ends may have single stranded overhangs i.e. sticky ends or be blunt. The sticky ends may match in sequence and orientation to allow ligation. The attachment step may be performed by ligation.

The sequence 5 to the ligation compatible end generated by the first restriction enzyme may be different from the sequence 5 to the ligation compatible end generated by the second restriction enzyme. This will ensure that the desired product cannot be re cut by either restriction enzyme.

The sequence 3 to the ligation compatible end generated by the first restriction enzyme may be different from the sequence 3 to the ligation compatible end generated by the second restriction enzyme. This will ensure that the desired product cannot be re cut by either restriction enzyme. Given the criteria provided throughout this specification one of ordinary skill in the art will recognize that many pairs of enzymes are suitable for use with this method.

Also provided by this disclosure is a polynucleotide fragment generated by any of the methods provided herein devices for performing the methods provided herein and systems for performing the methods provided herein.

The methods provided in this disclosure and portions thereof may also be used with each other. For example the non overlapping fragmentation methods may be used alone and or with the pseudo random fragmentation methods and or with the restriction enzyme mediated recycling methods. Likewise the pseudo random fragmentation methods may be used alone and or with the non overlapping fragmentation methods and or with the restriction enzyme mediated recycling methods. Similarly the restriction enzyme mediated recycling methods may be used alone and or with the non overlapping fragmentation methods and or with the pseudo random fragmentation methods.

Additional aspects and advantages of the present disclosure will become readily apparent to those skilled in this art from the following detailed description wherein only illustrative embodiments of the present disclosure are shown and described. As will be realized the present disclosure is capable of other and different embodiments and its several details are capable of modifications in various obvious respects all without departing from the disclosure. Accordingly the drawings and description are to be regarded as illustrative in nature and not as restrictive.

All publications patents and patent applications mentioned in this specification are herein incorporated by reference to the same extent as if each individual publication patent or patent application was specifically and individually indicated to be incorporated by reference.

While various embodiments of the invention have been shown and described herein it will be obvious to those skilled in the art that such embodiments are provided by way of example only. Numerous variations changes and substitutions may occur to those skilled in the art without departing from the invention. It should be understood that various alternatives to the embodiments of the invention described herein may be employed.

This disclosure provides methods compositions systems and devices for processing polynucleotides. Applications include processing polynucleotides for polynucleotide sequencing. Polynucleotides sequencing includes the sequencing of whole genomes detection of specific sequences such as single nucleotide polymorphisms SNPs and other mutations detection of nucleic acid e.g. deoxyribonucleic acid insertions and detection of nucleic acid deletions.

Utilization of the methods compositions systems and devices described herein may incorporate unless otherwise indicated conventional techniques of organic chemistry polymer technology microfluidics molecular biology and recombinant techniques cell biology biochemistry and immunology. Such conventional techniques include microwell construction microfluidic device construction polymer chemistry restriction digestion ligation cloning polynucleotide sequencing and polynucleotide sequence assembly. Specific non limiting illustrations of suitable techniques are described throughout this disclosure. However equivalent procedures may also be utilized. Descriptions of certain techniques may be found in standard laboratory manuals such as all from Cold Spring Harbor Laboratory Press and Oligonucleotide Synthesis A Practical Approach 1984 IRL Press London all of which are herein incorporated in their entirety by reference for all purposes.

The terminology used herein is for the purpose of describing particular embodiments only and is not intended to be limiting.

As used herein the singular forms a an and the are intended to include the plural forms as well unless the context clearly indicates otherwise. Furthermore to the extent that the terms including includes having has with such as or variants thereof are used in either the specification and or the claims such terms are not limiting and are intended to be inclusive in a manner similar to the term comprising .

The term about as used herein generally refers to a range that is 15 greater than or less than a stated numerical value within the context of the particular usage. For example about 10 would include a range from 8.5 to 11.5.

The term barcode as used herein generally refers to a label that may be attached to a polynucleotide or any variant thereof to convey information about the polynucleotide. For example a barcode may be a polynucleotide sequence attached to all fragments of a target polynucleotide contained within a particular partition. This barcode may then be sequenced with the fragments of the target polynucleotide. The presence of the same barcode on multiple sequences may provide information about the origin of the sequence. For example a barcode may indicate that the sequence came from a particular partition and or a proximal region of a genome. This may be particularly useful when several partitions are pooled before sequencing.

The term Mer as used herein to refer to restriction enzymes generally refers to the number of nucleotides in one strand of a restriction enzyme s recognition site. For example the enzyme CviQI has a recognition site of GTAC 4 nucleotides on one strand and is thus referred to as a 4Mer cutter. The enzyme StuI has a recognition site of AGGCCT 6 nucleotides on one strand and is thus referred to as a 6Mer cutter. 

The term microwell as used herein generally refers to a well with a volume of less than 1 mL. Microwells may be made in various volumes depending on the application. For example microwells may be made in a size appropriate to accommodate any of the partition volumes described herein.

The terms non overlapping and overlapping as used to refer to polynucleotide fragments generally refer to a collection of polynucleotide fragments without overlapping sequence or with overlapping sequence respectively. By way of illustration consider a hypothetical partition containing three copies of a genome top set of sequences . This genome may be fragmented randomly e.g. by shearing in a pipette or non randomly e.g. by digesting with a rare cutter . Fragmenting randomly produces overlapping sequences second set of sequences from top in Fragmented randomly to generate overlap because each copy of the genome is cut at different positions. After sequencing of the fragments which provides sequence contigs this overlap may be used to determine the linear order of the fragments thereby enabling assembly of the entire genomic sequence. By contrast fragmenting by digesting with a rare cutter produces non overlapping fragments because each copy of the same genome is cut at the same position third set of sequences from the top in Fragmented non randomly using RE 1 to generate non overlapping fragments . After sequencing these fragments it may be difficult to deduce their linear order due to the lack of overlap between the fragments. However as described in this disclosure the linear order may be determined by for example fragmenting the genome using a different technique. The fourth set of sequences from the top of demonstrates the use of a second rare cutter enzyme to generate a second set of non overlapping fragments Fragmented non randomly using RE 2 to generate non overlapping fragments . Because two different enzymes for example are used to generate the two sets of non overlapping fragments there is overlap between the fragments generated with the first rare cutter enzyme RE 1 and the fragments generated with the second rare cutter enzyme RE 2 . This overlap may then be used to assemble the linear order of the sequences and therefore the sequence of the entire genome.

The term partition as used herein may be a verb or a noun. When used as a verb e.g. partitioning the term refers to the fractionation of a substance e.g. a polynucleotide between vessels that can be used to sequester one fraction from another. Such vessels are referred to using the noun partition. Partitioning may be performed for example using microfluidics dilution dispensing and the like. A partition may be for example a well a microwell a droplet a test tube a spot or any other means of sequestering one fraction of a sample from another. In the methods and systems described herein polynucleotides are often partitioned into microwells.

The terms polynucleotide or nucleic acid as used herein are used herein to refer to biological molecules comprising a plurality of nucleotides. Exemplary polynucleotides include deoxyribonucleic acids ribonucleic acids and synthetic analogues thereof including peptide nucleic acids.

The term rare cutter enzyme as used herein generally refers to an enzyme with a recognition site that occurs only rarely in a genome. The size of restriction fragments generated by cutting a hypothetical random genome with a restriction enzyme may be approximated by 4 where N is the number of nucleotides in the recognition site of the enzyme. For example an enzyme with a recognition site consisting of 7 nucleotides would cut a genome once every 4bp producing fragments of about 16 384 bp. Generally rare cutter enzymes have recognition sites comprising 6 or more nucleotides. For example a rare cutter enzyme may have a recognition site comprising or consisting of 6 7 8 9 10 11 12 13 14 or 15 nucleotides. Examples of rare cutter enzymes include NotI GCGGCCGC XmaIII CGGCCG SstII CCGCGG SalI GTCGAC NruI TCGCGA NheI GCTAGC Nb.BbvCI CCTCAGC BbvCI CCTCAGC AscI GGCGCGCC AsiSI GCGATCGC FseI GGCCGGCC PacI TTAATTAA PmeI GTTTAAAC SbfI CCTGCAGG SgrAI CRCCGGYG SwaI ATTTAAAT BspQI GCTCTTC SapI GCTCTTC SfiI GGCC GGCC SEQ ID NO 1 CspCI CAA GTGG SEQ ID NO 2 AbsI CCTCGAGG CciNI GCGGCCGC FspAI RTGCGCAY MauBI CGCGCGCG MreI CGCCGGCG MssI GTTTAAAC PalAI GGCGCGCC RgaI GCGATCGC RigI GGCCGGCC SdaI CCTGCAGG SfaAI GCGATCGC SgfI GCGATCGC SgrDI CGTCGACG SgsI GGCGCGCC SmiI ATTTAAAT SrfI GCCCGGGC Sse2321 CGCCGGCG Sse83871 CCTGCAGG LguI GCTCTTC PciSI GCTCTTC AarI CACCTGC AjuI GAA TTGG SEQ ID NO 3 AloI GAAC TCC SEQ ID NO 4 Bad GAAG TAC SEQ ID NO 5 PpiI GAAC CTC SEQ ID NO 6 PsrI GAAC TAC SEQ ID NO 7 and others.

The term target polynucleotide as used herein generally refers to a polynucleotide to be processed. For example if a user intends to process genomic DNA into fragments that may be sequenced the genomic DNA would be the target polynucleotide. If a user intends to process fragments of a polynucleotide then the fragments of the polynucleotide may be the target polynucleotide.

This disclosure provides methods compositions systems and devices for the generation of non overlapping polynucleotide fragments. These fragments may be useful for downstream analyses such as DNA sequencing. For example with reference to a target polynucleotide such as genomic DNA may be fragmented to generate a plurality of non overlapping first polynucleotide fragments . This fragmentation may be performed for example by digesting the target polynucleotide with a rare cutter enzyme e.g. rare cutter enzyme 1 or an artificial restriction DNA cutter ARCUT Yamamoto et al. 2007 35 7 e53 . The first polynucleotide fragments may then be partitioned such that at least one partition comprises a first polynucleotide fragment with a unique sequence within that partition and optionally an additional first polynucleotide fragment with a different sequence . The partitioned first polynucleotide fragments may then be further fragmented to produce a plurality of non overlapping second polynucleotide fragments . This fragmentation may be performed for example by enzymatic digestion exposure to ultraviolet UV light ultrasonication and or mechanical agitation. The second polynucleotide fragments may be of a size that is appropriate for DNA sequencing i.e. a size that enables a DNA sequencer to obtain accurate sequence data for the entire fragment.

In order to facilitate DNA sequence assembly the second fragments may be attached to a barcode which may be attached to all of the second fragments disposed in a particular partition. The barcode may be for example a DNA barcode. With continued reference to after attachment of the barcode the barcoded fragments may be pooled into a partition comprising pooled barcoded sequences . Three barcodes are depicted as 1 2 and 3 in . The pooled fragments may be sequenced.

Certain methods of genome sequence assembly rely on the presence of overlapping fragments in order to generate higher order sequence data e.g. whole genome sequences from sequenced fragments. The methods compositions systems and devices provided herein may also be used to provide overlapping fragments. For example with continued reference to fragments overlapping with the first and second fragments described above may be generated by generating a plurality of non overlapping third polynucleotide fragments from the target polynucleotide . The third polynucleotide fragments may be generated for example by digesting the target polynucleotide with a rare cutter enzyme e.g. rare cutter enzyme 2 or ARCUT that is different from the rare cutter enzyme used to generate the first polynucleotide fragments. If rare cutter enzymes 1 and 2 are chosen to cut the target polynucleotide sequence at different positions the third polynucleotide fragments and the first polynucleotide fragments will overlap. The third polynucleotide fragments may then be processed as described above for the first polynucleotide fragments.

Specifically the third polynucleotide fragments may be partitioned such that at least one partition comprises a third polynucleotide fragment with a unique sequence within that partition and optionally an additional third polynucleotide fragment with a different sequence . These partitioned fragments may then be further fragmented to produce a plurality of non overlapping fourth polynucleotide fragments . The fourth polynucleotides fragments and the second polynucleotide fragments may overlap. As for the second polynucleotide fragments the fourth polynucleotide fragments may be generated by for example enzymatic digestion exposure to ultraviolet UV light ultrasonication and or mechanical agitation. The fourth fragments may be of a size that is appropriate for DNA sequencing i.e. a size that enables a DNA sequencer to obtain accurate sequence data for the entire fragment.

In order to facilitate DNA sequencing the fourth fragments may be attached to a barcode which may be attached to all of the fourth fragments disposed in a particular partition. The barcode may be for example a DNA barcode. After attachment of the barcode the barcoded fragments may be pooled into a partition comprising pooled barcoded sequences . Three barcodes are depicted as 4 5 and 6 in . The pooled fragments may be sequenced. The overlap between the sequences of the second fragments and the fourth fragments may be used to assemble higher order sequences such whole genome sequences.

The steps described above may be performed using a variety of techniques. For example certain steps of the methods may be performed in a device comprising microwell chambers microwells for example a microfluidic device. These microwells may be connected to each other or to a source of reagents by channels. The first and third fragments may be generated outside of the device and then introduced into the device or separate devices for further processing. Partitioning of the first and third fragments may accomplished using fluidic techniques. Generation of the second and fourth fragments may then occur within the microwells of the device or devices. These microwells may contain reagents for barcoding of the second and fourth fragments such as DNA barcodes ligase adapter sequences and the like. Microwells may feed or be directed into a common outlet so that barcoded fragments may be pooled or otherwise collected into one or more aliquots which may then be sequenced.

In another example the entire process could be performed within a single device. For example a device could be split into two sections. A first section may comprise a partition comprising rare cutter enzyme 1 generating first polynucleotide fragments and a second section may comprise a partition comprising rare cutter enzyme 2 generating third polynucleotide fragments . An aliquot of the target polynucleotide sequence may be placed into each of these partitions. Following digestion the enzyme may be inactivated and the samples may be partitioned fragmented barcoded pooled and sequenced as described above. For convenience this example has been described using rare cutter enzymes as the means of generating the first and third fragments. However this is not intended to be limiting here or anywhere else in this disclosure. One of ordinary skill in the art will readily recognize that other means of generating non overlapping or predominantly non overlapping fragments would be just as suitable as the use of rare cutter enzymes.

This disclosure also provides methods compositions systems and devices for fragmenting polynucleotides in a pseudo random manner. This may be performed by treating partitioned polynucleotides with more than one restriction enzyme. For example polynucleotides partitioned into microwells may be treated with combinations of restriction enzymes. Within each partition containing a particular combination of enzymes the cutting is defined and predictable. However across all of the partitions through the use of multiple combinations of restriction enzymes in different partitions the polynucleotide fragments generated approximate those obtained from methods of random fragmentation. However these polynucleotide fragments are generated in a much more controlled manner than random fragments generated by methods known in the art e.g. shearing . The partitioned pseudo randomly fragmented polynucleotides may be barcoded as described throughout this disclosure pooled and sequenced. The pseudo random fragmentation methods may be used with the non overlapping fragmentation methods described herein or with any other method described herein such as the high yield adapter barcode attachment method. Pseudo random fragmentation may occur by exposing a polynucleotide to multiple enzymes simultaneously sequentially or simultaneously and sequentially.

Thus this disclosure provides methods and systems for processing polynucleotides comprising generating pseudo random fragments of said polynucleotides. These pseudo random fragments are generated by treating a polynucleotide with more than one restriction enzyme. For example a polynucleotide may be treated with about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 50 or more restriction enzymes. A polynucleotide may be treated with at least 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 50 or more restriction enzymes. A polynucleotide may be treated with at least 2 but fewer than 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes. A polynucleotide may be treated with about 2 4 4 6 6 8 8 10 10 12 12 14 14 16 16 18 18 20 20 25 25 30 35 40 40 45 or 45 50 restriction enzymes.

The restriction enzymes may be chosen in order to maximize the number or fraction of fragments that will provide accurate sequencing data based on the size of the fragments generated by the pseudo random fragmentation. For present day sequencing technology accuracy degrades beyond a read length of about 100 nucleotides. Therefore fragments of about 200 or fewer nucleotides generally provide the most accurate sequence data since they can be sequenced from either end. Fragments below about 50 nucleotides are generally less desirable because although the produce accurate sequencing data they underutilize the read length capacity of current sequencing instruments which are capable of 150 to 200 base reads. Fragments of about 200 to about 400 nucleotides may be sequenced with systematic errors introduced as the read length increases beyond the initial 100 bases from each end. Sequence information from fragments greater than about 400 nucleotides is typically completely lost for those bases greater than 200 bases from either end. One of skill in the art will recognize that sequencing technology is constantly advancing and that the ability to obtain accurate sequence information from longer fragments is also constantly improving. Thus the pseudo random fragmentation methods presented herein may be used to produce optimal fragment lengths for any sequencing method.

In some cases fragments may be defined by the distance of their component nucleotides from a restriction site measured in nucleotides . For example each nucleotide within a polynucleotide fragment generated by the pseudo random fragmentation method may be less than about 10 50 75 100 125 150 175 200 250 300 350 400 550 600 1000 5000 10000 or 100000 nucleotides from the restriction site of an enzyme to which the polynucleotide is exposed. Each nucleotide within a polynucleotide fragment may be about 10 50 75 100 125 150 175 200 250 300 350 400 550 600 1000 5000 10000 or 100000 nucleotides from the restriction site of an enzyme to which the polynucleotide is exposed. Each nucleotide within a polynucleotide fragment may be at least about 10 50 75 100 125 150 175 200 250 300 350 400 550 600 1000 5000 10000 or 100000 nucleotides from the restriction site of an enzyme to which the polynucleotide is exposed.

In some cases at least about 5 10 15 20 25 30 35 40 45 50 55 60 65 70 75 80 85 90 or 95 of the nucleotides comprising a target polynucleotide sequence are within about 10 50 75 100 125 150 175 200 250 300 350 400 550 600 1000 5000 10000 or 100000 nucleotides from the restriction site of an enzyme to which the polynucleotide is exposed. All combinations of these percentages and polynucleotide lengths are contemplated.

In some cases at less than about 1 5 10 25 30 35 40 45 or 50 of the nucleotides comprising a target polynucleotide sequence are within about 1 5 10 50 200 250 300 350 400 550 600 1000 5000 10000 or 100000 nucleotides from the restriction site of an enzyme to which the polynucleotide is exposed. All combinations of these percentages and polynucleotide lengths are contemplated.

The pseudo random fragmentation methods may be used to obtain fragments of about 10 to 50 nucleotides 46 to 210 nucleotides 50 to 250 nucleotides 250 to 400 nucleotides 400 to 550 nucleotides 550 to 700 nucleotides 700 to 1000 nucleotides 1000 to 1300 nucleotides 1300 to 1600 nucleotides 1600 to 1900 nucleotides 1900 to 2200 nucleotides or 2200 to 3000 nucleotides. The pseudo random fragmentation methods may be used to obtain fragments with a mean or median of about 40 nucleotides 60 nucleotides 80 nucleotides 100 nucleotides 120 nucleotides 130 nucleotides 140 nucleotides 160 nucleotides 180 nucleotides 200 nucleotides 250 nucleotides 300 nucleotides 400 nucleotides 500 nucleotides 600 nucleotides 700 nucleotides 800 nucleotides 900 nucleotides 1000 nucleotides 1200 nucleotides 1400 nucleotides 1600 nucleotides 1800 nucleotides 2000 nucleotides 2500 nucleotides 3000 nucleotides or more. The pseudo random fragmentation methods may be used to obtain fragments with a mean or median of at least about 40 nucleotides 60 nucleotides 80 nucleotides 100 nucleotides 120 nucleotides 130 nucleotides 140 nucleotides 160 nucleotides 180 nucleotides 200 nucleotides 250 nucleotides 300 nucleotides 400 nucleotides 500 nucleotides 600 nucleotides 700 nucleotides 800 nucleotides 900 nucleotides 1000 nucleotides 1200 nucleotides 1400 nucleotides 1600 nucleotides 1800 nucleotides 2000 nucleotides 2500 nucleotides 3000 nucleotides or more. The pseudo random fragmentation methods may be used to obtain fragments with a mean or median of less than about 40 nucleotides 60 nucleotides 80 nucleotides 100 nucleotides 120 nucleotides 130 nucleotides 140 nucleotides 160 nucleotides 180 nucleotides 200 nucleotides 250 nucleotides 300 nucleotides 400 nucleotides 500 nucleotides 600 nucleotides 700 nucleotides 800 nucleotides 900 nucleotides 1000 nucleotides 1200 nucleotides 1400 nucleotides 1600 nucleotides 1800 nucleotides 2000 nucleotides 2500 nucleotides or 3000 nucleotides.

In some examples the pseudo random fragmentation methods provided herein are used to generate fragments wherein a particular percentage or fraction of the fragments generated fall within any of the size ranges described herein. For example about 0 2 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 52 54 56 58 60 62 64 66 68 70 72 74 76 78 80 82 84 86 88 90 92 94 96 98 or 100 of the fragments generated may fall within any of the size ranges described herein.

In some examples multiple 4Mer cutters may be used to provide a distribution of about 18 of fragments of about 50 nucleotides or less about 38 of fragments of about 200 nucleotides or less about 25 of fragments between about 200 and about 400 nucleotides and about 37 of fragments greater than about 400 nucleotides e.g. see .

Additionally the pseudo random fragmentation method may be designed to minimize the percentage of fragments greater than a certain number of nucleotides in length in order to minimize the loss of sequence information. For example the method may be designed to yield less than about 0.1 0.5 1 2 5 10 20 or 50 fragments greater than 100 nucleotides. The method may be designed to yield less than about 0.1 0.5 1 2 5 10 20 or 50 fragments greater than 150 nucleotides. The method may be designed to yield less than about 0.1 0.5 1 2 5 10 20 or 50 fragments greater than 200 nucleotides. The method may be designed to yield less than about 0.1 0.5 1 2 5 10 20 or 50 fragments greater than 300 nucleotides and so on. As the ability of sequencing technologies to accurately read long DNA fragments increases the pseudo random fragmentation methods of the invention may be used to generate sequences suitable for any chosen read length.

Enzymes for use with the pseudo random fragmentation method described herein may be chosen for example based on the length of their recognition site and their compatibility with certain buffer conditions to allow for combination with other enzymes . Enzymes may also be chosen so that their cutting activity is methylation insensitive or sensitive to methylation. For example restriction enzymes with shorter recognition sites generally cut polynucleotides more frequently. Thus cutting a target polynucleotide with a 6Mer cutter will generally produce more large fragments than cutting the same polynucleotide with a 4Mer cutter e.g. compare . Cutting a target polynucleotide with a plurality of enzymes e.g. 2 3 4 5 6 7 or more may produce a greater number or fraction of fragments in the optimal size range for DNA sequencing than cutting with a single enzyme see . Any restriction enzyme may be used with this method. Many are named in this specification but others are known in the art.

This disclosure also provides methods of selecting a plurality of enzymes for pseudo random fragmentation of a polynucleotide sequence. For example a target polynucleotide may be exposed separately to 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 or 20 restriction enzymes. The size distribution of the target polynucleotide fragments is then determined for example by electrophoresis. The combination of enzymes providing the greatest number of fragments that are capable of being sequenced can then be chosen. The method can also be carried out in silico.

The enzymes may be disposed within the same partition or within a plurality of partitions. For example any of the plurality of enzyme number described herein may be disposed within a single partition or across partitions. For example a polynucleotide may be treated with about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 50 or more restriction enzymes in the same partition or across partitions. A polynucleotide may be treated with at least 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 50 or more restriction enzymes in the same partition or across partitions. A polynucleotide may be treated with at least 2 but fewer than 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes in the same partition or across partitions. A polynucleotide may be treated with about 2 4 4 6 6 8 8 10 10 12 12 14 14 16 16 18 18 20 20 25 25 30 35 40 40 45 or 45 50 restriction enzymes in the same partition or across partitions.

The distribution of the restriction enzymes among the partitions will vary depending on the restriction enzymes the target polynucleotide and the desired fragment size. In some cases each restriction enzyme may be distributed across an equivalent number of partitions so that the number of partitions occupied by each restriction enzyme is equivalent. For example if 10 restriction enzymes are used in a device containing 1 000 partitions each enzyme may be present in 100 partitions. In other cases each restriction enzyme may be distributed across a non equivalent number of partitions so that the number of partitions occupied by each restriction enzyme is not equivalent. For example if 10 restriction enzymes are used in a device containing 1 000 partitions enzymes 1 8 may be present in 100 partitions each enzyme 9 may be present in 50 partitions and enzyme 10 may be present in 150 partitions. Placement of restriction enzymes in an unequal number of partitions may be beneficial for example when an enzyme generates a desired product at a low yield. Placing this low yield enzyme in more partitions will therefore expose more of the target polynucleotide to the enzyme increasing the amount of the desired product e.g. fragment of a certain size or composition that can be formed from the enzyme. Such an approach may be useful for accessing portions of a target polynucleotide e.g. a genome that are not cut by enzymes producing polynucleotide fragments at a higher yield. The restriction site and efficiency of an enzyme composition of the target polynucleotide and efficiency and side products generated by the enzyme may all be among the factors considered when determining how many partitions should receive a particular enzyme.

In some cases different numbers of restriction enzymes may be used in a single partition and across all partitions. For example 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used in each partition while 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used across all partitions. All combinations of these numbers are included within the invention. Non limiting specific examples include the use of 1 restriction enzyme per partition and 2 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions 2 restriction enzymes per partition and 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions 3 restriction enzymes per partition and 4 5 6 7 8 9 or 10 restriction enzymes across all partitions 4 restriction enzymes per partition and 5 6 7 8 9 or 10 restriction enzymes across all partitions 5 restriction enzymes per partition and 6 7 8 9 or 10 restriction enzymes across all partitions 6 restriction enzymes per partition and 7 8 9 or 10 restriction enzymes across all partitions 7 restriction enzymes per partition and 8 9 or 10 restriction enzymes across all partitions 8 restriction enzymes per partition and 9 or 10 restriction enzymes across all partitions and 9 restriction enzymes per partition and 10 or more restriction enzymes across all partitions.

In some cases at least 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used in each partition while at least 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used across all partitions. All combinations of these numbers are included within the invention. Non limiting specific examples include the use of at least 1 restriction enzyme per partition and at least 2 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at least 2 restriction enzymes per partition and at least 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at least 3 restriction enzymes per partition and at least 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at least 4 restriction enzymes per partition and at least 5 6 7 8 9 or 10 restriction enzymes across all partitions at least 5 restriction enzymes per partition and at least 6 7 8 9 or 10 restriction enzymes across all partitions at least 6 restriction enzymes per partition and at least 7 8 9 or 10 restriction enzymes across all partitions at least 7 restriction enzymes per partition and at least 8 9 or 10 restriction enzymes across all partitions at least 8 restriction enzymes per partition and at least 9 or 10 restriction enzymes across all partitions and at least 9 restriction enzymes per partition and at least 10 or more restriction enzymes across all partitions.

In some cases at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used in each partition while at most 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used across all partitions. All combinations of these numbers are included within the invention. Non limiting specific examples include the use of at most 1 restriction enzyme per partition and at most 2 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at most 2 restriction enzymes per partition and at most 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at most 3 restriction enzymes per partition and at most 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at most 4 restriction enzymes per partition and at most 5 6 7 8 9 or 10 restriction enzymes across all partitions at most 5 restriction enzymes per partition and at most 6 7 8 9 or 10 restriction enzymes across all partitions at most 6 restriction enzymes per partition and at most 7 8 9 or 10 restriction enzymes across all partitions at most 7 restriction enzymes per partition and at most 8 9 or 10 restriction enzymes across all partitions at most 8 restriction enzymes per partition and at most 9 or 10 restriction enzymes across all partitions and at most 9 restriction enzymes per partition and at most 10 or more restriction enzymes across all partitions.

In some cases at least 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used in each partition while at most 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used across all partitions. All combinations of these numbers are included within the invention. Non limiting specific examples include the use of at least 1 restriction enzyme per partition and at most 2 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at least 2 restriction enzymes per partition and at most 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at least 3 restriction enzymes per partition and at most 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at least 4 restriction enzymes per partition and at most 5 6 7 8 9 or 10 restriction enzymes across all partitions at least 5 restriction enzymes per partition and at most 6 7 8 9 or 10 restriction enzymes across all partitions at least 6 restriction enzymes per partition and at most 7 8 9 or 10 restriction enzymes across all partitions at least 7 restriction enzymes per partition and at most 8 9 or 10 restriction enzymes across all partitions at least 8 restriction enzymes per partition and at most 9 or 10 restriction enzymes across all partitions and at least 9 restriction enzymes per partition and at most 10 or more restriction enzymes across all partitions.

In some cases at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used in each partition while at least 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 25 30 35 45 45 or 50 restriction enzymes or more may be used across all partitions. All combinations of these numbers are included within the invention. Non limiting specific examples include the use of at most 1 restriction enzyme per partition and at least 2 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at most 2 restriction enzymes per partition and at least 3 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at most 3 restriction enzymes per partition and at least 4 5 6 7 8 9 or 10 restriction enzymes across all partitions at most 4 restriction enzymes per partition and at least 5 6 7 8 9 or 10 restriction enzymes across all partitions at most 5 restriction enzymes per partition and at least 6 7 8 9 or 10 restriction enzymes across all partitions at most 6 restriction enzymes per partition and at least 7 8 9 or 10 restriction enzymes across all partitions at most 7 restriction enzymes per partition and at least 8 9 or 10 restriction enzymes across all partitions at most 8 restriction enzymes per partition and at least 9 or 10 restriction enzymes across all partitions and at most 9 restriction enzymes per partition and at least 10 or more restriction enzymes across all partitions.

As described throughout this disclosure certain methods of the invention involve the addition of barcodes adapters or other sequences to fragmented target polynucleotides. Barcodes may be polynucleotide barcodes which may be ligated to the fragmented target polynucleotides or added via an amplification reaction. As described throughout this disclosure fragmentation of target polynucleotides may be performed using one or more restriction enzymes contained within a partition e.g. a microwell where the fragmentation is performed. The partition may also contain a polynucleotide barcode and a ligase which enables the attachment of the barcode to the fragmented polynucleotide. In some cases an adapter may be used to make a fragmented target polynucleotide compatible for ligation with a barcode. The presence of adapters fragmented target polynucleotide barcodes restriction enzymes and ligases in the same partition may lead to the generation of undesirable side products that decrease the yield of a desired end product. For example self ligation may occur between adapters target polynucleotide fragments and or barcodes. These self ligations reduce the amount of starting material and decrease the yield of the desired product for example a polynucleotide fragment properly ligated to a barcode and or and adapter.

This disclosure provides methods compositions systems and devices for addressing this problem and increasing the yield of a desired product. The problem is addressed by pairing a first restriction enzyme and a second restriction enzyme. The two restriction enzymes create compatible termini upon cutting but each enzyme has a different recognition sequence.

Ligation of two pieces of DNA generated after cutting with the first restriction enzyme will regenerate the recognition site for the first restriction enzyme allowing the first restriction enzyme to re cut the ligated DNA. Likewise ligation of two pieces of DNA generated after cutting with the second restriction enzyme will regenerate the recognition site for the second restriction enzyme allowing the second restriction enzyme to re cut the ligated DNA. However ligation of one piece of DNA generated after cutting with the first restriction enzyme and one piece of DNA generated after cutting with the second restriction enzyme will result in ligated DNA that is unrecognizable and therefore uncuttable by both the first and second enzymes. The result is that any multimers of fragmented target polynucleotides are re cut and any multimers of adapter or other molecules e.g. barcodes are also re cut. However when a fragmented target polynucleotide is properly ligated to an adapter or barcode the restriction sites for both enzymes are not present and the correctly ligated molecule may not be re cut by either enzyme.

An example of this method is illustrated in and additional pairs of enzymes that may be used with the method are provided in . Any pair of enzymes may be used so long as they meet the following criteria 1 the enzymes should create identical or at least similar ligatable termini upon cutting and 2 the enzymes should have different recognition sequences. The enzymes may be selected to avoid or minimize cutting of certain polynucleotide sequences such as barcodes adapters and other polynucleotide components of a sample processing or preparation platform. The enzymes may be selected for methylation insensitivity or methylation sensitivity. The enzymes may also be selected to be active under s single set of environmental conditions such as buffer conditions temperature etc. Minimizing the cutting of barcodes and adapters may be accomplished by pairing certain enzymes with certain barcodes and or adapters.

This method may be used to increase the yield of any of the barcoding methods described herein. The regeneration of the starting materials e.g. fragmented target polynucleotide adapters and barcodes allows these starting materials another opportunity to form the desired products i.e. fragmented target polynucleotides ligated to barcodes optionally with adapters . This greatly increases the yield of the reaction and therefore decreases the amount of starting material required to produce the necessary amount of the desired products while limiting the amount of undesirable side products and lost sequence information.

The methods described above may be used to achieve about 75 85 95 96 97 98 99 or 99.5 yield w w . The methods may be used to achieve at least about 75 85 95 96 97 98 99 or 99.5 yield w w .

The methods described above may use for example a pair of restriction enzyme selected from the group consisting of MspI NarI BfaI NarI BfaI NdeI HinPII ClaI MseI NdeI CviQI NdeI Taq I AcII RsaI PmeI AluI EcoRV BstUI PmeI DpnI StuI HaeIII PmeI and HpyCH4V SfoI. This list of enzymes is provided for purposes of illustration only and is not meant to be limiting.

The methods described above may generally use any two enzymes that create ligatable termini upon cutting but that have different recognition sequences. However the method is not limited to ligation. For example multimers formed after amplification of side products formed by association of compatible ends could also be re cut using the methods described above.

More than one pair of enzymes may also be used. The number of pairs of enzymes chosen will vary depending on the number of undesirable side products formed in a reaction. For example 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 or more pairs of enzymes may be used. Treatment of a polynucleotide with the enzymes may be sequential simultaneous or both.

Target polynucleotides processed according to the methods provided in this disclosure may be DNA RNA peptide nucleic acids and any hybrid thereof where the polynucleotide contains any combination of deoxyribo and ribo nucleotides. Polynucleotides may be single stranded or double stranded as specified or contain portions of both double stranded or single stranded sequence. Polynucleotides may contain any combination of nucleotides including uracil adenine thymine cytosine guanine inosine xanthine hypoxanthine isocytosine isoguanine and any nucleotide derivative thereof. As used herein the term nucleotide may include nucleotides and nucleosides as well as nucleoside and nucleotide analogs and modified nucleotides including both synthetic and naturally occurring species. Target polynucleotides may be cDNA mitochondrial DNA mtDNA messenger RNA mRNA ribosomal RNA rRNA transfer RNA tRNA nuclear RNA nRNA small interfering RNA siRNA small nuclear RNA snRNA small nucleolar RNA snoRNA small Cajal body specific RNA scaRNA microRNA miRNA double stranded dsRNA ribozyme riboswitch or viral RNA. Target polynucleotides may be contained on a plasmid cosmid or chromosome and may be part of a genome. In some cases a target polynucleotide may comprise one or more genes and or one or more pseudogenes. A pseudogene generally refers to a dysfunctional relative of a gene that has lost its protein coding ability and or is otherwise no longer expressed in the cell.

Target polynucleotides may be obtained from a sample using any methods known in the art. A target polynucleotide processed as described herein may be obtained from whole cells cell preparations and cell free compositions from any organism tissue cell or environment. In some instances target polynucleotides may be obtained from bodily fluids which may include blood urine serum lymph saliva mucosal secretions perspiration or semen. In some instances polynucleotides may be obtained from environmental samples including air agricultural products water and soil. In other instances polynucleotides may be the products of experimental manipulation including recombinant cloning polynucleotide amplification as generally described in PCT US99 01705 polymerase chain reaction PCR amplification purification methods such as purification of genomic DNA or RNA and synthesis reactions.

Genomic DNA may be obtained from naturally occurring or genetically modified organisms or from artificially or synthetically created genomes. Target polynucleotides comprising genomic DNA may be obtained from any source and using any methods known in the art. For example genomic DNA may be isolated with or without amplification. Amplification may include PCR amplification multiple displacement amplification MDA rolling circle amplification and other amplification methods. Genomic DNA may also be obtained by cloning or recombinant methods such as those involving plasmids and artificial chromosomes or other conventional methods see Sambrook and Russell . cited supra. Polynucleotides may be isolated using other methods known in the art for example as disclosed in Vols. I IV or . If the isolated polynucleotide is an mRNA it may be reverse transcribed into cDNA using conventional techniques as described in Sambrook and Russell . cited supra.

Target polynucleotides may also be isolated from target organisms or target cells . The terms target organism and target cell refer to an organism or cell respectively from which target polynucleotides may be obtained. Target cells may be obtained from a variety of organisms including human mammal non human mammal ape monkey chimpanzee plant reptilian amphibian avian fungal viral or bacterial organisms. Target cells may also be obtained from a variety of clinical sources such as biopsies aspirates blood urine formalin fixed embedded tissues and the like. Target cells may comprise a specific cell type such as a somatic cell germline cell wild type cell cancer or tumor cells or diseased or infected cell. A target cell may refer to a cell derived from a particular tissue or a particular locus in a target organism. A target cell may comprise whole intact cells or cell preparations.

Target polynucleotides may also be obtained or provided in specified quantities. Amplification may be used to increase the quantity of a target polynucleotide. Target polynucleotides may quantified by mass. For example target polynucleotides may be provided in a mass ranging from about 1 10 10 50 50 100 100 200 200 1000 1000 10000 ng. Target polynucleotides may be provided in a mass of at least about 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 50 100 500 1000 1500 2000 2500 3000 3500 4000 4500 5000 5500 6000 6500 7000 7500 8000 8500 9000 9500 or 10000 ng. Target polynucleotides may be provided in a mass of less than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 50 100 500 1000 1500 2000 2500 3000 3500 4000 4500 5000 5500 6000 6500 7000 7500 8000 8500 9000 9500 or 10000 ng.

Target polynucleotides may also be quantified as genome equivalents. A genome equivalent is an amount of polynucleotide equivalent to one haploid genome of an organism from which the target polynucleotide is derived. For example a single diploid cell contains two genome equivalents of DNA. Target polynucleotides may be provided in an amount ranging from about 1 10 10 50 50 100 100 1000 1000 10000 10000 100000 or 100000 1000000 genome equivalents. Target polynucleotides may be provided in an amount of at least about 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 50 100 500 1000 1500 2000 2500 3000 3500 4000 4500 5000 5500 6000 6500 7000 7500 8000 8500 9000 9500 10000 20000 30000 40000 50000 60000 70000 80000 90000 100000 200000 300000 400000 500000 600000 700000 800000 900000 or 1000000 genome equivalents. Target polynucleotides may be provided in an amount less than about 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 50 100 500 1000 1500 2000 2500 3000 3500 4000 4500 5000 5500 6000 6500 7000 7500 8000 8500 9000 9500 10000 20000 30000 40000 50000 60000 70000 80000 90000 100000 200000 300000 400000 500000 600000 700000 800000 900000 or 1000000 genome equivalents.

Target polynucleotide may also be quantified by the amount of sequence coverage provided. The amount of sequence coverage refers to the average number of reads representing a given nucleotide in a reconstructed sequence. Generally the greater the number of times a region is sequenced the more accurate the sequence information obtained. Target polynucleotides may be provided in an amount that provides a range of sequence coverage from about 0.1 10 10 50 50 100 100 200 or 200 500 . Target polynucleotide may be provided in an amount that provides at least about 0.1 0.2 0.3 0.4 0.5 0.6 0.7 0.8 0.9 1.0 5 10 25 50 100 125 150 175 or 200 sequence coverage. Target polynucleotide may be provided in an amount that provides less than about 0.2 0.3 0.4 0.5 0.6 0.7 0.8 0.9 1.0 5 10 25 50 100 125 150 175 or 200 sequence coverage.

Fragmentation of polynucleotides is used as a step in a variety of processing methods described herein. The size of the polynucleotide fragments typically described in terms of length quantified by the linear number of nucleotides per fragment may vary depending on the source of the target polynucleotide the method used for fragmentation and the desired application. Moreover while certain methods of the invention are illustrated using a certain number of fragmentation steps the number of fragmentation steps provided is not meant to be limiting and any number of fragmentation steps may be used. For example 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 or more fragmentation steps may be used.

Fragments generated using the methods described herein may be about 1 10 10 20 20 50 50 100 50 200 100 200 200 300 300 400 400 500 500 1000 1000 5000 5000 10000 10000 100000 100000 250000 or 250000 500000 nucleotides in length. Fragments generated using the methods described herein may be at least about 10 20 100 200 300 400 500 1000 5000 10000 100000 250000 500000 or more nucleotides in length. Fragments generated using the methods described herein may be less than about 10 20 100 200 300 400 500 1000 5000 10000 100000 250000 500000 nucleotides in length.

Fragments generated using the methods described herein may have a mean or median length of about 1 10 10 20 20 50 50 100 50 200 100 200 200 300 300 400 400 500 500 1000 1000 5000 5000 10000 10000 100000 100000 250000 or 250000 500000 nucleotides. Fragments generated using the methods described herein may have a mean or median length of at least about 10 20 100 200 300 400 500 1000 5000 10000 100000 250000 500000 or more nucleotides. Fragments generated using the methods described herein may have a mean or median length of less than about 10 20 100 200 300 400 500 1000 5000 10000 100000 250000 500000 nucleotides.

Numerous fragmentation methods are described herein and known in the art. For example fragmentation may be performed through physical mechanical or enzymatic methods. Physical fragmentation may include exposing a target polynucleotide to heat or to UV light. Mechanical disruption may be used to mechanically shear a target polynucleotide into fragments of the desired range. Mechanical shearing may be accomplished through a number of methods known in the art including repetitive pipetting of the target polynucleotide sonication and nebulization. Target polynucleotides may also be fragmented using enzymatic methods. In some cases enzymatic digestion may be performed using enzymes such as using restriction enzymes.

While the methods of fragmentation described in the preceding paragraph and in some paragraphs of the disclosure are described with reference to target polynucleotides this is not meant to be limiting above or anywhere else in this disclosure. Any means of fragmentation described herein or known in the art can be applied to any polynucleotide used with the invention. In some cases this polynucleotide may be a target polynucleotide such as a genome. In other cases this polynucleotide may be a fragment of a target polynucleotide which one wishes to further fragment. In still other cases still further fragments may be still further fragmented. Any suitable polynucleotide may be fragmented according the methods described herein.

A fragment of a polynucleotide generally comprises a portion of the sequence of the targeted polynucleotide from which the fragment was generated. In some cases a fragment may comprise a copy of a gene and or pseudogene including one included in the original target polynucleotide. In some cases a plurality of fragments generated from fragmenting a target polynucleotide may comprise fragments that each comprise a copy of a gene and or pseudogene.

Restriction enzymes may be used to perform specific or non specific fragmentation of target polynucleotides. The methods of the present disclosure may use one or more types of restriction enzymes generally described as Type I enzymes Type II enzymes and or Type III enzymes. Type II and Type III enzymes are generally commercially available and well known in the art. Type II and Type III enzymes recognize specific sequences of nucleotide base pairs within a double stranded polynucleotide sequence a recognition sequence or recognition site . Upon binding and recognition of these sequences Type II and Type III enzymes cleave the polynucleotide sequence. In some cases cleavage will result in a polynucleotide fragment with a portion of overhanging single stranded DNA called a sticky end. In other cases cleavage will not result in a fragment with an overhang creating a blunt end. The methods of the present disclosure may comprise use of restriction enzymes that generate either sticky ends or blunt ends.

Restriction enzymes may recognize a variety of recognition sites in the target polynucleotide. Some restriction enzymes exact cutters recognize only a single recognition site e.g. GAATTC . Other restriction enzymes are more promiscuous and recognize more than one recognition site or a variety of recognition sites. Some enzymes cut at a single position within the recognition site while others may cut at multiple positions. Some enzymes cut at the same position within the recognition site while others cut at variable positions.

The present disclosure provides method of selecting one or more restriction enzymes to produce fragments of a desired length. Polynucleotide fragmentation may be simulated in silico and the fragmentation may be optimized to obtain the greatest number or fraction of polynucleotide fragments within a particular size range while minimizing the number or fraction of fragments within undesirable size ranges. Optimization algorithms may be applied to select a combination of two or more enzymes to produce the desired fragment sizes with the desired distribution of fragments quantities.

A polynucleotide may be exposed to two or more restriction enzymes simultaneously or sequentially. This may be accomplished by for example adding more than one restriction enzyme to a partition or by adding one restriction enzyme to a partition performing the digestion deactivating the restriction enzyme e.g. by heat treatment and then adding a second restriction enzyme. Any suitable restriction enzyme may be used alone or in combination in the methods presented herein.

Fragmenting of a target polynucleotide may occur prior to partitioning of the target polynucleotide or fragments generated from fragmenting. For example genomic DNA gDNA may be fragmented using for example a restriction enzyme prior to the partitioning of its generated fragments. In another example a target polynucleotide may be entered into a partition along with reagents necessary for fragmentation e.g. including a restriction enzyme such that fragmentation of the target polynucleotide occurs within the partition. For example gDNA may be fragmented in a partition comprising a restriction enzyme and the restriction enzyme is used to fragment the gDNA.

In some cases a plurality of fragments may be generated prior to partitioning using any method for fragmentation described herein. Some or all of the fragments of the plurality for example may each comprise a copy of a gene and or a pseudogene. The fragments can be separated and partitioned such that each copy of the gene or pseudogene is located in a different partition. Each partition for example can comprise a different barcode sequence such that each copy of the gene and or pseudogene can be associated with a different barcode sequence using barcoding methods described elsewhere herein. Via the different barcode sequences each gene and or pseudogene can be counted and or differentiated during sequencing of the barcoded fragments. Any sequencing method may be used including those described herein.

For example using restriction enzymes genomic DNA gDNA can be fragmented to generate a plurality of non overlapping fragments of the gDNA. At least some of the fragments of the plurality may each comprise a copy of a gene and or a pseudogene. The fragments may be separated and partitioned such that each copy of the gene or pseudogene is located in a different partition. Each partition for example can comprise a different barcode sequence such that each copy of the gene and or pseudogene may be barcoded with a different barcode sequence. Via the different barcode sequences the genes and or pseudogenes may be counted and or differentiated after sequencing of the barcoded fragments. Any sequencing method may be used including those described herein.

As described throughout the disclosure certain methods systems and compositions of the disclosure may utilize partitioning of polynucleotides into separate partitions e.g. microwells droplets of an emulsion . These partitions may be used to contain polynucleotides for further processing such as for example cutting ligating and or barcoding.

Any number of devices systems or containers may be used to hold support or contain partitions of polynucleotides and their fragments. In some cases partitions are formed from droplets emulsions or spots on a substrate. Weizmann et al. Nature Methods 2006 Vol. 3 No. 7 pages 545 550 . Suitable methods for forming emulsions which can be used as partitions or to generate microcapsules include the methods described in Weitz et al. U.S. Pub. No. 2012 0211084 . Partitions may also be formed through the use of wells microwells multi well plates and microwell arrays. Partitioning may be performed using piezoelectric droplet generation e.g. Bransky et al. 2009 9 516 520 . Partitioning may be performed using surface acoustic waves e.g. Demirci and Montesano 2007 7 1139 1145 .

Such partitions may be pre loaded with reagents to perform a particular reaction. For example a capsule containing one or more reagents may be placed within a microwell. After adding a polynucleotide sample to the well the capsule may be made to release its contents. The contents of the capsule may include for example restriction enzymes ligases barcodes and adapters for processing the polynucleotide sample placed in the microwell.

In some cases such partitions may be droplets of an emulsion. For example a droplet of an emulsion may be an aqueous droplet in an oil phase. The droplet may comprise for example one or more reagents e.g. restriction enzymes ligases polymerases reagents necessary for nucleic acid amplification e.g. primers DNA polymerases dNTPs buffers a polynucleotide sample and a barcode sequence. In some cases the barcode sequence polynucleotide sample or any reagent may be associated with a solid surface within a droplet. In some cases the solid surface is a bead. In some cases the bead is a gel bead see e.g. Agresti et al. U.S. Patent Publication No. 2010 0136544 . In some cases the droplet is hardened into a gel bead e.g. via polymerization .

Polynucleotides may be partitioned using a variety of methods. For example polynucleotides may be diluted and dispensed across a plurality of partitions. A terminal dilution of a medium comprising polynucleotides may be performed such that the number of partitions or wells exceeds the number of polynucleotides. The ratio of the number of polynucleotides to the number of partitions may range from about 0.1 10 0.5 10 1 10 2 10 10 100 100 1000 or more. The ratio of the number of polynucleotides to the number of partitions may be about 0.1 0.5 1 2 4 8 10 20 50 100 or 1000. The ratio of the number of polynucleotides to the number of partitions may be at least about 0.1 0.5 1 2 4 8 10 20 50 100 or 1000. The ratio of the number of polynucleotides to the number of partitions may be less than about 0.1 0.5 1 2 4 8 10 20 50 100 or 1000.

The number of partitions employed may vary depending on the application. For example the number of partitions may be about 5 10 50 100 250 500 750 1000 1500 2000 2500 5000 7500 or 10 000 or more. The number of partitions may be at least about 5 10 50 100 250 500 750 1000 1500 2000 2500 5000 7500 or 10 000 or more. The number of partitions may be less than about 5 10 50 100 250 500 750 1000 1500 2000 2500 5000 7500 or 10 000.

The volume of the partitions may vary depending on the application. For example the volume of the partitions may be about 1000 l 900 l 800 l 700 l 600 l 500 l 400 l 300 l 200 l 100 l 50 l 25 l 10 l 5 l 1 l 900 nL 800 nL 700 nL 600 nL 500 nL 400 nL 300 nL 200 nL 100 nL 50 nL 25 nL 10 nL or 5 nL. The volume of the partitions may be at least about 1000 l 900 l 800 l 700 l 600 l 500 l 400 l 300 l 200 l 100 l 50 l 25 l 10 l 5 l 1 l 900 nL 800 nL 700 nL 600 nL 500 nL 400 nL 300 nL 200 nL 100 nL 50 nL 25 nL 10 nL or 5 nL. The volume of the partitions may be less than about 1000 l 900 l 800 l 700 l 600 l 500 l 400 l 300 l 200 l 100 l 50 l 25 l 10 l 5 l 1 l 900 nL 800 nL 700 nL 600 nL 500 nL 400 nL 300 nL 200 nL 100 nL 50 nL 25 nL 10 nL or 5 nL.

Particular polynucleotides may also be targeted to specific partitions. For example in some cases a capture reagent such as an oligonucleotide probe may be immobilized in a partition to capture specific polynucleotides through hybridization.

Polynucleotides may also be partitioned at a particular density. For example polynucleotides may be partitioned such that each partition contains about 1 5 5 10 10 50 50 100 100 1000 1000 10000 10000 100000 or 100000 1000000 polynucleotides per well. Polynucleotides may be partitioned so that each partition contains about 1 5 10 50 100 1000 10000 100000 1000000 or more polynucleotides per well. Polynucleotides may be partitioned so that each partition contains less than about 1 5 10 50 100 1000 10000 100000 or 1000000 polynucleotides per well. Polynucleotides may be partitioned so that each partition contains at least about 1 5 10 50 100 1000 10000 100000 or 1000000 polynucleotides per well.

Polynucleotides may be partitioned such that at least one partition comprises a polynucleotide sequence with a unique sequence compared to all other polynucleotide sequences contained within the same partition. This may be true for about 1 5 10 20 30 40 50 60 70 80 90 or more of the partitions. This may be true for less than about 1 5 10 20 30 40 50 60 70 80 90 or more of the partitions. This may be true for more than about 1 5 10 20 30 40 50 60 70 80 90 or more of the partitions.

Downstream applications for example DNA sequencing may rely on the barcodes to identify the origin of a sequence and for example to assemble a larger sequence from sequenced fragments. Therefore it may be desirable to add barcodes to the polynucleotide fragments generated by the methods described herein. Barcodes may be of a variety of different formats including polynucleotide barcodes. Depending upon the specific application barcodes may be attached to polynucleotide fragments in a reversible or irreversible manner. Barcodes may also allow for identification and or quantification of individual polynucleotide fragments during sequencing.

Barcodes may be loaded into partitions so that one or more barcodes are introduced into a particular partition. Each partition may contain a different set of barcodes. This may be accomplished by directly dispensing the barcodes into the partitions enveloping the barcodes e.g. in a droplet of an emulsion or by placing the barcodes within a container that is placed in a partition e.g. a microcapsule .

For example a population of microcapsules may be prepared such that a first microcapsule in the population comprises multiple copies of identical barcodes e.g. polynucleotide bar codes etc. and a second microcapsule in the population comprises multiple copies of a barcode that differs from the barcode within the first microcapsule. In some cases the population of microcapsules may comprise multiple microcapsules e.g. greater than about 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 100 500 1000 5000 10000 100000 1000000 10000000 100000000 or 1000000000 microcapsules each containing multiple copies of a barcode that differs from that contained in the other microcapsules. In some cases the population may comprise greater than about 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 100 500 1000 5000 10000 100000 1000000 10000000 100000000 or 1000000000 microcapsules with identical sets of barcodes. In some cases the population may comprise greater than about 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 100 500 1000 5000 10000 100000 1000000 10000000 100000000 or 1000000000 microcapsules wherein the microcapsules each comprise a different combination of barcodes. For example in some cases the different combinations overlap such that a first microcapsule may comprise e.g. barcodes A B and C while a second microcapsule may comprise barcodes A B and D. In another example the different combinations do not overlap such that a first microcapsule may comprise e.g. barcodes A B and C while a second microcapsule may comprise barcodes D E and F. The use of microcapsules is of course optional. All of the combinations described above and throughout this disclosure may also be generated by dispending barcodes and other reagents directly into partitions e.g. microwells .

The barcodes may be loaded into the partitions at an expected or predicted ratio of barcodes per species to be barcoded e.g. polynucleotide fragment strand of polynucleotide cell etc. . In some cases the barcodes are loaded into partitions such that more than about 1 2 3 4 5 6 7 8 9 10 20 50 100 500 1000 5000 10000 or 200000 barcodes are loaded per species. In some cases the barcodes are loaded in the partitions so that less than about 1 2 3 4 5 6 7 8 9 10 20 50 100 500 1000 5000 10000 or 200000 barcodes are loaded per species. In some cases the average number of barcodes loaded per species is less than or greater than about 0.0001 0.001 0.01 0.1 1 2 3 4 5 6 7 8 9 10 20 50 100 500 1000 5000 10000 or 200000 barcodes per species.

When more than one barcode is present per polynucleotide fragment such barcodes may be copies of the same barcode or multiple different barcodes. For example the attachment process may be designed to attach multiple identical barcodes to a single polynucleotide fragment or multiple different barcodes to the polynucleotide fragment.

The methods provided herein may comprise loading a partition e.g. a microwell droplet of an emulsion with the reagents necessary for the attachment of barcodes to polynucleotide fragments. In the case of ligation reactions reagents including restriction enzymes ligase enzymes buffers adapters barcodes and the like may be loaded into a partition. In the case barcoding by amplification reagents including primers DNA polymerases DNTPs buffers barcodes and the like may be loaded into a partition. As described throughout this disclosure these reagents may be loaded directly into the partition or via a container such as a microcapsule. If the reagents are not disposed within a container they may be loaded into a partition e.g. a microwell which may then be sealed with a wax or oil until the reagents are used.

Barcodes may be ligated to a polynucleotide fragment using sticky or blunt ends. Barcoded polynucleotide fragments may also be generated by amplifying a polynucleotide fragment with primers comprising barcodes.

Barcodes may be assembled combinatorially from smaller components designed to assemble in a modular format. For example three modules 1A 1B and 1C may be combinatorially assembled to produce barcode 1ABC. Such combinatorial assembly may significantly reduce the cost of synthesizing a plurality of barcodes. For example a combinatorial system consisting of 3 A modules 3 B modules and 3 C modules may generate 3 3 3 27 possible barcode sequences from only 9 modules.

Microcapsules and microwell capsule array MCA devices may be used to perform the polynucleotide processing methods described herein. MCA devices are devices with a plurality of microwells. Microcapsules are introduced into these microwells before after or concurrently with the introduction of a sample.

Microwells may comprise free reagents and or reagents encapsulated in microcapsules. Any of the reagents described in this disclosure may be encapsulated in a microcapsule including any chemicals particles and elements suitable for sample processing reactions involving a polynucleotide. For example a microcapsule used in a sample preparation reaction for DNA sequencing may comprise one or more of the following reagents enzymes restriction enzymes e.g. multiple cutters ligase polymerase fluorophores oligonucleotide barcodes adapters buffers dNTPs ddNTPs and the like.

Additional exemplary reagents include buffers acidic solution basic solution temperature sensitive enzymes pH sensitive enzymes light sensitive enzymes metals metal ions magnesium chloride sodium chloride manganese aqueous buffer mild buffer ionic buffer inhibitor enzyme protein polynucleotide antibodies saccharides lipid oil salt ion detergents ionic detergents non ionic detergents oligonucleotides nucleotides deoxyribonucleotide triphosphates dNTPs dideoxyribonucleotide triphosphates ddNTPs DNA RNA peptide polynucleotides complementary DNA cDNA double stranded DNA dsDNA single stranded DNA ssDNA plasmid DNA cosmid DNA chromosomal DNA genomic DNA viral DNA bacterial DNA mtDNA mitochondrial DNA mRNA rRNA tRNA nRNA siRNA snRNA snoRNA scaRNA microRNA dsRNA ribozyme riboswitch and viral RNA polymerase ligase restriction enzymes proteases nucleases protease inhibitors nuclease inhibitors chelating agents reducing agents oxidizing agents fluorophores probes chromophores dyes organics emulsifiers surfactants stabilizers polymers water small molecules pharmaceuticals radioactive molecules preservatives antibiotics aptamers and pharmaceutical drug compounds.

In some cases a microcapsule comprises a set of reagents that have a similar attribute e.g. a set of enzymes a set of minerals a set of oligonucleotides a mixture of different bar codes a mixture of identical bar codes . In other cases a microcapsule comprises a heterogeneous mixture of reagents. In some cases the heterogeneous mixture of reagents comprises all components necessary to perform a reaction. In some cases such mixture comprises all components necessary to perform a reaction except for 1 2 3 4 5 or more components necessary to perform a reaction. In some cases such additional components are contained within a different microcapsule or within a solution within a partition e.g. microwell of the device.

In some cases only microcapsules comprising reagents are introduced. In other cases both free reagents and reagents encapsulated in microcapsules are loaded into the device either sequentially or concurrently. In some cases reagents are introduced to the device either before or after a particular step. In some cases reagents and or microcapsules comprising reagents are introduced sequentially such that different reactions or operations occur at different steps. The reagents or microcapsules may be also be loaded at steps interspersed with a reaction or operation step. For example microcapsules comprising reagents for fragmenting polynucleotides e.g. restriction enzymes may be loaded into the device followed by loading of microcapsules comprising reagents for ligating bar codes and subsequent ligation of the bar codes to the fragmented molecules.

Microcapsules may be pre formed and filled with reagents by injection. For example the picoinjection methods described in Abate et al. Proc. Natl. Acad. Sci. U.S.A. 2010 107 45 19163 19166 and Weitz et al. U.S. Pub. No. 2012 0132288 may be used to introduce reagents into the interior of microcapsules described herein. These methods can also be used to introduce a plurality of any of the reagents described herein into microcapsules.

Microcapsules may be formed by any emulsion technique known in the art. For example the multiple emulsion technique of Weitz et al. U.S. Pub. No. 2012 0211084 may be used to form microcapsules or partitions for use with the methods disclosed herein.

Generally the methods and compositions provided herein are useful for preparation of polynucleotide fragments for downstream applications such as sequencing. Sequencing may be performed by any available technique. For example sequencing may be performed by the classic Sanger sequencing method. Sequencing methods may also include high throughput sequencing pyrosequencing sequencing by synthesis single molecule sequencing nanopore sequencing sequencing by ligation sequencing by hybridization RNA Seq Illumina Digital Gene Expression Helicos next generation sequencing single molecule sequencing by synthesis SMSS Helicos massively parallel sequencing clonal single molecule Array Solexa shotgun sequencing Maxim Gilbert sequencing primer walking and any other sequencing methods known in the art.

In some cases varying numbers of fragments are sequenced. For example in some cases about 30 90 of the fragments are sequenced. In some cases about 35 85 40 80 45 75 50 70 55 65 or 50 60 of the fragments are sequenced. In some cases at least about 30 40 50 60 70 80 or 90 of the fragments are sequenced. In some cases less than about 30 40 50 60 70 80 or 90 of the fragments are sequenced.

In some cases sequences from fragments are assembled to provide sequence information for a contiguous region of the original target polynucleotide that is longer than the individual sequence reads. Individual sequence reads may be about 10 50 50 100 100 200 200 300 300 400 or more nucleotides in length.

The identities of the barcode tags may serve to order the sequence reads from individual fragments as well as to differentiate between haplotypes. For example during the partitioning of individual fragments parental polynucleotide fragments may separated into different partitions. With an increase in the number of partitions the likelihood of a fragment from both a maternal and paternal haplotype contained in the same partition becomes negligibly small. Thus sequence reads from fragments in the same partition may be assembled and ordered.

This disclosure also provides methods and compositions to prepare polynucleotide fragments in such a manner that may enable phasing or linkage information to be generated. Such information may allow for the detection of linked genetic variations in sequences including genetic variations e.g. SNPs mutations indels copy number variations transversions translocations inversions etc. that are separated by long stretches of polynucleotides. The term indel refers to a mutation resulting in a colocalized insertion and deletion and a net gain or loss in nucleotides. A microindel is an indel that results in a net gain or loss of 1 to 50 nucleotides. These variations may exist in either a cis or trans relationship. In a cis relationship two or more genetic variations exist in the same polynucleotide or strand. In a trans relationship two or more genetic variations exist on multiple polynucleotide molecules or strands.

Methods provided herein may be used to determine polynucleotide phasing. For example a polynucleotide sample e.g. a polynucleotide that spans a given locus or loci may be partitioned such that at most one molecule of polynucleotide is present per partition e.g. microwell . The polynucleotide may then be fragmented barcoded and sequenced. The sequences may be examined for genetic variation. The detection of genetic variations in the same sequence tagged with two different bar codes may indicate that the two genetic variations are derived from two separate strands of DNA reflecting a trans relationship. Conversely the detection of two different genetic variations tagged with the same bar codes may indicate that the two genetic variations are from the same strand of DNA reflecting a cis relationship.

Phase information may be important for the characterization of a polynucleotide fragment particularly if the polynucleotide fragment is derived from a subject at risk of having or suspected of a having a particular disease or disorder e.g. hereditary recessive disease such as cystic fibrosis cancer etc. . The information may be able to distinguish between the following possibilities 1 two genetic variations within the same gene on the same strand of DNA and 2 two genetic variations within the same gene but located on separate strands of DNA. Possibility 1 may indicate that one copy of the gene is normal and the individual is free of the disease while possibility 2 may indicate that the individual has or will develop the disease particularly if the two genetic variations are damaging to the function of the gene when present within the same gene copy. Similarly the phasing information may also be able to distinguish between the following possibilities 1 two genetic variations each within a different gene on the same strand of DNA and 2 two genetic variations each within a different gene but located on separate strands of DNA.

Methods provided herein may also be used to prepare polynucleotide contained within cells in a manner that enables cell specific information to be obtained. The methods enable detection of genetic variations e.g. SNPs mutations indels copy number variations transversions translocations inversions etc. from very small samples such as from samples comprising about 10 100 cells. In some cases about 1 5 10 20 30 40 50 60 70 80 90 or 100 cells may be used in the methods described herein. In some cases at least about 1 5 10 20 30 40 50 60 70 80 90 or 100 cells may be used in the methods described herein. In other cases at most about 5 10 20 30 40 50 60 70 80 90 or 100 cells may be used in the methods described herein.

In an example a method comprises partitioning a cellular sample or crude cell extract such that at most one cell or extract of one cell is present per partition lysing the cells fragmenting the polynucleotides contained within the cells by any of the methods described herein attaching the fragmented polynucleotides to barcodes pooling and sequencing.

As described elsewhere herein the barcodes and other reagents may be contained within a microcapsule. These microcapsules may be loaded into a partition e.g. a microwell before after or concurrently with the loading of the cell such that each cell is contacted with a different microcapsule. This technique may be used to attach a unique barcode to polynucleotides obtained from each cell. The resulting tagged polynucleotides may then be pooled and sequenced and the barcodes may be used to trace the origin of the polynucleotides. For example polynucleotides with identical barcodes may be determined to originate from the same cell while polynucleotides with different barcodes may be determined to originate from different cells.

The methods described herein may be used to detect the distribution of oncogenic mutations across a population of cancerous tumor cells. For example some tumor cells may have a mutation or amplification of an oncogene e.g. HER2 BRAF EGFR KRAS in both alleles homozygous others may have a mutation in one allele heterozygous and still others may have no mutation wild type . The methods described herein may be used to detect these differences and also to quantify the relative numbers of homozygous heterozygous and wild type cells. Such information may be used for example to stage a particular cancer and or to monitor the progression of the cancer and its treatment over time.

In some examples this disclosure provides methods of identifying mutations in two different oncogenes e.g. KRAS and EGFR . If the same cell comprises genes with both mutations this may indicate a more aggressive form of cancer. In contrast if the mutations are located in two different cells this may indicate that the cancer is more benign or less advanced.

Methods of the disclosure may be applicable to processing samples for the detection of changes in gene expression. A sample may comprise a cell mRNA or cDNA reverse transcribed from mRNA. The sample may be a pooled sample comprising extracts from several different cells or tissues or a sample comprising extracts from a single cell or tissue.

Cells may be placed directly into an partition e.g. a microwell and lysed. After lysis the methods of the invention may be used to fragment and barcode the polynucleotides of the cell for sequencing. Polynucleotides may also be extracted from cells prior to introducing them into a partition used in a method of the invention. Reverse transcription of mRNA may be performed in a partition described herein or outside of such a partition. Sequencing cDNA may provide an indication of the abundance of a particular transcript in a particular cell over time or after exposure to a particular condition.

The methods presented throughout this disclosure provide several advantages over current polynucleotide processing methods. First inter operator variability is greatly reduced. Second the methods may be carried out in microfluidic devices which have a low cost and can be easily fabricated. Third the controlled fragmentation of the target polynucleotides allows the user to produce polynucleotide fragments with a defined and appropriate length. This aids in partitioning the polynucleotides and also reduces the amount of sequence information loss due to the present of overly large fragments. The methods and systems also provide a facile workflow that maintains the integrity of the processed polynucleotide. Additionally the use of restriction enzymes enables the user to create DNA overhangs sticky ends that may be designed for compatibility with adapters and or barcodes.

This example demonstrates a method for the generation of non overlapping DNA fragments suitable for DNA sequencing and other downstream applications. An implementation of this method is schematically illustrated in .

With reference to a target polynucleotide genomic DNA is fragmented with the enzyme NotI to generate a plurality of non overlapping first polynucleotide fragments . The first polynucleotide fragments are partitioned into separate microwells in a microdevice such that each microwell comprises a plurality of fragments but only a single fragment with a particular sequence . The left hand side of illustrates three microwells one is labeled each containing three exemplary unique fragments corresponding to the first polynucleotide fragments . Referring again to the left hand side of the left most well contains fragments A1 B2 and C3 the middle well contains fragments B1 A2 and A3 and the right most well contains fragments C1 C2 and B3.

The partitioned fragments are then further fragmented to generate a plurality of non overlapping second polynucleotide fragments . Referring again to the left hand side of each member of the second polynucleotide fragments is designated by its first fragment identifier e.g. A1 B2 etc. followed by a 1 or a 2 . For example first fragment A1 is fragmented to produce second fragments A1 1 and A1 2. First fragment B2 is fragmented to produce second fragments B2 1 and B2 2 and so on. For the sake of simplicity only two second fragments are shown for each first fragment. This is of course not meant to be limiting as any number of fragments may be generated at any step of the process.

The second set polynucleotide fragments are barcoded and the barcoded sequences are pooled. Referring to the lower left hand side of the labels 1 2 and 3 represent three different barcode sequences used to label the second fragments . The labeled sequences are designated . Optionally adapter sequences not shown are used to make the second fragments compatible for ligation with the barcodes. The barcoding is performed while the fragments are still partitioned before pooling. The pooled barcoded sequences are then sequenced.

With continued reference to the methods described above are then repeated using a second rare cutter enzyme XmaIII to digest the genomic DNA and generate a plurality of non overlapping third polynucleotide fragments . The third polynucleotide fragments and the first polynucleotide fragments are overlapping because they are generated with different rare cutter enzymes that cut the target polynucleotides at different sites. The third polynucleotide fragments are partitioned into separate microwells in a microdevice such that each microwell comprises a plurality of fragments but only a single fragment with a particular sequence . The right hand side of illustrates three microwells one is labeled each containing three exemplary unique fragments corresponding to the third polynucleotide fragments . Referring again to the right hand side of the left most well contains fragments D1 E2 and F3 the middle well contains fragments E1 D2 and D3 and the right most well contains fragments F1 F2 and E3.

With continued reference to The partitioned fragments are then further fragmented to generate a plurality of non overlapping fourth polynucleotide fragments . The fourth polynucleotide fragments and the second polynucleotide fragments are overlapping because they are generated by fragmenting the third and first fragments respectively which were generated with rare cutter enzymes that cut the target polynucleotide at different sites as described above. Referring again to the right hand side of each member of the fourth set of polynucleotide fragments is designated by its third fragment identifier e.g. D1 E2 etc. followed by a 1 or a 2 . For example third fragment D1 is fragmented to produce fourth fragments D1 1 and D1 2. Third fragment E2 is fragmented to produce fourth fragments E2 1 and E2 2 and so on. For the sake of simplicity only two fourth fragments are shown for each third fragment. This is of course not meant to be limiting as any number of fragments may be generated.

The fourth polynucleotides fragments are barcoded and the barcoded sequences are pooled. Referring to the lower right hand side of the numbers 4 5 and 6 represent three different barcode sequences used to label the fourth fragments . The labeled sequences are designated . Optionally adapter sequences not shown are used to make the fourth fragments compatible for ligation with the barcodes. The barcoding is performed while the fragments are still partitioned before pooling. The pooled barcoded sequences are then sequenced.

The example above describes sequencing the barcoded second fragments separately from the barcoded fourth fragments. The barcoded second fragments and the barcoded fourth fragments may also be combined and the combined sample may be sequenced. One or more steps of the process may be carried out in a device. The steps carried out in a device may be carried out in the same device or in different devices.

After sequencing sequence contigs are assembled and the overlapping sequences between the second fragments and the fourth fragments are used to assemble the sequence of the genome.

A simulation was performed to evaluate the size distribution of fragments generated by a 6Mer cutter StuI a 4Mer cutter CviQI and two to seven 4Mer cutters. Random 1 Mbp DNA sequences were generated in silico and cuts were simulated based on the occurrence of the recognition sites for each of the restriction enzymes within the random sequences.

As shown in 3 of 271 fragments 1.5 were considered low yield since they were 50 bases or smaller. Fourteen fragments 5 were considered high accuracy since they were 200 bases or smaller i.e. each base of the fragment is within 100 bases of a restriction site and could be sequenced with high accuracy . Eleven fragments 4 were between 200 and 400 bases and would generate data that is both accurate 0 100 bases from each end and inaccurate 100 200 bases from each end . The remaining 246 fragments 91 were greater than 400 bases and would generate accurate 0 100 inaccurate 100 200 and no 200 bases from a restriction site sequence data. Overall only 5 of the 1 Mbp random sequence was within 100 bases from a restriction site and would generate accurate sequence data.

Next simulated cuts were made in a random 1 Mbp DNA sequence using combinations of one to seven different 4Mer cutters. The 4Mer cutters were A CviQI G TAC B BfaI C TAG C HinPII G CGC D CviAII C ATG E Taq I T CGA F MseI T TAA and G MspI C CGG . The results of these simulations are shown in . As shown in increasing the number of 4Mer cutter enzymes from one to seven increases the number of fragments with nucleotides within 100 nucleotides of a restriction site. Therefore cutting the randomly generated 1 Mbp DNA sequence with more than one 4Mer cutter results in more fragments that may be fully sequenced than cutting with a single 4Mer cutter.

The number of enzymes used to cut a sequence can be chosen so that a particular fraction of a target nucleotide e.g. a genomic sequence within 100 nucleotides of a restriction enzyme is achieved. For example the fraction of a random genome within 100 nucleotides of a restriction site for a 4Mer cutter is equal to 1 0.44 where x is the number of independent 4Mer cutters. Similarly the fraction of a random genome within 100 nucleotides of a restriction site for a 5Mer cutter is equal to 1 0.25 where x is the number of independent 5Mer cutters. For a 6Mer cutter the fraction of a random genome within 100 nucleotides of a restriction site is equal to 1 0.95 where x is the number of independent 6Mer cutters.

Table 1 shows the percentage of sequences with a length greater than 100 nucleotides for each of the seven enzymatic treatments described above. These sequences are considered those likely to result in missing data. Increasing the number of enzymes decreases the percentage of sequences greater than 100 nucleotides. The number of enzymes and their restriction site recognition length may be chosen in order to minimize the loss of sequence information from sequences greater than 100 nucleotides from a restriction site while also minimizing the generation of sequences less than 50 nucleotides which are undesirable because the underutilize the read length capacity of sequencing instruments. The presence of these fragments may be minimized or avoided by selecting restriction enzymes that cut more rarely but at the potential price of reduced sequencing coverage of the DNA i.e. more fragments may have bases 100 bases from a restriction site . These fragments may also be physically removed by a size selection step. Since these fragments are small and some fraction of the bases represented in the small fragments may be covered in larger fragments from other enzymes the effect on coverage would likely be minimal.

The exemplary 4Mer cutter methods presented herein are optimized to provide fragments compatible with current DNA sequencing technology which may achieve accurate read lengths up to about 100 nucleotides from the terminus of a fragment. One of ordinary skill in the art will readily recognize that other restriction enzymes e.g. 5Mer cutters 6Mer cutters etc. would be suitable for DNA sequencing technologies capable of accurately reading larger fragments of DNA e.g. 300 400 or more nucleotides . The methods presented in this disclosure are of course generalizable and may be used to obtain DNA fragments of any size distribution compatible with present or future sequencing technology.

As described elsewhere herein many downstream applications of the polynucleotide processing methods provided herein may utilize polynucleotide barcodes. An adapter may be used to provide compatible ends for the attachment of a barcode to a polynucleotide fragment e.g. by ligation or PCR . In these cases the desired products may be for example B TPF B or B A TPF A B where B represents a barcode A represents an adapter and TPF represents a target polynucleotide fragment. However in some cases undesirable side products may form for example from the self ligation of barcodes adapters and or target polynucleotide fragments. This example demonstrates one solution to this potential problem.

However the reaction described above may also result in several unwanted side products including multimers produced by self ligation of the fragmented genomic DNA and adapters or other molecules such as barcodes which are not shown . For the sake of simplicity illustrates this concept by showing only self ligation of fragmented genomic DNA and adapters.

One unwanted side product is a multimer of genomic DNA fragments. This may occur for example if genomic DNA fragments with compatible ends are ligated to each other after cutting. In cutting of Genomic DNA with MspI generates compatible ends that may be ligated by the ligase present in the partition. Similarly Adapter 1 and Adapter 2 as shown have compatible ligatable ends and may also be ligated to form multimers.

As indicated in one solution to this problem is to pair one enzyme in this example MspI with a second enzyme in this example NarI . In this example MspI re cuts genomic DNA multimers produced by self ligation of genomic DNA fragments. Therefore MspI recycles unwanted genomic DNA fragment multimers back into genomic DNA fragments which may then be correctly ligated to the adapters. Similarly NarI cuts multimers of Adapter 1 and Adapter 2 into monomers of Adapter 1 and monomers of Adapter 2 which may then be correctly ligated to genomic DNA fragments. This recycles unwanted adapter multimers back into the desired starting materials of Adapter 1 and Adapter 2.

The enzymes are chosen such that the desired product i.e. the genomic DNA fragment with adapters on each end does not contain a recognition site for either enzyme. Therefore the product will not be re cut by any enzyme contained within the partition. This process increases the yield of the desired product while minimizing the number of unwanted side products and reducing the amount of starting material required to produce a desired amount of a product. As described in this disclosure a pair of enzymes may be chosen so that one enzyme recognizes one undesirable side product and regenerates a starting material and another recognizes another undesirable side product and regenerates another starting material but neither enzyme recognizes the desired product. This can be done for an unlimited number of side products.

In general one strategy for selecting such pairs is to choose two enzymes that create identical or similar ligatable termini after cutting but have recognition sequences of different lengths. shows examples of such pairs of enzymes. The enzymes provided in provide sticky ends while those provided in provide blunt ends.

The exemplary embodiment shown in uses Genomic DNA and two adapters Adapter 1 and Adapter 2 as starting materials. Therefore in this embodiment MspI is used not only to regenerate genomic DNA fragments after self ligation but also to generate the genomic DNA fragments in the first place from Genomic DNA. Of course this is optional as one may introduce pre fragmented genomic DNA into the partition and the method is still applicable.

Similarly the embodiment shown in shows two separate adapter molecules as starting materials. Adapter molecules may also be provided as a single polynucleotide sequence which is then cut by an enzyme contained within the partition in this example NarI to generate ligation compatible ends for attachment to the fragmented genomic DNA. The method is also applicable to other polynucleotides described throughout this disclosure and to methods of attachment based on techniques other than ligation e.g. attachment of an adapter or a barcode by PCR .

Pseudo complimentary nucleotides that preferentially bind natural nucleotides over themselves e.g. 1996 35 11170 11176 1996 15 2470 2475 may also be used to minimize or avoid the formation of certain multimers for example adapter adapter multimers and barcode barcode multimers. If adapters and or barcodes and or other polynucleotides are synthesized using pseudo complimentary nucleotides they will prefer to hybridize with naturally occurring polynucleotide fragments e.g. genomic DNA fragments rather than themselves therefore leading to a higher yield of the desired product.

As described throughout this disclosure the polynucleotide processing methods described herein may involve the treatment of partitioned polynucleotides with a variety of reagents. These reagents may include for example restriction enzymes ligases phosphatases kinases barcodes adapters or any other reagent useful in polynucleotide processing or in a downstream application such as sequencing. shows two exemplary methods of providing reagents. On the left hand side of reagents are provided within a microcapsule. The microcapsule that is shown in has an outer shell 3 an intermediate non aqueous layer 2 and an inner aqueous drop contained within the intermediate non aqueous layer 1ABC RE . This droplet is made by a water oil water emulsion technique followed by polymerization of the outermost water layer 3 to form a shell. Reagents are contained within the inner aqueous phase of the capsule. The left hand side of shows an exemplary embodiment with four reagents contained within the aqueous phase of the capsule namely three barcode reagents 1A 1B and 1C and a restriction enzyme RE . The embodiment shown is merely exemplary. The reagents may be located in any part of the capsule.

The capsule is dispensed into a partition e.g. a microwell . A target polynucleotide and a ligase are then added to the partition. The capsule is made to release its contents by exposure to a stimulus such as a change in temperature a solvent or stirring. The restriction enzyme fragments the target polynucleotide and the ligase attaches the barcode reagents to the target polynucleotide fragments generated by the restriction enzyme.

The restriction digestion and ligation may proceed according to any of the methods described herein for example by non overlapping fragmentation techniques by pseudo random fragmentation methods and or by pairing of restriction enzymes to recycle unwanted side products into new starting products e.g. target polynucleotide fragments and barcodes . Adapters may also be included within the microcapsule. The barcodes shown in are modular. For example barcode components 1A 1B and 1C may ligate to form barcode 1A 1B 1C .

The right hand side of shows the same reagents dispensed into a microwell followed by sealing with sealant e.g. a wax or oil to prevent evaporation before use. This approach may be substituted for the approach described above where the reagents are placed within microcapsules. Both approaches are used to produce partitions e.g. microwells pre loaded with reagents for DNA fragmentation and barcoding. In order to fragment and barcode DNA using reagents dispensed within a microwell a user unseals a partition and introduces a target polynucleotide and a ligase or any other reagents applicable for the method the user is conducting . As described above the restriction enzyme fragments the target polynucleotide and the ligase attaches the barcode reagents to the target polynucleotide fragments generated by the restriction enzyme. Of course both approaches may be combined by placing certain reagents in the microwell and others in the microcapsule.

While preferred embodiments of the present invention have been shown and described herein it will be obvious to those skilled in the art that such embodiments are provided by way of example only. Numerous variations changes and substitutions will now occur to those skilled in the art without departing from the invention. It should be understood that various alternatives to the embodiments of the invention described herein may be employed in practicing the invention. It is intended that the following claims define the scope of the invention and that methods and structures within the scope of these claims and their equivalents be covered thereby.

